%  ========================================================================
%  Copyright (c) 1995-2012 The University of Washington
%
%  Licensed under the Apache License, Version 2.0 (the "License");
%  you may not use this file except in compliance with the License.
%  You may obtain a copy of the License at
%
%      http://www.apache.org/licenses/LICENSE-2.0
%
%  Unless required by applicable law or agreed to in writing, software
%  distributed under the License is distributed on an "AS IS" BASIS,
%  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
%  See the License for the specific language governing permissions and
%  limitations under the License.
%  ========================================================================
%

% Documentation for UW thesis document style for LaTeX
% by Jim Fox
% fox@washington.edu
%
%    Revised for version 2012/06/19 of uwthesis.cls
%
%    This document is contained in a single file ONLY because
%    I wanted to be able to distribute it easily.  A real thesis ought
%    to be contained on many files (e.g., one for each chapter, at least).
%
%    To help you identify the files and sections in this large file
%    I use the string '==========' to identify new files.
%
%    To help you ignore the unusual things I do with this sample document
%    I try to use the notation
%       
%    % --- sample stuff only -----
%    special stuff for my document, but you don't need it in your thesis
%    % --- end-of-sample-stuff ---


%    Printed in twoside style now that that's allowed
%
 
\documentclass [12pt, twoside] {uwthesis}[2012/06/19]

% max's custom font addition

%\newcommand\customfont[1]{{\usefont{T1}{custom}{m}{n} #1 }}



%\usefont{T1}{custom}{m}{n} %#1

\usepackage{fontspec,xltxtra,xunicode}
\defaultfontfeatures{Mapping=tex-text}
\setromanfont[Mapping=tex-text]{Hoefler Text}
\setsansfont[Scale=MatchLowercase,Mapping=tex-text]{Gill Sans}
\setmonofont[Scale=MatchLowercase]{Andale Mono}
 \usepackage{gensymb}
%
% The following line would print the thesis in a postscript font 

% \usepackage{natbib}
% \def\bibpreamble{\protect\addcontentsline{toc}{chapter}{Bibliography}}


% INITIALIZE THINGS BEFORE HERE IF YOU WANT THINGS TO SHOW UP IN THE TOC
\setcounter{tocdepth}{1}  % Print the chapter and sections to the toc
 

% ==========   Local defs and mods
%

% --- sample stuff only -----
% These format the sample code in this document
\usepackage{hyperref}
\usepackage{graphicx}
\usepackage{wrapfig}
\usepackage{chngcntr}
\usepackage{longtable}
\usepackage{amsmath}
\usepackage{multicol}
\usepackage{supertabular}
\usepackage{float}
\DeclareGraphicsExtensions{.pdf,.png,.jpg}
\usepackage{caption}
%mv ~/Downloads/texshade.zip /Library/ \\

%\usepackage{texshade}
\usepackage{alltt}  % 
\newenvironment{demo}
  {\begin{alltt}\leftskip3em
     \def\\{\ttfamily\char`\\}%
     \def\{{\ttfamily\char`\{}%
     \def\}{\ttfamily\char`\}}}
  {\end{alltt}}
 
% metafont font.  If logo not available, use the second form
%
% \font\mffont=logosl10 scaled\magstep1
\let\mffont=\sf
% --- end-of-sample-stuff ---
 

\begin{document}
 
% ==========   Preliminary pages
%
% ( revised 2012 for electronic submission )
%

\prelimpages
 
%
% ----- copyright and title pages
%
\Title{Certain observations concerning the effects of epistasis on complex traits and the evolution of genomes.}
\Author{Maximilian Press}
\Year{2016}
\Program{Department of Genome Sciences}

\Chair{Christine Queitsch}{Associate Professor}{Department of Genome Sciences}
\Chair{Elhanan Borenstein}{Associate Professor}{Department of Genome Sciences}
\Signature{First committee member}
%\Signature{Next committee member}
%\Signature{etc}

\copyrightpage

% \titlepage  

% --- sample stuff only -----
% unusual footnote not found in a real thesis
% You just use the \titlepage as commented out above

%{$\degree$text{A dissertation %
%  \footnote[2]{an egocentric imitation, actually}\\
%  submitted in partial fulfillment of the\\ requirements for the degree of}
 %\def\thefootnote{\fnsymbol{footnote}}
 %\let\footnoterule\relax
 \titlepage
% }
\setcounter{footnote}{0}

% --- end-of-sample-stuff ---
 
%
% ----- signature and quoteslip are gone
%

%
% ----- abstract
%


\setcounter{page}{-1}
\abstract{%

The informational content of genomes is usually interpreted as a sum of one-to-one relationships between genotypes at certain genomic positions and phenotypic outcomes. While such interpretations have the virtue of simplicity, they are often unsuccessful in elucidating the working of biological systems. Many have called for such models to explicitly consider epistasis, which can  be defined as any consideration of interactions between genomic elements. In this thesis, I consider some empirical cases where epistasis may help us to understand how genomes evolve and how genotype-phenotype maps are built. In the first part of this thesis, I consider a particular case of a fast-evolving genetic element (the \emph{ELF3} short tandem repeat in \emph{Arabidopsis thaliana}) that shows widespread epistasis, and propose that such elements are likely to accumulate epistatic interactions by acting as mutational modifiers. This element is a polyglutamine-encoding trinucleotide in the \emph{A. thaliana} gene \emph{ELF3}. I go on to show some molecular mechanisms by which the element participates in epistasis, their phenotypic consequences, and make some observations on other short tandem repeats. Briefly, these observations suggest that we may be able to specifically identify such epistatic hubs among highly variable genetic elements. In the second part of this thesis, I start with the assumption of epistasis between genes, and explore how this assumption can be used to understand the evolution of bacterial genome content. First, I take Hsp90, the known epistatic hub, and infer its coevolution with other genes through coordinated gains and losses across bacterial diversity. I further extend the underlying phylogenetic model to predict new "clients" of bacterial Hsp90, which have remained elusive when pursued through purely experimental approaches. Collaborators were able to validate certain of these predicted clients. Last, I attempt an analogy between prokaryotic genome evolution and the much better-understood field of protein evolution. I propose that, like protein evolution by substitution, genome evolution by horizontal acquisition of genes is substantially constrained by epistasis. I go on to infer the existence of such epistatic dependencies, where one gene in an ancestral genome promotes the acquisition of a second gene. A network of such dependencies shows a chronological structuring of gene acquisitions through prokaryotic evolution, suggesting universal assembly patterns by which genomes acquire functions. I go on to show that these dependencies are taxonomically universal (i.e. not restricted to particular phyla), and that they are sufficient to make reasonably good predictions about what genes a genome will gain in the future. This predictability of genome evolution by horizontal transfer supports a major assertion of the protein evolutionists, that constraining epistasis leads to predictable evolutionary outcomes. Together, these observations indicate that the genetic architecture of traits and the content of genomes are shaped by the existence of networks of gene-gene dependencies, reflecting the complex wiring of underlying biological functions.

}
 
%
% ----- contents & etc.
%
\tableofcontents
\listoffigures
\listoftables   
%
% ----- glossary 
%

%
% ----- acknowledgments
%
\acknowledgments{% \vskip2pc
  % {\narrower\noindent
I would like to thank my advisors, Christine Queitsch and Elhanan Borenstein, for letting me work through my ideas as far as I did (and equally, for curtailing those ideas when they got ridiculous). I would also like to thank Bob Kaplan, Katie Peichel, and Sue Biggins for saving Elhanan and Christine some trouble in mentoring me before I started my doctoral work. 

I would like to thank my thesis committee: Joe Felsenstein, Willie Swanson, and Evgeni Sokurenko. Joe in particular was generous with his time throughout my studies, in discussing both technical points of character evolution and the history of genetics.

I would like to thank my co-belligerents in the Queitsch and the Borenstein labs, for what was surely a miracle of patience.

I would like to thank other trainees at the University of Washington for lots of discussions and lessons that have helped me in developing my ideas,  in particular Matthew Snyder. 

I would like to thank my parents for everything. 

I would like to thank everyone else, because there are a lot of you.

And Sarah.
% \par}
}

\newpage

\acknowledgments{

\begin{quotation}

\noindent I will ask you to mark again that rather typical feature of the development of our subject; how so much progress depends on the interplay of techniques, discoveries and new ideas, probably in that order of decreasing importance. \\
\textbf{Sydney Brenner}

\end{quotation}


\begin{verse}

That generation's dream, aviled \\
In the mud, in Monday's dirty light,

That's it, the only dream they knew, \\
Time in its final block, not time 

To come, a wrangling of two dreams. \\
Here is the bread of time to come, 

Here is its actual stone.  The bread \\
Will be our bread, the stone will be

Our bed and we shall sleep by night. \\
We shall forget by day, except

The moments when we choose to play \\
the imagined pine, the imagined jay.

\textbf{Wallace Stevens}

\end{verse}

}



%
% ----- dedication
%
\dedication{\begin{center}to my Sarah\end{center}}



%
% end of the preliminary pages


 
%
% ==========      Text pages
%

\textpages
 
% ========== Chapter 1
 
\chapter {Introduction} 
\section{Historical notes on heredity, genotypes, and phenotypes}
The first decades of the 20th century were an exciting time for genetics. Mendel's work had been rediscovered, Galton's was never forgotten, and the debate between Darwinists and Lamarckists was waged with increasingly precise experimental tests. The crucial question under study was the mechanism of heredity; that is, how are observable differences in character among organisms propagated across generations \cite{Johannsen1911}? For example, Galton chose to investigate the relative roles of `nature' and `nurture' by studying the characteristics of twins \cite{Galton1875}, and Johannsen self-fertilized crop plants to obtain genetically stable `pure lines' where the same qualities could be studied more exactly \cite{Johannsen1909}. These investigations almost unconsciously led to a further question: how does the ontogeny of an organism give rise to a character? Specifically, immediately following Johannsen, Woltereck used `pure lines' of Daphnia to show that specific morphological changes could be reproducibly achieved by either manipulating growth conditions or substituting different types of Daphnia  \cite{Woltereck1909}. 

The first question, of heredity, is by far the easier, having been answered in formal terms by the succeeding century of research into chromosome theory, genetic mapping, and a litany of other inheritance mechanisms. The second question, being much more open-ended, must be answered anew in each case. For instance, Woltereck's observation that the head-height of \emph{Hyalodaphnia cucullata} may be increased by a combination of heat and rich food is unlikely to generalize to the articulation of finger bones in humans, though in both cases heritable variation in these traits may be attributed to definite loci on inherited chromosomes in the respective organisms. However, I would argue that the examination of causes in the first question (heredity) requires a reasonably good answer to the second (physiological mechanism).

However, for many years our ability to ascertain such mechanism was essentially nil with respect to the molecular activity of the heritable material itself. It was generally agreed that some chemical activity probably distinguished genes, but the majority of important work was determined by studying segregation ratios from crosses. Nonetheless, the ideas of `genes' and of `genotypes' provided rich material for early geneticists, assisting the resolution of quite complicated segregation patterns. Specifically, understanding the gene as a discrete locus with influence upon a character (or `phenotype') allowed the development of Mendelian thought. The biometricians (such as Galton) were, in contrast, interested in exploring the phenomena underlying continuous variation in characters. This view of hereditary variation was apparently in conflict with the Mendelian model of a few discrete packets of  genetic information. Out of this debate, from the Mendelian side, came the now-familiar idea of the `gene' (a discrete genetic determinant of a character), the `genotype' (a fixed complement of genes in a given organism), and the `phenotype' (the directly observable character which can be measured upon a given organism) \cite{Johannsen1911}. 

Fisher put an end to this dispute with a magisterial mathematical framework \cite{Fisher1918} showing that Mendelian segregation of genes could in principle lead to the continuous variation in phenotypes observed by the biometricians. 

\subsection{Fisher's innovations.}
For the purposes of this thesis, I will emphasize some relevant conceptual changes ushered in by Fisher's quantitative genetic framework in his 1918 paper \cite{Fisher1918} and subsequent work. 

\subsubsection{Fisher on quantitative genetics} 
First, Fisher implicitly assumed there is some direct, biologically meaningful mapping between variation in phenotypes and genotypes, such that variation in the phenotype is decomposable into quantities attributable to specific genes. This anticipated the `genotype-phenotype map' concept popularized later \cite{Alberch1991}. Interestingly, this direct abstraction of the genotype idea had previously been explicitly discouraged by Johannsen \cite{Johannsen1911}, who cautioned that such a leap was dangerous in ignorance of the actual hereditary material and the mechanisms by which phenotypes were generated from the hereditary material. However, at the time the resolution of the Mendelian/biometrician divide was too desirable to be laid aside for such misgivings. In consequence, Fisher's framework  dealt with idealized, purely abstract genes, whose existence and influence had more to do with mathematical convenience than with direct observation. I emphasize this point because, in recent years, direct observation of genetic material has become routine, and the resulting data analyzed using Fisher's methods. 

Second, Fisher used a series of assumptions about the structure of populations, the number of relevant genes, and the way that genes work together to generate phenotype to create a mathematically tractable model of how phenotypes are created. Specifically, he assumed that the number of genes contributing to any phenotype was large, with relatively small contributions from each gene. When this is the case, and the population of organisms tends to infinity in size, then the phenotype in question will be normally distributed across the population. 

Together with the concept of the genotype mentioned above, Fisher shows that the normally distributed phenotypic variance (written $\sigma_P^2$) can be decomposed into independent portions attributable to each Mendelian `factor' or gene $i$ (among $n$ genes total) and to a non-genetic error term ($e$):
\begin{equation}
\sigma_P^2 = \sum\limits_{i=1}^{n}\sigma_i^2 + \sigma_{e}^2
\end{equation}
 This independence between genes leads to the property called `additivity', because the genetic variance of the phenotype can be computed as a simple linear combination $\sigma_a^2 =  \sum\limits_{i=1}^{n}\sigma_i^2$ of the variance attributable to each gene. Similarly, the expected value of the phenotype can be computed as a linear combination of the effects of each gene\footnote{The framework laid out in this 1918 paper is alternately famous for introducing one of the most popular statistical methods, the analysis of variance (ANOVA). The idea of decomposing variances was generalizable to any problem in detecting associations between a quantitative, normally-distributed dependent variable to discrete factorial variables. ANOVA has subsequently seen wide usage beyond genetics. Certain properties of ANOVA, specifically its deficiencies in jointly estimating main effects and interaction terms, have been criticized both in applied statistics \cite{Neyman1935} and quantitative genetics \cite{Lewontin1974}. These examples will be discussed briefly below.}:
\begin{eqnarray}\label{eqn:fisher_decompose}
E[Phenotype] = \alpha + \sum\limits_{i=1}^n \beta_iG_i + \epsilon
\end{eqnarray}
Where there are $n$ genes, $G_i$ is the a 0/1 indicator of an alternate allele at the $i$-th gene (or locus), $\beta_i$ is the the difference attributable to the alternate allele at the $i$-th locus, $\alpha$ is the intercept term (corresponding to the phenotype when all loci $G_i$ take the value 0), and $\epsilon$ is the error introduced by all other factors (sampling error, measurement error, environmental variation). For simplicity, this example considers only a haploid system. With many independent genes, this decomposability yields a series of predicted phenotypic correlations between relatives of different degree within the population. These phenotypic correlations are related to the `heritability' of phenotypes (sometimes written $h^2$), or the proportion of phenotypic variation that is attributable to genetic variation ($h^2 = \frac{\sigma_a^2}{\sigma_P^2}$). Estimates of these correlations were the real object of Fisher's study, which was formulating a Mendelian mathematical basis for the well-known phenotypic resemblance between relatives. 

Notably, Fisher also treated classes of phenotype-controlling genetic variation that would not show up in correlations among relatives due to distortions. These were interactions among genes, or `epistasis', which violated Fisher's assumption about the independence of genes. To introduce epistasis into the model of Equation ~\ref{eqn:fisher_decompose}, we can add terms corresponding to the interactions between genes:
\begin{equation} \label{eqn:addepi}
E[Phenotype] = \alpha + \sum\limits_{i=1}^n \beta_iG_i +\sum\limits_{i=1}^n \sum\limits_{j\neq i}^n \beta_{i,j}G_{i,j} + \epsilon.
\end{equation}
Where $G_{i,j}$ and $\beta_{i,j}$ correspond, respectively, to an indicator for the joint genotype at loci $i$ and $j$, and the effect attributable to the interaction (which can be non-symmetric). I shall discuss epistasis in more detail below, and simply note here that Fisher considered this class of genetic control of phenotype as a component of $\epsilon$: an acceptable error similar to error in measuring the phenotype. 

Thus, Fisher emphasizes the estimation of phenotype based on only the readily tractable component of genotypic variation, and does not claim to provide a causal model by which specific genes (as opposed to ideal Mendelian factors) influence phenotypes. In this, Fisher followed the example of the preceding Mendelians, for whom the purely hypothetical genotype was much less interesting than the ability to dissect phenotypic variation in terms of discrete factors varying between monolithic genetic varieties. However, as time went on, geneticists tended to replace the mathematical abstractions of genes in Fisher's model with alleles actually isolated in nature. While this was perhaps a natural application of the theory, it led to considerable problems when analyzing real data, as shall be discussed later. 

\subsubsection{Fisher on evolutionary genetics} 
In later work, Fisher exploited his ideas on quantitative genetics to derive a mathematical theory of how these hereditary principles would behave in an evolutionary setting \cite{Fisher1930}. Throughout, he relied strongly on Darwin's intuition that, in order for natural selection to operate on phenotype, phenotypic variation must be heritable \cite{Wimsatt1980}. Otherwise, selection will be ineffective at promoting phenotypic change from generation to generation. Fisher's prior work provided tools by which heritability could be directly estimated. In consequence, Fisher came to equate the rate of evolutionary change in a population with the heritability of the phenotype under selection. Stated differently, selection uses up genetic variation to effect phenotypic change. For instance, the change of a phenotype $Z$ in response to a selection on individuals with specific values of $Z$ can be written 
\begin{equation} \label{eqn:breeder}
\Delta Z = h^2S,
\end{equation}
 where $h^2$ is the heritability of $Z$, and $S$ signifies the intensity of selection \cite{Felsenstein2013}. In the ideal case, the trait of `fitness', or reproductive success, is substituted for the trait subject to quantitative analysis in Fisher's earlier work. 

This system provides an intuitive formulation by which evolution by selection can proceed within a population according to Mendelian principles of segregating factors controlling phenotypic variation, though once again the actual identities of these factors were in practice irrelevant. For example, if a population of organisms with a trait $Z$ with average value $\overline{Z}$ is subject to a selection under which the selected subpopulation $Z'$ has mean $\overline{Z'}$, then we can rewrite equation ~\ref{eqn:breeder} to give the expected phenotype of the next generation $Y$: 
\begin{equation} \label{eqn:responseselxn}
E[Y] = \overline{Z} + h^2(\overline{Z'} - \overline{Z})
\end{equation}
Thus showing a direct relationship between phenotypic change across generations and the genetic variance, and thereby showing how Mendelian factors can contribute to evolutionary change. Of course, under the standard model, this selection will reduce $h^2$ in the next generation by changing the allele frequencies of the population, which will now be somewhat biased towards the alleles responsible for the difference $\Delta Z$ by a quantity proportional to the initial $h^2$ \cite{Visscher2008}. It is in this sense that genetic variance can be used up in selection.
% follow up here with rest of example from joe's book. talk about ratio of selection coefficient S to population size

The assumptions of this framework are the same strong assumptions made for Fisher's quantitative genetics framework, concerning very large panmictic populations, where fitness is determined by a large number of independently-contributing genes. For instance, the $h^2$ mentioned above is sometimes called the `narrow-sense' heritability, in that it includes only additive genetic variation ($\sigma_a^2$), as opposed to the more inclusive `broad-sense heritability', which explicitly includes non-additive genetic phenomena such as epistasis and dominance. 

As previously mentioned, Fisher assumes that additivity is a satisfactory model for describing the relationship of genotypes, phenotypes, and fitness. In a Fisherian world, fitness can be visualized as a smooth hill that one is climbing, where the peak of the hill is the fitness optimum, and higher elevation indicates higher fitness. Because each step up the hill increases fitness, one is guaranteed to reach the peak if one always chooses paths of higher fitness; there is only one peak, and all paths lead monotonically from lower fitness to optimal fitness. 

\subsection{Wright's problems.}
The analogy of fitness as a landscape in which one climbs is generally traced back to Wright, some years younger than Fisher. Wright was dissatisfied with the idea of a single stable peak of fitness for a population, drawing on both theory of his own devising and multiple empirical examples from the literature\cite{Wright1931}. While the immediate and more famous disputes between Fisher and Wright centered on models for evolutionary genetics, at its root the argument came back to the problem of epistasis in quantitative genetics. In the remainder of this section I treat these subjects insofar as I find them relevant to the thesis.

\subsubsection{Mutations} % not necessary????
Following the earlier work of De Vries and Muller on spontaneous and induced mutations, Wright reminds us that mutations (which at that point were ascertained entirely according to phenotypic changes) 

mutations are deleterious, rare, because random change in complex organization is usually bad. complex organization implies interaction, combinatorics...

\subsubsection{Population structure}
Fisher once wrote to Wright (quoted in \cite{Wade1992}), ``...I believe that N [population size] must usually be the total population on the planet [of the organism in question]...'' This is obviously a rather expansive view of the idea of a population. Moreover, it is unclear the degree to which distinguishable local varieties should be included. Overall, it is unsatisfying that such an important assumption (large population size) should be so vaguely met. 

Thus, Fisher dealt with many mathematical complexities of evolutionary theory by simply assuming very large $N$ ($\frac{1}{4N} \ll s$ where $s$ is the selection coefficient, and $Nm \gg 1$ where $m$ is the rate of migration among populations, $Nu \gg 1$ where $u$ is the mutation rate) \cite{Fisher1930}. Wright investigated the consequences of varying $N$ extensively outside of these boundaries \cite{Wright1931}, finding qualitatively quite different behaviors among models. Specifically, selection acts relatively slowly in both very large populations (because of the extremely long time to fixation of beneficial alleles) and in very small populations even strong selection can fail to favor beneficial alleles (when $s < \frac{1}{N}$). Although Fisher favored the first of these scenarios, Wright judged that neither was fast enough to yield the observed diversity of life outside of special cases such as artificial selection of agricultural varieties.

Consequently, the more interesting cases are when $ s \not\gg u \not\gg \frac{1}{4N} $, an intermediate region of parameter space where selection can still act, but where genetic drift is strong enough to lead to substantial variation in gene frequencies under selection. Furthermore, the time required for a favorable allele to go to fixation will be somewhat smaller in a smaller population. 

However, Wright considered this to still be too inefficient to generate sufficient evolutionary change. He therefore considered the well-known observation that species tend to be found as dispersed, loosely defined populations in a `metapopulation' network, where individuals generally stay in the same place, but occasionally emigrate. 

Fisher wrote (also quoted in \cite{Wade1992}): 
\begin{quote}
...the population used to determine [the value of fitness] comprises, not merely the whole of a species in any one generation attaining maturity, but is conceived to contain all the genetic combinations possible, with frequencies appropriate to their actual probabilities of occurrence and survival, whatever these may be, and if the average is based upon the statures attained by all these genotypes in all possible environmental circumstances, with frequencies appropriate to the actual probabilities of encountering these circumstances. 
\end{quote}

\subsubsection{Speciation} % merge into last section

\section{The many names of epistasis.}
We shall now finally deal with epistasis, the central idea of this thesis. Epistasis is famously ill-defined \cite{Phillips1998,Phillips2008}, but it can be considered a rough-and-ready word to capture the influence of interactions between genetic loci. These interactions could take the form of physical associations between encoded gene products or residues, the joint action of two loci on a trait, or the relative ordering of genes in genetic pathways. By all accounts, cytological interactions between gene products occur at a massive scale \cite{Krogan2006}, and gene interactions are crucial for organismal viability under laboratory-derived mutant analysis \cite{Costanzo2010}. Less clear is the importance of epistasis for phenotypic variation in natural populations, which some claim to be negligible \cite{Hill2008} and others claim to be all-important \cite{Huang2012}. 

We are left in the curious situation where despite the overwhelming mechanistic importance of interactions, there are cogent and rigorous arguments for the unimportance of epistasis for heritable variation \cite{Robinson2014}. These arguments revolve centrally around the idea of additivity as introduced by Fisher \cite{Fisher1918}, which is invoked as the more important source of genetic variation. Mathematically, epistasis is defined as a quantitative departure from the expectations of additivity \cite{Cheverud1995,Poelwijk2015}, and thus it shall be instructive to briefly examine the idea of additivity before moving on to different definitions of epistasis relevant to this thesis.

\subsection{The uses of additivity.}
As may be seen in Equation ~\ref{eqn:fisher_decompose}, additive models consider that genes contribute to phenotype independently of one another, such that the effects of two loci are in no way dependent upon one another\footnote{This is true of normally distributed quantitative phenotypes; in contrast, for log-normally distributed phenotypes, the analogous multiplicative model is appropriate}. It has been widely acknowledged by both Fisher and his latter-day adherents that this model is a convenient approximation rather than truth, and widely defended as such on the basis of parsimony \cite{Fisher1918,Williams1966,Wimsatt1980,Wade1992,Skipper2002,Robinson2014}. Certainly, comparing Equation ~\ref{eqn:fisher_decompose} to Equation ~\ref{eqn:addepi}, one will immediately notice that the pairwise epistatic terms in the model of phenotype increase in the order of the square of $n$, the number of loci, whereas the additive terms are equal in number to $n$. The inclusion of interactions of higher order (3-way or 4-way in addition to pairwise) makes matters worse. In any non-trivial example, this can lead to an alarming number of parameters of the model, leading potentially to problems in both estimation and interpretation of the parameters. 

Nonetheless, since the assumption of additivity is so widely used, it is worth examining its meaning in the context of quantitative genetics. So let us consider some specific thought experiments, featuring a premature stop-inducing allele \emph{FBN1*} of the \emph{FBN1} gene that acts as an additive contributor to human height on the order of 10cm (a large effect) \cite{Robinson2014}. 

\begin{enumerate}
\item The allele \emph{FBN1'} was identified in human populations, probably of European descent given the geography of the research group that discovered such alleles. Among people of European descent, this allele then changes height by 10cm. If it is truly additive, genetic background is irrelevant, and thus we may expect the same effect in humans from other populations, and probably in Neanderthal or Denisovan individuals. To venture further, this allele would have the same 10cm effect (or a similar scaled transformation) in any metazoan, or plant, or bacterium.
\item The quantitative effects of allele \emph{FBN1'} must be independent of alleles at any other locus. There are many known alleles segregating in the human population with potentially refractory pleiotropic effects, such as recessive embryonic lethal mutations, but if \emph{FBN1'} is additive, it necessarily increases adult height by 10cm even in such cases.
\end{enumerate}
These cases are obviously specious, and no one would believe in the existence of such effects. Nonetheless, they may help us consider what we mean when we discuss additivity. The first thought experiment may be dismissed on a technicality: any estimate of an additive main effect is a local estimate specific to a population. Thus, the 10cm estimate is specific to the population of humans of European descent in which it was measured. This additive estimate should thus not be applied to any other group of humans or any other species without first re-measuring it in a group of humans of European descent and the other group in combination, potentially with an experimental design accounting for confounding from species differences. This would clearly be a very singular study, but it would be appropriate under the additive logic. 

The second case is somewhat trickier to dismiss. There is potentially a case to be made against it, but it is hard to argue that even the hypothetical \emph{FBN1'} effect is additive in such cases. It may follow then that there is no allele of any meaningful effect that is completely independent of the genetic background (alleles with no effect, however, are necessarily independent). Obviously, no researcher has ever argued otherwise. Instead, as mentioned above, reference is generally made to the parsimony and adequacy of additive models, rather than their accurate portrayal of causality (as is often repeated, ``all models are wrong, some models are useful''). 

The most sophisticated argument in favor of additive models, which was made by Fisher himself , is that epistatic causal effects are modeled as part of the additive variance \cite{Fisher1918,Cheverud1995,Wagner1998,Phillips2008,Robinson2014,Monahan2015}. In any given population, epistatic effects of a locus will have some average effect across the genotypes in the population, which can be approximated as part of the additive effect of that locus. With large sample sizes and a representative sample of genotypes and environmental conditions, this will lead to quite accurate phenotypic predictions, as stressed by the quote from Fisher above and by others \cite{Hill2008}. In such cases, it is argued that the explanatory sufficiency of the model is worth any number of conceptual inconsistencies (see for instance Box 2 of \cite{Robinson2014}).

\subsection{Statistical, physiological, and molecular epistases.}
As suggested above, even in the specific case of epistasis signifying gene interactions controlling complex traits, the word can have multiple meanings \cite{Phillips1998}. `Molecular epistasis' is perhaps the simple case where it has been shown, for instance through synthetic combination of single gene knockout mutants \cite{Costanzo2010}, that there is an interaction between genes. However, this synthetic genotype and these alleles may never have existed in nature, and thus they signify a mechanistic link rather than a source of phenotypic variation in natural populations. `Statistical' and `physiological' epistasis can, however, concern natural variation; I describe them briefly below but will otherwise direct the reader to Cheverud and Routland \cite{Cheverud1995} for a lucid description. 

Physiological epistasis may be measured by enumerating all genotypes of some set of loci and alleles, measuring the phenotypes of each genotype, and computing the difference between the phenotype of each genotype and the expected additive phenotype based on averaging of other genotypes which share the same alleles. Statistical epistasis may be measured more easily, by simply collecting a population sample of the organism in question, measuring their genotypes and phenotypes, and using a model like Equation ~\ref{eqn:addepi} to estimate the interaction terms by least squares or a similar method. Naturally, it is impractical to measure physiological or molecular epistasis for very many loci or alleles, but the estimates of physiological epistasis are necessarily more accurate \cite{Cheverud1995,Poelwijk2015}.  

To refer to the thought-experiments above while discussing the idea of additivity, I may perhaps clarify the differences between these types of epistasis. 


\section{Epistasis in the evolutionary process}

 
\subsection{Fast-evolving genetic elements}


\subsubsection{Short tandem repeats (STRs)}

\subsubsection{Horizontal acquisition of genetic material}


\subsection{Evolutionary implications of fast-evolving genome architectures}

\subsubsection{Tinkering and kludging in evolution}

\subsubsection{Parallelism and Predictability}


 
% ========== MAX put in a chapter here
\chapter{Background-dependent effects of polyglutamine variation in the \emph{Arabidopsis thaliana} gene \emph{ELF3}}

A version of this chapter was published under the following reference: 

Soledad F. Undurraga, Maximilian O. Press, Matthieu Legendre, Nora Bujdoso, Jacob Bale, Hui Wang, Seth J. Davis, Kevin J. Verstrepen, and Christine Queitsch. Background-dependent effects of polyglutamine variation in the \emph{Arabidopsis thaliana} gene \emph{ELF3}. Proceedings of the National Academy of Sciences of the United States of America, 109(47):19363-7, November 2012.

Soledad Undurraga, Jacob Bale, Nora Bujdoso, and Hui Wang contributed transgenic lines, experiments, and  figures. 

Supplementary figures and tables can be found in Appendix A.


\section{Summary}
Tandem repeats (TRs) have extremely high mutation rates and are often considered to be neutrally evolving DNA. However, in coding regions, TR copy number mutations can significantly affect phenotype and may facilitate rapid adaptation to new environments. In several human genes, TR copy number mutations that expand polyglutamine (polyQ) tracts beyond a certain threshold cause incurable neurodegenerative diseases. PolyQ-containing proteins exist at a considerable frequency in eukaryotes, yet the phenotypic consequences of natural variation in polyQ tracts that are not associated with disease remain largely unknown. Here, we use Arabidopsis thaliana to dissect the phenotypic consequences of natural variation in the polyQ tract encoded by \emph{EARLY FLOWERING 3} (\emph{ELF3}), a key developmental gene. Changing ELF3 polyQ tract length affected complex ELF3-dependent phenotypes in a striking and non-linear manner. Some natural ELF3 polyQ variants phenocopied elf3-loss-function mutants in a common reference background, although they are functional in their native genetic backgrounds. To test the existence of background-specific modifiers, we compared the phenotypic effects of ELF3 polyQ variants between two divergent backgrounds, Col and Ws, and found dramatic differences.  Our data support a model in which variable polyQ tracts drive adaptation to internal genetic environments. 

\section{Introduction}
In coding regions, tandem repeat (TR) copy number variation can have profound phenotypic effects \cite{Gemayel2010}. For example, TR copy number mutations that expand polyglutamine (polyQ) tracts past a threshold number of glutamines can cause incurable neurodegenerative diseases such as Huntington's disease and Spinocerebellar Ataxias \cite{Gatchel2005, Orr2012}. PolyQ tract length correlates with onset and severity of polyQ expansion disorders, but for intermediate polyQ tracts this correlation is far weaker (4-8), suggesting the existence of genetic and environmental modifiers (9-12). Despite their potential for pathogenicity, variable polyQ tracts occur frequently in eukaryotic proteins, many of them functioning in development and transcription (1, 13-15). Model organism studies have suggested that coding TRs are an important source of quantitative genetic variation that facilitates evolutionary adaptation (1, 16-19). For example, TR copy number variation in the yeast gene FLO1 correlates linearly with flocculation (20), a phenotype that is important for stress survival (17). As polyQ tracts often mediate protein interactions (2, 3, 21), polyQ-encoding TR copy number mutations could produce large and possibly adaptive phenotypic shifts. 
	To determine the phenotypic impact of naturally occurring polyQ variation (18, 22, 23) in a genetically tractable model, we focused on the gene ELF3, which encodes a polyQ tract that is highly variable across divergent Arabidopsis thaliana strains (accessions) (19, 24). ELF3 is a core component of the circadian clock and a potent repressor of flowering, and is considered a ``hub protein'' for its many interactions with various proteins (24-31). Consequently, elf3 loss-of-function mutants show pleiotropic phenotypes: they flower early, show poor circadian function, and grow long embryonic stems (hypocotyls) in light (25-27, 29, 30, 32). Single nucleotide polymorphisms (SNPs) in ELF3 affect shade avoidance, a fitness-relevant plant trait (24, 33). ELF3 polyQ variation has been suggested to correlate with two parameters of the circadian clock, period and phase (19). The ELF3 polyQ tract may mediate ELF3 membership in protein complexes, though thus far no ELF3-binding protein is known to bind it (26, 28-30). We discovered that altering polyQ tract length has dramatic effects on ELF3-dependent phenotypes and that these effects are dependent on genetic background.

\section{Methods}

\subsection{Plant Materials and Growth Conditions.}
The 181 Arabidopsis thaliana accessions are as previously described (1). The loss-of- function EARLY FLOWERING 3 (elf3) mutants are: (i) \emph{elf3-4}, containing a CCR2::LUC transgene (ecotype Ws) (2, 3) and 2) \emph{elf3-200}, the GABI750E02 T-DNA insertion mutant (ecotype Col-0) (4). For hypocotyl experiments, seeds were sterilized with Ethanol and plated onto 1x Murashige and Skoog (MS) basal salt medium supplemented with 1x MS vitamins, 1\% sucrose, 0.05\% Mes (wt/vol), and 0.24\% (wt/vol) phytagel. After stratification in the dark at 4$\degree$ C for 3 d, plates were transferred to an incubator (Conviron) that was set to either short day (SD) (8L:16D at 20$\degree$ C) or long day (LD) (16L:8D at 22$\degree$ C : 20$\degree$ C), with light supplied at $\mathrm{100 \mu mol * m^2 * s^{-1}}$ by cool-white fluorescent bulbs. For growth on soil, seeds were stratified at 4$\degree$ C for 3 d, and then grown in Sunshine \#4 soil under cool-white fluorescent light at either LD or SD at 20 $\degree$C. Seedlings used for RNA extractions were grown on soil under LD conditions and harvested on day 10. Samples for ELF3 expression measurements were collected at Zeitgeber time (ZT) 20. Samples for Phytochrome- interacting Factor 5 (PIF5) expression measurements were collected a ZT 8. Samples for and Pseudoresponse regulator 9 (PRR9) expression measurements were collected at ZT 0, 5, and 8.

\subsection{Generation of ELF3 Transgenic Plants.}
To generate A. thaliana transgenics carrying different ELF3 tandem repeat (TR) alleles, the cDNA clone RAFL09-28-E05 (RIKEN BRC) (5, 6), containing the ELF3 coding region and 3' UTR (Col-0 accession) was used. This cDNA clone lacks the small 5' intron. Two re- striction sites, Nar1 and Nco1, were inserted into the ELF3 coding sequence using the QuikChange Site-Directed Muta- genesis kit (Stratagene) (primer information in Table A4). The polyglutamine (polyQ)-encoding region was amplified from ac- cessions containing selected TR copy number alleles (primer information in Table A4, TR allele information in Table A1). These PCR products were digested with Nar1/Nco1 and ligated into the previously mutagenized ELF3 coding region. An artifi- cial allele lacking the TR was generated by site-directed muta- genesis (primer information in Table A4). Mutated plasmids and all ligation products were sequenced to ensure accuracy. The ELF3 alleles were cloned into pENTR1A (Invitrogen). A 2-kbp NotI fragment containing the ELF3 promoter was inserted up- stream of each ELF3 coding sequence. The fragments containing the ELF3 promoter, ELF3 coding sequence, and the ELF3 3' UTR were recombined using Gateway LR Clonase II (In- vitrogen) into a modified pB7WG2 (7), which lacks the CaMV- 35S promoter. The region encoding the polyQ tract of each construct was sequenced to ensure accurate TR copy number. The plasmids were used to transform Agrobacterium tumefaciens GV3101. Subsequently, Arabidopsis elf3 mutants were trans- formed by the flower dip method (8). Transformants were se- lected on Basta (Liberty herbicide; Bayer Crop Science) and propagated for three to four generations. The accuracy of the transgenes was confirmed by PCR (primer information in Table A4). All Ws phenotypic assays were performed in homozygous transgenic plants with expression levels between 0.8- and 4.5- times the respective ELF3 wild-type (Figure A1C); for Col lines, transgene expression levels were between 0.3- and 4.3-times the respective ELF3 wild-type (Figure A1D). Analyzed plant lines are in Tables A2-A4.

\subsection{RNA Extractions and Real-Time PCR.}
Total RNA was extracted from 30-mg frozen tissue using the SV Total RNA Isolation System (Promega). Subsequently, 2 $\mathrm{\mu g}$ of RNA were subjected to DNase treatment using Ambion Turbo DNA-free Kit (Applied Biosystems). RNA integrity and purity were checked with an Agilent Bioanalyzer using the RNA 6000 Nano Kit (Agilent Technologies). For cDNA synthesis, 200 ng of DNase-treated RNA was reverse-transcribed using the Transcriptor First Strand cDNA Synthesis Kit (Roche) and oligo dT primers. Transcript abundance was determined by real-time quantitative PCR using the LightCycler 480 system (Roche), with LightCycler 480 SYBR Green I Master (Roche) and the following PCR conditions: 5 min at 95 $\degree$C, followed by 35 cycles of 15 s at 95 $\degree$C, 20 s at 55 $\degree$C, and 20 s at 72 $\degree$C. To ensure that PCR products were unique, a melting- curve analysis was performed after the amplification. UBC21 expression (At5g25760) was used as a reference. All quantitative RT- PCR primers were designed with the LightCycler Probe Design Software (Roche). Sequences for real-time PCR primers are shown in Table A4. Relative quantification was determined with the $\Delta\Delta \mathrm{C_{T}}$ Method (9). Error was calculated as previously described (10).

\subsection{Thermal Asymmetric Interlaced PCR.} High-efficiency thermal asymmetric interlaced (TAIL)-PCR was performed as previously described (11) to obtain the flanking sequence of the construct in- tegration site (left border). Briefly, a preamplification step was performed with primers LAD and LB-0a (Table A4), followed by primary TAIL-PCR with primers AC1 (11) and LB-1a (Table A4), and 1 $\mathrm{\mu L}$ of a 1/40 dilution of the preamplification product as a template. A secondary TAIL-PCR with primers AC2 (11) and LB-2a (Table A3) was performed with 1 $\mathrm{\mu L}$ of a 1/10 dilution of the primary TAIL-PCR product. Next, 3-kbp products were extracted from agarose gels and subsequently Sanger-sequenced. Only sequences containing the T-DNA left border were considered.

\subsection{Developmental Phenotype Assays.} For measurements of hypocotyl length, seedlings were grown on vertical plates for 15 d in a pseudorandomized design under either SD or LD conditions (12). Hypocotyl length was measured with ImageJ on digital images (http://rsbweb.nih.gov/ij/). For measurement of flowering time, seeds were planted in sheet pots (36 pots per tray) in a random- ized design and trays were rotated daily. Flowering time was re- corded as the day when the inflorescence reached 1 cm in height. Rosette leaf number was determined on the same day. Petiole- length/leaf-length (PL/LL) ratio for leaf four was determined on day 45. Least-square means for all traits were derived from a linear regression analysis for each trait separately. ELF3 TR copy number was modeled as a nominal variable and independent transgenic lines carrying the same ELF3 TR allele were analyzed together. We tested for significant phenotypic differences con- ferred by the different ELF3 TR alleles by using Tukey-HSD tests with $\mathrm{\alpha = 0.05}$ that accommodate nonnormal data.

\subsection{Luciferase Imaging and Period Analysis.} Luciferase assays were performed with lines containing the CCR2::LUC reporter. Seeds were surface sterilized with a 70\% (vol/vol) ethanol wash followed by a second wash with 33\% (vol/vol) Klorix with Triton X-100, and then rinsed twice with sterile water. Seeds were plated on MS3 medium [pH 5.7, 3\% (wt/vol) sucrose, 1.5\% (wt/vol) PhytoAgar, and 15 $\mu$g/mL hygromycin B]. They were subsequently stratified for 4 d at 4 $\degree$C in the dark and entrained under 12-h light:12-h dark cycles under white fluorescent light ($\mathrm{\sim10 \mu mol * m^{-2} * s^{1}}$) at 22 $\degree$C. On the sixth day, a minimum of 24 seedlings per line was transferred to 96-well TopCount (Perkin-Elmer) plates containing 200 mg MS3 agar. We added 5 mM Luciferin in 0.01\% Triton X-100 and entrained seedlings for another cycle before luminescence was detected using a Packard/Perkin-Elmer Top- Count Scintillation and Luminescence Counter. Red and blue light-emitting diodes ( $\mathrm{100 \mu mol * m^{-2} * s^{-1}}$) were used as a light source during this analysis. During the first 24 h of luminescence detection, plants were grown in 12-h light:12-h dark and then released under constant light conditions to measure the free- running period. Each individual was measured approximately every 30 min for a minimum of 5 d. Luminescence levels were quantified and analyzed as previously described (2, 3) using the macro suites TopTempII and Biological Rhythms Analysis Software System (13). Period length and relative amplitude error (RAE) were estimated using fast Fourier transform nonlinear least squares (14). Period values scored with RAE values below 0.4 were considered robustly rhythmic (15).
\subsection{Principal Component Analysis.} We clustered our phenotypic data using principal component analysis (PCA) to find patterns corresponding to genotypes. We excluded the phenotype of rosette leaf number in SD, for which data were missing for several alleles. The phenotypes included in the analysis are: Days to flowering in SD and LD conditions, hypocotyl length under SD and LD PL/LL for the fourth leaf in SD, and rosette leaf number in LD. For analyses involving Col lines, the SD PL/LL ratio phenotype was omitted because of lack of data, and PCA was thus based on the remaining five phenotypic variables. For each phenotype in each genetic background (either Ws or Col-0), we calculated the mean phenotype of the independently generated lines for each \emph{ELF3-TR} allele, giving us a 28 x 6 matrix of mean phenotypes for the 28 genotypes for each of six phenotypic variables. Within each background, we ranked the genotypes for each phenotype. Ranks were transformed into a standard normal distribution based on their percentile, using the R function qnorm. Using this transformed dataset, we performed PCA using the R function $prcomp$ (R Foundation for Statistical Computing, http://www.r-project.org/, 2011). We performed PCA for each background separately, and then for both backgrounds together. Rank-normalization was necessary to compare (i) phenotypes measured on different scales and (ii) Ws- and Col-derived plants, between which backgrounds absolute phenotypic differences exist. Consequently, the rank-normalization increases stability of our estimates, as our dataset is relatively small and PCA as assumptions of normality were not met by our raw dataset. PCA on raw values scaled to a standard normal distribution gave similar results. Biplots were generated with the R $biplot$ function on $prcomp$ function output.

\section{\emph{\emph{ELF3-TR}} variation affects ELF3-dependent phenotypes. }
Among 181 natural A. thaliana accessions, the \emph{ELF3-TR} encoded between 7 and 29Q (Table A1, Figure A1a). For comparison, polyQ expansions over 20Q are associated with disease in the context of the SCA6 gene, though most other disease-associated polyQ expansions are longer (2, 19, 24). The most frequent \emph{ELF3-TR} encoded 16Q, whereas the shortest TR (7Q) was found in the reference strain Col-0. We set out to test whether naturally occurring \emph{ELF3-TR} alleles affect ELF3-dependent phenotypes and whether they do so in a linear manner as suggested by association studies (19) and found for coding TR variation in other genes (16, 20). We generated expression-matched transgenic lines for most natural \emph{ELF3-TR} alleles in the loss-of-function \emph{elf3-4} mutant (Ws background, Table A2, Figure A1c) (32) and measured their flowering time and circadian clock-related phenotypes (Figures 1, SAa-g). \emph{ELF3-TR} variation significantly affected ELF3-dependent phenotypes, but there was no evidence of a linear relationship. The different \emph{ELF3-TR} alleles resulted in phenotypes ranging from nearly full complementation of \emph{elf3-4} to nearly phenocopying the loss-of-function mutant. We used principal components analysis to describe the complex effects of \emph{ELF3-TR} alleles on all tested ELF3-dependent phenotypes (PCA, Figures 1a, A2h-j). Principal component 1 (PC1) corresponds to general functionality of ELF3 in all measured phenotypes, with wild-type Ws and mutant \emph{elf3-4} defining the extremes. Separation along PC1 is driven by the tendency of plants with functional ELF3 to show short hypocotyls, late flowering, increased rosette leaf number, and short petioles (Figures 1b-d, A2). The endogenous ELF3-16Q allele complemented both the early-flowering and long-hypocotyl phenotypes of \emph{elf3-4} (Figures 1b-d, A2). In contrast, both the long ELF3-23Q and the short ELF3-7Q allele (endogenous TR alleles in Br-0/Bur-0 and Col-0, respectively) behaved similarly to the \emph{elf3-4} loss-of-function allele (Figures 1b-d, A2), although they are functional in their native backgrounds. Neither Col-0 nor Br-0 and Bur-0 show the phenotypic characteristics of elf3-mutants (early flowering (34), long hypocotyls (35) and long petioles (36)), suggesting that \emph{ELF3-TR} alleles may interact with background-specific modifiers. ELF3-0Q, an artificial ELF3 allele lacking the TR, partially complemented \emph{elf3-4} (Figures 1a, S2). Hence, the polyQ-encoding TR is not necessary for all ELF3 function, but changes in TR copy number are sufficient to enhance or ablate ELF3 function. 

\begin{figure}
\includegraphics[width=4in]{undurraga2012_figures/MainText/Figure1Q.png}
\caption[ELF3-TR variation has nonlinear phenotypic effects.]{}
\end{figure}

\addtocounter{figure}{-1}
\captionsetup[figure]{list=no}
\captionof{figure}{ELF3-TR variation has nonlinear phenotypic effects. (A) PCA of developmental traits of all ELF3-TR copy number variants. A. thaliana images illustrate ELF3-TR effects on the traits days to flower and hypocotyl length under SD and LD, petiole-length/leaf-length ratio (PL/LL) under SD only, and rosette leaf number under LD only. The contributions of specific phenotypes to PCs are in Figure A2J. Representative TR copy number alleles are shown from an analysis including all alleles (for all alleles see Figure A2 H and I). (B) Days to flower under SD conditions for selected lines. n = 6 plants per transgenic line. (C) Hypocotyl length at 15 d under SD for selected lines. n = 20-30 seedlings per transgenic line. (D) PL/LL of the fourth leaf for selected lines. Data are from the same plants as in B. (E) Plants carrying the ELF3-20Q allele (Center) are specific hypomorphs under SD with the elongated petioles of the elf3-4 mutant (vector control, VC, Right) and a wild-type flowering phenotype (Ws, Left). ELF3-TR alleles are indicated with the number of Qs encoded, Ws is wild-type, VC is the elf3-4 vector control. Error bars are SEMs. Genotypes labeled with different letters differed significantly in phenotype by Tukey's HSD test. For all Ws-background phenotype data, see Figure A2 AG. Data are from multiple independently generated expression-matched (Figure A1C) T3 and T4 lines for each TR copy number allele (Tables A2, A3). These experiments were repeated at least once with similar results. The tested ELF3-20Q lines contained unique insertions that did not affect genes with known function.}
\label{fig:chap1fig1}
\captionsetup[figure]{list=yes}

PC2 separated ELF3-20Q and ELF3-11Q, which behaved as hypomorphs in certain phenotypes but not others (Figure 1a). For example, ELF3-20Q plants had significantly longer hypocotyls than wild-type and its petioles phenocopied the extremely long petioles of the \emph{elf3-4} mutant (Figure 1c-e), but they did not differ from wild-type in flowering time (days to flower, Figure 1b). The existence of both general and specific hypomorphs suggests that polyQ variation affects the multiple ELF3 functions separately.
As part of a protein complex, ELF3 affects expression of Phytochrome-interacting Factor 5 (PIF5) and Pseudo-response regulator 9 (PRR9) (28, 37, 38). PIF5 and PRR9 expression were strongly affected by ELF3 polyQ variation (Figure A3). ELF3-16Q phenocopied wild-type PRR9 and PIF5 expression, and the hypomorphic ELF3-23Q phenocopied \emph{elf3-4} (28, 37, 38), mirroring their developmental phenotypes. Consistent with their divergence along PC2 (Figure 1a), ELF3-11Q and ELF3-20Q differed in their effect on PRR9 expression, but not on PIF5 expression (Figure A3a, b), demonstrating that ELF3 polyQ variation differentially affects the regulation of downstream genes.

\begin{figure}
\includegraphics[width=6in]{undurraga2012_figures/MainText/Figure2Q.png}
\caption[ELF3-TR variation modulates the precision of the circadian clock.]{ELF3-TR variation modulates the precision of the circadian clock. (A) RAE of CCR2::LUC circadian oscillation in seedlings with indicated ELF3-TR alleles. Bars represent 99\% confidence intervals. (B) Mean values of circadian period and RAE (points) were measured in seedlings with indicated ELF3-TR alleles. Dotted ellipses represent SEMs for both period and RAE. Note that plants with high RAE have extremely unreliable estimates of circadian period. Bioluminescence rhythms from the CCR2::LUC reporter in ELF3-TR transgenic lines were used to measure circadian parameters under LL after 5 d of entrainment in 12-h light:12-h dark cycles. n$\geq$100 seedlings for all genotypes. Aggregate data from four independent experiments are shown. See Fig. A4 for RAE and period data for all alleles.}
\label{fig:chap1fig2}
\end{figure}

\section{\emph{ELF3-TR} variation modulates the precision of the circadian clock}
To directly assess the role of ELF3 polyQ variation in the circadian clock, we used the CCR2::LUC reporter system (25, 39). We observed little difference in circadian period among wild-type Ws and tested \emph{\emph{ELF3-TR}} alleles (Figure A4a), contradicting a previously observed association of TR copy number with period in natural accessions (19). However, we found that the relative amplitude error (RAE) of oscillation varies substantially across \emph{\emph{ELF3-TR}} genotypes (Figures 2a, S4b). RAE measures the precision of a circadian period (40): high RAE values (> 0.4) indicate poor oscillation and clock dysfunction (41). The endogenous Ws ELF3-16Q nearly complemented the \emph{\emph{elf3-4}} RAE defect, whereas the TR alleles ELF3-7Q, ELF3-10Q, and ELF3-23Q showed higher RAE, approaching arrhythmic \emph{elf3-4} levels (Figure 2a, b), consistent with their hypomorphic performance in other ELF3 traits (close to \emph{elf3-4} in PC1, Figure 1a). Together, these results suggest that ELF3 polyQ tract length is a critical determinant of circadian clock precision, but not period length, in A. thaliana. 

\section{\emph{\emph{ELF3-TR}} variation interacts with genetic background.}
To test our hypothesis that \emph{ELF3-TR} variation interacts with genetic background, we regenerated all \emph{ELF3-TR} transgenic lines in the \emph{elf3-200} loss-of-function mutant with matched transgene expression (Col background, Table A3, Figure A1d) (42). We used PCA to compare \emph{ELF3-TR} effects between Ws and Col backgrounds (Figures 3a, S5). 
The Col-specific ELF3-7Q allele complemented \emph{elf3-200} in some traits such as flowering time (in short days, SD) and hypocotyl length (in long days, LD), but not others (Figures 3a, b, S5, S6). This result may be due to the absence of the small 5' intron from the ELF3 construct used in this study. However, there was still a dramatic spread of phenotypes: all longer \emph{ELF3-TR} alleles (>20 Qs) nearly complemented \emph{elf3-200}, delaying flowering and shortening hypocotyls, whereas few of the shorter alleles did (Figures 3, S5, S6). Results were similar when the Col data were analyzed alone (Figure A6). Thus, in contrast to our results in the Ws background, \emph{ELF3-TR}s appeared to show a threshold effect for TR copy number in the Col background. We speculate that the intensive laboratory propagation of the Col-0 accession may have altered selection on the \emph{ELF3-TR}, resulting in an extremely short ``hypomorphic'' allele, whereas under natural conditions a longer TR might be more functional. 
Comparing TR allele effects between the two backgrounds revealed striking differences. For example, the ELF3-23Q allele was generally hypomorphic in the Ws background (\emph{elf3-4}), whereas it produced highly functional ELF3 in the Col background (\emph{elf3-200}, Figure 3). In turn, the ELF3-16Q allele produced highly functional ELF3 in the Ws background (\emph{elf3-4}), but was generally hypomorphic in the Col background (\emph{elf3-200}). The consistent performance of the artificial ELF3-0Q allele across backgrounds suggests that the background effect is TR-dependent (Figures 3a, S5). Collectively, our results support that \emph{ELF3-TR} alleles interact with background-specific modifiers. 

\begin{figure}
\includegraphics[width=5in]{undurraga2012_figures/MainText/Figure3Q.png}
\caption[The phenotypic effects of ELF3-TR variation are strongly background-dependent. ]{}
\end{figure}
\addtocounter{figure}{-1}
\captionsetup[figure]{list=no}
\captionof{figure}{The phenotypic effects of ELF3-TR variation are strongly background-dependent. (A) PCA of developmental traits of all ELF3-TR alleles in Ws and Col genetic backgrounds. Shared color indicates a given ELF3-TR allele in both genetic backgrounds. A. thaliana images are as in Fig. 1A. The contributions of phenotypes to principal components are similar to Fig. 1A, except that PC2 is inverted (no effect on interpretation, loadings in Fig. A5C). Representative TR copy number alleles are shown from an analysis including all alleles (for all alleles see Fig. A5; for Col-background specific PCA, see Fig. A6). (B) Days to flower under SD and hypocotyl length under LD differ for particular TR alleles between Ws (Upper) and Col (Lower) backgrounds. ELF3-TR alleles are indicated with the number of Qs encoded, Ws and Col-0 are wild-type, elf3-4 and elf3-200 are respective vector controls (VC). Error bars represent SEM. Genotypes labeled with different letters differed significantly in phenotype by Tukey-HSD test. For all Col-background phenotype data, see Fig. S6 AG. Data are from multiple independently generated expression-matched (Fig. A1C and D) T3 and T4 lines for each TR copy number allele (Tables A2, A3). These experiments were repeated at least once with similar results.}
\captionsetup[figure]{list=yes}


\section{Col \emph{ELF3} allele is not haploinsufficient in Col x Ws hybrids.}
To address whether Ws and Col-specific background effects are sufficient for altered hybrid phenotypes, we generated F1 populations between wild-type and elf3 null plants in the Ws and Col backgrounds and measured ELF3 function by assessing hypocotyl length. Ws x Col F1 hybrids resembled their wild-type parents (Figure 4, top). F1 hybrids containing both loss-of-function alleles had significantly longer hypocotyls than either parent. Both ELF3 alleles were haplosufficient in F1 crosses within their native backgrounds, as expected for recessive mutants . In stark contrast, we observe that ELF3-Col, but not ELF3-Ws, phenocopied the extreme hypocotyl length of the double loss-of-function mutant. Consistent with the results from our transgenic lines, our F1 hybrid data suggest that full ELF3 function depends on a permissive genetic background.

% add section showing follow-up fucked us
Unfortunately, propagation of these $\mathrm{F_{1}}$ hybrids to the $\mathrm{F_{2}}$ generation and subsequent Col x Ws crosses revealed that these data do not generalize to other crosses (Figure ~\ref{fig:chap1f1}, Figure A7), and probably represent a spontaneous mutation in the Col background leading to ELF3 inactivation. Repetition of the experiment with newly generated $\mathrm{F_{1}}$s led to inconsistent results, and propagation of various batches to the $\mathrm{F_{2}}$ generation (Figure A7) supported a mutation linked to the \emph{ELF3} locus in the Col parent underlying apparent haploinsufficiency. In the face of such equivocal evidence, we suggest that more intensive genetic or biochemical experiments will be necessary to determine the relevant background modifiers of \emph{ELF3-TR} variation. For such approaches, refer to Chapter 5.

\begin{figure}
\includegraphics[width=5in]{oldbad_vs_newboring_f1data.pdf}
\caption[Inconsistent haploinsufficiency of Col x Ws $\mathrm{F_{1}s.}$ ]{Top: ELF3-Col appears haploinsufficient in a hybrid Col x Ws genetic background. Bottom: Haploinsufficiency disappears in new seed batches. Hypocotyl length under SD was measured in seedlings from parental and F1 lines. elf3-ws is the elf3-4 loss-of-function mutant in the Ws background; elf3-col is the elf3-200 loss-of-function mutant in the Col background. Reciprocal crosses for each F1 showed similar results. n $\geq$ 15 for each genotype, except for Ws x elf3-ws in the top panel (n = 5). Each experiment was repeated with similar results (drawing from the same seed batch).}
\label{fig:chap1f1}
\end{figure}
%\addtocounter{figure}{-1}
%\captionsetup[figure]{list=no}
%\captionof{figure}{Top: ELF3-Col appears haploinsufficient in a hybrid Col ? Ws genetic background. Bottom: Haploinsufficiency disappears in new seed batches. Hypocotyl length under SD was measured in seedlings from parental and F1 lines. elf3-ws is the elf3-4 loss-of-function mutant in the Ws background; elf3-col is the elf3-200 loss-of-function mutant in the Col background. Reciprocal crosses for each F1 showed similar results. n $\geq$ 15 for each genotype, except for Ws x elf3-ws in the top panel (n = 5). Each experiment was repeated with similar results (drawing from the same seed batch).}
%\captionsetup[figure]{list=yes}


\section{Discussion}
Our results demonstrate that natural ELF3 polyQ variation that is not associated with disease has dramatic phenotypic consequences, and that these consequences depend on genetic background. For ELF3, in at least the Ws background, the relationship between TR copy number and phenotype does not follow a linear or threshold pattern as observed for other coding TR and polyQ disorders (1, 2, 16, 17, 20). Studies correlating TR variation with phenotype often apply linear models, treating TR copy number as a quantitative variable (19, 22, 23). Our data show that this approach is not appropriate for all TRs. 
Instead, \emph{ELF3-TR} alleles seem ``matched'' to specific genetic backgrounds, in which they are functional, whereas they are incompatible with other backgrounds. Variable TRs, and the \emph{ELF3-TR} in particular, have been previously suggested as agents of adaptation to new external environments (1, 16, 17, 20, 24, 44). Our results, demonstrating strong background effects, suggest that polyQ-encoding TRs are also agents of coadaptation within genomes.
We speculate that the observed background effects arise from background-specific polymorphisms in genes encoding physically interacting proteins (26, 28-30). TRs have a far higher mutation rate than non-repeated regions ($\mathrm{10^{-4}}$ per site per generation for TR vs. $\mathrm{10^{-8}}$ for SNPs) (45, 46) and, as we show, their expansion or contraction can have dramatic phenotypic impact. ELF3's partner proteins may have acquired compensatory mutations to accommodate new \emph{ELF3-TR} variants and vice versa. Alternative explanations for the background effects are compensatory mutations in ELF3 (intragenic suppressors), or ELF3 interactions that are unique to a given background. Intragenic variation and protein modification can play an important role in polyQ-mediated phenotypes (47, 48). At least for the ELF3-Col allele, however, our F1 data are not consistent with intragenic suppressors. 
Consistent with polyQ-mediated background effects, in at least one case, a modifier mutation has been shown to delay onset of Huntington's disease (11). Hypothetically, population genetic approaches could identify incompatible alleles that may contribute to variable disease onset in patients with polyQ expansions and to ELF3-dependent background effects in A. thaliana. However, the great diversity of TR alleles compared to SNP alleles and the small number of individuals carrying specific TR alleles render a population genetics approach infeasible. Extensive genetic mapping or other experimental approaches will be needed to identify the determinants of \emph{ELF3-TR} dependent background effects. 
As TRs are rapidly evolving, we speculate that polyQ-mediated incompatibilities and the resulting fitness loss in hybrids and their offspring may contribute to disruption of gene flow between closely related populations. This speciation mechanism would be of particular importance for organisms with many polyQ-encoding TRs, thousands of offspring, and an inbreeding life style. Even in humans, however, about 1\% of proteins contain polyQ tracts (13, 14, 45). Our results identify TR copy number variation, and in particular polyQ variation, as a phenotypically important class of genetic variation that warrants genome-wide assessment in model organisms, crops, and humans alike. 


% ========== MAX put in a chapter here
\chapter{The conserved \emph{PFT1} tandem repeat is crucial for proper flowering in \emph{Arabidopsis thaliana}}

A version of this chapter was published under the following reference: 

Pauline Rival, Maximilian O. Press, Jacob Bale, Tanya Grancharova, Soledad F. Undurraga, and Christine Queitsch. The Conserved PFT1 Tandem Repeat is Crucial for Proper Flowering in \emph{Arabidopsis thaliana}. \emph{Genetics}, 198(2):747-754, August 2014.

Pauline Rival, Jacob Bale, Tanya Grancharova, and Soledad Undurraga contributed transgenic lines and experiments.

Supporting figures and tables can be found in Appendix B.

\section{Summary}
It is widely appreciated that short tandem repeat (STR) variation underlies substantial phenotypic variation in organisms. Some propose that the high mutation rates of STRs in functional genomic regions facilitate evolutionary adaptation. Despite their high mutation rate, some STRs show little to no variation in populations. One such STR occurs in the Arabidopsis thaliana gene PFT1 (MED25), where it encodes an interrupted polyglutamine tract. Though the PFT1 STR is large (~270 bp), and thus expected to be extremely variable, it shows only minuscule variation across A. thaliana strains. We hypothesized that the PFT1 STR is under selective constraint, due to previously undescribed roles in PFT1 function. We investigated this hypothesis using plants expressing transgenic PFT1 constructs with either an endogenous STR or with synthetic STRs of varying length. Transgenic plants carrying the endogenous PFT1 STR generally performed best in complementing a pft1 null mutant across adult PFT1-dependent traits. In stark contrast, transgenic plants carrying a PFT1 transgene lacking the STR phenocopied a pft1 loss-of-function mutant for flowering time phenotypes, and were generally hypomorphic for other traits, establishing the functional importance of this domain. Transgenic plants carrying various synthetic constructs occupied the phenotypic space between wild-type and pft1-loss-of-function mutants. By varying PFT1 STR length, we discovered that PFT1 can act as either an activator or repressor of flowering in a photoperiod-dependent manner. We conclude that the PFT1 STR is constrained to its approximate wild-type length by its various functional requirements. Our study implies that there is strong selection on STRs not only to generate allelic diversity, but also to maintain certain lengths pursuant to optimal molecular function.

\section{Introduction}
Short tandem repeats (STRs, microsatellites) are ubiquitous and unstable genomic elements that have extremely high mutation rates (Subramanian et al. 2003; Legendre et al. 2007; Eckert and Hile 2009), leading to STR unit number variation within populations. STR variation in coding and regulatory regions can have significant phenotypic consequences (Gemayel et al. 2010). For example, several devastating human diseases, including Huntingtons disease and spinocerebellar ataxias, are caused by expanded STR alleles (Hannan 2010). However, STR variation can also confer beneficial phenotypic variation and may facilitate adaptation to new environments (Fondon et al. 2008; Gemayel et al. 2010). For example, in Saccharomyces cerevisiae natural polyQ variation in the FLO1 protein underlies variation in flocculation, which is important for stress resistance and biofilm formation in yeasts (Verstrepen et al. 2005). Natural STR variants of the Arabidopsis thaliana gene ELF3, which encode variable polyQ tracts, can phenocopy elf3 loss-of-function phenotypes in a common reference background (Undurraga et al. 2012). Moreover, the phenotypic effects of ELF3 STR variants differed dramatically between the divergent backgrounds Col and Ws, consistent with the existence of background-specific modifiers. Genetic incompatibilities involving variation in several other STRs have been described in plants, flies, and fish (Peixoto et al. 1998; Scarpino et al. 2013; Rosas et al. 2014). Taken together, these observations argue that STR variation underlies substantial phenotypic variation, and may also underlie some genetic incompatibilities.
The A. thaliana gene PHYTOCHROME AND FLOWERING TIME 1 (PFT1, MEDIATOR 25, MED25) contains an STR of unknown function. In contrast to the comparatively short and pure ELF3 STR, the PFT1 STR encodes a long (~90 amino acids in PFT1, vs. 7-29 for ELF3), periodically interrupted polyQ tract. The far greater length of the PFT1 STR leads to the prediction that its allelic variation should be greater than that of the highly variable ELF3 STR (Legendre et al. 2007, http://www.igs.cnrs-mrs.fr/TandemRepeat/Plant/index.php). However, in a set of diverse A. thaliana strains, PFT1 STR variation was negligible compared to that of the ELF3 STR (Supp. Table 1). Also, unlike ELF3, the PFT1 polyQ is conserved in plants as distant as rice, though its purity decreases with increasing evolutionary distance from A. thaliana. A glutamine-rich C-terminus is conserved even in metazoan MED25 (File S1). Recent studies of coding STRs suggested that there may be different classes of STR. Specifically, conserved tandem repeats appear in genes with substantially different functions from genes containing non-conserved tandem repeats (Schaper et al. 2014). Consequently, PFT1/MED25 polyQ conservation may functionally differentiate the PFT1 STR from the ELF3 STR.
PFT1 encodes a subunit of Mediator, a conserved multi-subunit complex that acts as a molecular bridge between enhancer-bound transcriptional regulators and RNA polymerase II to initiate transcription (Bckstrm et al. 2007; Conaway and Conaway 2011). PFT1/MED25 is shared across multicellular organisms but absent in yeast. In A. thaliana, the PFT1 protein binds to at least 19 different transcription factors (Elfving et al. 2011; Ou et al. 2011; evik et al. 2012; Chen et al. 2012) and has known roles in regulating a diverse set of processes such as organ size determination (Xu and Li 2011), ROS signaling in roots (Sundaravelpandian et al. 2013), biotic and abiotic stress (Elfving et al. 2011; Kidd et al. 2009; Chen et al. 2012), phyB-mediated-light signaling, shade avoidance and flowering (Cerdn and Chory 2003; Wollenberg et al. 2008; Iigo, Alvarez, et al. 2012; Klose et al. 2012). 
PFT1 was initially identified as a nuclear protein that negatively regulates the phyB pathway to promote flowering in response to specific light conditions (Cerdn and Chory 2003; Wollenberg et al. 2008). Recently, Iigo and colleagues (2012) showed that PFT1 activates CONSTANS (CO) transcription and FLOWERING LOCUS T (FT) transcription in a CO-independent manner. Specifically, proteasome-dependent degradation of PFT1 is required to activate FT transcription and to promote flowering (Iigo, Giraldez, et al. 2012). The wide range of PFT1-dependent phenotypes is unsurprising given its function in transcription initiation, yet it remains poorly understood how PFT1 integrates these many signaling pathways.
Given the conservation of the PFT1 polyQ tract and the known propensity of polyQ tracts for protein-protein and protein-DNA interactions (Escher et al. 2000; Schaefer et al. 2012), we hypothesized that this polyQ tract plays a role in the integration of multiple signaling pathways and is hence functionally constrained in length. We tested this hypothesis by generating transgenic lines expressing PFT1 with STRs of variable length and evaluating these lines for several PFT1-dependent developmental phenotypes. We show that the PFT1 STR is crucial for PFT1 function, and that PFT1-dependent phenotypes vary significantly with the length of the PFT1 STR. Specifically, the endogenous STR allele performed best for complementing the flowering and shade avoidance defects of the \emph{pft1-2} null mutant, though not for early seedling phenotypes. Our data indicate that most assayed PFT1-dependent phenotypes require a permissive PFT1 STR length. Taken together, our results suggest that the natural PFT1 STR length is constrained by the requirement of integrating multiple signaling pathways to determine diverse adult phenotypes. 

\section{Methods}
\subsection{Cloning}
A 1000 bp region directly upstream of the PFT1 coding region was amplified and cloned into the pBGW gateway vector (Karimi et al. 2002) to create the entry vector pBGW-PFT1p. A full-length PFT1 cDNA clone, BX816858, was obtained from the French Plant Genomic Resources Center (INRA, CNRGV), and used as the starting material for all our constructs. The PFT1 gene was cloned into the pENTR4 gateway vector (Invitrogen) and the repeat region was modified by site-directed mutagenesis with QuikChange (Agilent Technologies), followed by restriction digestions and ligations. The modified PFT1 alleles were finally transferred to the pBGW-PFT1p vector via recombination using LR clonase (Invitrogen) to yield the final expression vectors. Seven constructs expressing various polyQ lengths (Table B2), plus an empty vector control, were used to transform homozygous \emph{pft1-2} mutants by the floral dip method (Clough and Bent 1998). Putative transgenics were selected for herbicide resistance with Basta (Liberty herbicide; Bayer Crop Science) and the presence of the transgene was confirmed by PCR analysis. Homozygous T3 and T4 plants with relative PFT1 expression levels between 0.5 and 4 times the expression of Col-0 were utilized for all experiments described. A minimum of two independent lines per construct was used for all experiments.  

\subsection{Expression analysis}
All protocols were performed according to manufactures recommendations unless otherwise noted. Total RNA was extracted from 30mg of 10-days-old seedlings with the Promega SV Total RNA Isolation System (Promega). 2 ?g of total RNA were subjected to an exhaustive DNaseI treatment using the Ambion Turbo DNA-free Kit (Life Technologies). cDNA was synthesized from 100-300 ng of DNase-treated RNA samples with the Roche Transcriptor First Strand cDNA Synthesis Kit (Roche). Quantitative Real-Time PCR was performed in a LightCyler\textregistered 480 system (Roche) using the 480 DNA SYBR Green I Master kit. Three technical replicates were done for each sample. RT-PCR was performed under the following conditions: 5 min at 95 $\degree$C, followed by 35 cycles of 15 s at 95 $\degree$C, 20 s at 55 $\degree$C, and 20s at 72 $\degree$C. After amplification, a melting-curve analysis was performed. Expression of UBC21 (At5g25760) was measured as a reference in each sample, and used to calculate relative PFT1 expression. All expression values were normalized relative to WT expression, which was always set to 1.0. To measure splice forms, the protocol was the same but reactions were carried out in a standard thermal cycler and visualized on 2\% agarose stained with ethidium bromide. For primers, see Table B4. 

\subsection{Plant Materials and Growth Conditions}
Homozygous plants for the T-DNA insertional mutant SALK 129555, pft1-2, were isolated by PCR analysis from an F2 population obtained from the Arabidopsis Stock Center (ABRC) (Alonso et al. 2003). Plants were genotyped with the T-DNA specific primer LBb1 (\url{http://signal.salk.edu/tdna_FAQs.html}) and gene-specific primers (Table B4).
	Seeds were stratified at 4oC for 3 days prior to shifting to the designated growth conditions, with the shift day considered day 0. For flowering time experiments, plants were seeded using a randomized design with 15-20 replicates per line in 4x9 pot trays. Trays were rotated 180o and one position clockwise everyday in order to further reduce any possible position effect. Plants for LD were grown in 16 hours of light and 8 hours of darkness per 24 hour period. Bolting was called once the stem reached 1 cm in height. 
Full strength MS media containing MES, vitamins, 1\% sucrose, and 0.24\% phytagar was used for hypocotyl experiments. For germination experiments, half-strength MS media was used, supplemented with 1\% sucrose, 0.5 g/L MES, and 2.4 g/L phytagel containing 200 mM NaCl or H2O mock treatment with the pH adjusted to 5.7. All media was sterilized by autoclaving with 30 minutes of sterilization time. Seeds for tissue culture were surface sterilized with ethanol treatment prior to plating and left at 4oC for 3 days prior to shifting to the designated growth conditions. Plants for hypocotyl experiments were grown with 16 hours at 22 $\degree$C and 8 hours at 20oC in continuous darkness following an initial 2 hour exposure to light in order to induce germination. Germination experiments were scored on day 4 under LD at 20-22 $\degree$C. ImageJ software was utilized to make all hypocotyl and root length measurements. Raw phenotypic data are included as File S3.

\subsection{Statistical analysis}
All statistical analyses and plots were performed in R version 2.15.1 with ? = 0.05 (R Development Core Team 2012). Phenotypic data were analyzed using the analysis of variance (ANOVA), followed by Tukeys HSD tests for the differences of groups within the ANOVA. Tukeys HSD is a standard post-hoc test for multiple comparisons of the means of groups with homogeneous variance that corrects for the number of comparisons performed. Principal component analysis was performed using the prcomp() function after scaling each phenotypic variable to mean=0 and variance=1 across lines (phenotypes are not measured on the same quantitative scale; for example, SD flowering time ranges from 80 to 140 days, whereas LD rosette leaves ranges ~5-15 leaves).

\subsection{Sequence Analysis}
Length of \emph{ELF3} and \emph{PFT1} STRs were determined by Sanger (dideoxy) sequencing. Raw sequencing data are available on the \emph{Genetics} website (http://www.genetics.org/content/198/2/747.long). PFT1 and MED25 reference amino acid sequences were obtained from KEGG (Ogata et al. 1999) and aligned with Clustal Omega v1.0.3 with default options (Sievers et al. 2011).

\section{Natural variation of \emph{PFT1} STR}
We used Sanger sequencing to evaluate our expectation of high PFT1 STR variation across A. thaliana strains. However, we observed only three alleles of very similar size (encoding 88, 89 and 90 amino acids, Table B1), in contrast to six different alleles of the much shorter ELF3 STR among these strains, some of which are three times the length of the reference allele \cite{Undurraga2012}. These data implied that the PFT1 and ELF3 STRs respond to different selective pressures. In coding STRs, high variation has been associated with positive selection \cite{Laidlaw2007}, though some basal level of neutral variation is expected due to the high mutation rate of STRs. We hypothesized that the PFT1 STR was constrained to this particular length by PFT1s functional requirements. 
To test this hypothesis, we generated transgenic A. thaliana carrying PFT1 transgenes with various STR lengths in an isogenic \emph{pft1-2} mutant background. These transgenics included an empty vector control (VC), 0R, 0.34R, 0.5R, .75R, 1R (endogenous \emph{PFT1} STR allele), 1.27R, and 1.5R constructs. All STRs are given as their approximate proportion of WT STR length  for instance, the 1R transgenic line contains the WT STR allele in the \emph{pft1-2} background (Table B2). We used expression analysis to select transgenic lines with similar PFT1 expression levels (Table B3). 

\section{The PFT1 STR length is essential for wild-type flowering and shade avoidance}
We first evaluated the functionality of the different transgenic lines in flowering phenotypes. Removing the STR entirely substantially delayed flowering under long days (LD, phenotypes days to flower, rosette leaf number at flowering; Figure 1A). In LD, any STR allele other than 0R was able to rescue the \emph{pft1-2} late-flowering phenotype. Indeed, one allele (1.5R) showed earlier flowering than WT (Figure 1B, 1C), whereas other alleles provided a complete or nearly complete rescue of the \emph{pft1-2} mutant (Figure 1D). 
In short days (SD), we observed an unexpected reversal in rosette leaf phenotypes (compare SD and LD rosette leaves, Figures 1B, 1D). Rather than flowering late (adding more leaves) as in LD, the loss-of-function \emph{pft1-2} mutant appeared to flower early (fewer leaves at onset of flowering). Only the endogenous STR (1R) fully rescued this unexpected phenotype (Figure 1D). We observed the same mean trend for days to flowering in SD, although differences were not statistically significant, even for \emph{pft1-2} (Figure 1D). This discrepancy may be due to insufficient power, or to a physiological decoupling of number of rosette leaves at flowering and days to flowering phenotypes in \emph{pft1-2} under SD conditions. Regardless, our results indicate that pft1-2s late-flowering phenotype is specific to LD conditions. Our observation of this reversal in flowering time-related phenotypes appears to contradict previous data (Cerdn and Chory 2003). However, a closer examination of this data reveals that the previously reported rosette leaf numbers in SD for the \emph{pft1-2} mutant show a similar trend. PFT1 STR length shows an approximately linear positive relationship with the SD rosette leaf phenotype, forming an allelic series of phenotypic severity. This allelic series strongly supports our observation of either slower growth rate (i.e. delayed addition of leaves) or early flowering of \emph{pft1-2} as measured by SD rosette leaves at flowering.
	PFT1 genetically interacts with the red/far-red light receptor phyB, which governs petiole length through the shade avoidance response \cite{Cerdan2003,Wollenberg2008}. We measured petiole length at bolting for plants grown under LD to evaluate the strength of their shade avoidance response, and thus whether the genetic interaction is affected by repeat length. Like the flowering time phenotypes, we found that the 1R allele most effectively rescued the long-petiole phenotype of the \emph{pft1-2} null among all STR alleles (Figure 2), though some alleles (e.g. 1.5R) show a rescue that is nearly as good.
In summary, plants expressing the 1R transgene most closely resembled wild-type plants across a range of adult phenotypes. In contrast, the other STR alleles showed inconsistent performance across these phenotypes, rescuing only some phenotypes or at times out-performing wild-type.

\section{PFT1 STR alleles fail to rescue early seedling phenotypes}
We next assessed quantitative phenotypes in early seedling development, some of which had been previously connected to PFT1 function. Specifically, we measured hypocotyl and root length of dark-grown seedlings and examined germination in the presence of salt (known to be defective in \emph{pft1} mutants) (Elfving et al. 2011). The \emph{pft1-2} mutant showed the previously reported effect on hypocotyl length as well as a novel defect in root length (Figure 3A). None of the transgenic lines, including the one containing the 1R allele, effectively rescued these \emph{pft1-2} phenotypes (Figure 3A). Similarly, 1R was not able to rescue the germination defect of \emph{pft1-2} on high-salt media. However, both the 1.5R and 0.5R alleles were able to rescue this phenotype (Figure 3B). In summary, no single STR allele, including the endogenous 1R, was consistently able to rescue the early seedling phenotypes of the \emph{pft1-2} mutant. One explanation for the failure of the endogenous STR (PFT1-1R) to rescue early seedling phenotypes is that the PFT1 transgene represents only the larger of two splice forms. The smaller PFT1 splice form, which we did not test, may play a more important role in early seedling development. To explore this hypothesis, we measured mRNA levels of the two splice forms in pooled 7-day seedlings grown under the tested conditions and various adult tissues at flowering in Col-0 plants. However, we found that both splice forms were expressed in all samples, and in all samples the larger splice form was the predominant form (data not shown). The possibility remains that downstream regulation or tissue-specific expression may lead to a requirement for the smaller splice form in early seedlings.

\section{Summarizing PFT1 STR function across all tested phenotypes}
Given the complex phenotypic responses to PFT1 STR substitutions, results were equivocal as to which STR allele demonstrated the most wild-type-like phenotype across traits, as measured by its sufficiency in rescuing \emph{pft1-2} null phenotypes. To summarize the various phenotypes, we calculated the mean of each quantitative phenotype for each allele, and used principal component analysis (PCA) to visualize the joint distribution of phenotypes observed. 
All STR alleles were distributed between the \emph{pft1-2} null and wild-type (WT) in PC1, which was strongly associated with adult traits and represented a majority of phenotypic variation among lines (Figure 4). PC1 showed that 1R was the most generally efficacious allele for adult phenotypes. However, 1R showed incomplete rescue in early seedling phenotypes such as hypocotyl length, which drove PC2. All STR alleles showed substantial rescue in adult phenotypes, and even the 0R allele showed a partial rescue in some phenotypes; however, rescue of early seedling phenotypes was generally poor for all alleles. The first principal component also captured our observation that the \emph{pft1-2} flowering defect reversed sign in SD vs. LD: according to Figure 4, SD and LD quantitative phenotypes are both strongly represented on principal component 1, but they show opposite directionality. We take this observation as support of this hitherto-unknown complexity in PFT1 function. 

\section{Discussion}
STR-containing proteins pose an intriguing puzzle they are prone to in-frame mutations, which in many instances lead to dramatic phenotypic changes (Gemayel et al. 2010). Although STR-dependent variation has been linked to adaptation in a few cases, the presence of mutationally labile STRs in functionally important core components of cell biology seems counterintuitive. PFT1, also known as MED25, is a core component of the transcriptional machinery across eukaryotes and contains an STR that is predicted to be highly variable in length. Contrary to this prediction, we found PFT1 STR variation to be minimal, consistent with substantial functional constraint. The existing residual variation (~2\% of reference STR length, as opposed to >100\% for the \emph{ELF3} STR in the same \emph{A. thaliana} strains) suggests that the PFT1 STR is mutationally labile like other STRs. In fact, several of the synthetic PFT1 alleles examined in this study arose spontaneously during cloning. Strong functional constraint, however, may select against such deviations in STR length \emph{in planta}. 
Here, we establish the essentiality of the full-length \emph{PFT1} STR and its encoded polyQ tract for proper PFT1 function in \emph{A. thaliana}. We found that diverse developmental phenotypes were altered by the substitution of alternative STR lengths for the endogenous length. Leveraging the support of the \emph{PFT1} STR allelic series, we report new aspects of PFT1 function in flowering time and root development. 

\subsection{The PFT1 STR is required for PFT1 function in adult traits}
The PFT1 0R lines did not effectively complement \emph{pft1-2} for adult phenotypes, suggesting a crucial role of the PFT1 STR in regulating the onset of flowering and shade avoidance. Generally, PFT1-1R was most effective in producing wild-type-like adult phenotypes. The precise length of the STR, however, seemed less important for the onset of flowering in LD. With exception of PFT1-0R, all other STR alleles were also able to rescue the loss-of-function mutant to some extent, suggesting that as long as some repeat sequence is present, the PFT1 gene product can fulfill this function. Under other conditions, and for other adult phenotypes, requirements for PFT1 STR length appeared more stringent. Specifically, under SD, the rosette leaf number phenotype of the \emph{pft1-2} mutant can only be rescued by PFT1-1R, while STR alleles perform worse with increasing distance from this length optimum.

\subsection{pft1-2 mutants are late-flowering in LD but not SD}
\emph{pft1-2} plants had fewer rosette leaves at flowering in SD, but more rosette leaves in LD, consistent with previous, largely undiscussed observations \cite{Cerdan2003}. Under LD conditions, \emph{pft1} null mutants flowered late, as described in several previous studies \cite{Cerdan2003,Wollenberg2008}, but we observe no such phenotype under SD conditions, contradicting at least one prior study \cite{Cerdan2003}. These data suggest that while PFT1 functions as a flowering activator under LD, its role is more complex under SD. 
One recent study showed that PFT1 function in LD is dependent upon its ability to bind E3 ubiquitin ligases \cite{Inigo2012}. Inhibition of proteasome activity also prevents PFT1 from promoting FT transcription and thus inducing flowering, suggesting that degradation of PFT1 or associated proteins is a critical feature of PFT1s transcriptional activation of flowering in LD. If this degradation is somehow down-regulated in SD, PFT1 could switch from a flowering activator to a repressor, through decreased Mediator complex turnover at promoters. Recent studies raised the possibility that different PFT1-dependent signaling cascades have different requirements for PFT1 turnover \cite{Ou2011,Kidd2009}, which may contribute to the condition-specific PFT1 flowering phenotype we observe. Conservatively, we conclude that the regulatory process that mediates the phenotypic reversal between LD and SD depends on the endogenous PFT1 STR allele, suggesting that the polyQ is crucial to PFT1s activity as both activator and potentially as a repressor of flowering. 

\subsection{Incomplete complementation of germination and hypocotyl length by the PFT1 constructs}
Whereas \emph{pft1-2} adult phenotypes were rescued by the PFT1-1R allele, most of our transgenic lines could not fully rescue \emph{pft1-2} early seedling phenotypes of 1) germination under salt, 2) hypocotyl length, and 3) root length. The PFT1 gene is predicted to have two different splice forms, the larger of which was used to generate our constructs (both splice forms contain the STR). Several studies have shown that, under stress conditions, different splice forms of the same gene can play distinct roles \cite{Yan2012,Leviatan2013,Staiger2013}. We note that the conditions under which PFT1-1R fails to complement are also potentially stressful conditions (artificial media, sucrose, high salt, dark). The shorter splice form of PFT1 may be required in signaling pathways triggered under stress conditions. We presume that the failure to complement results from a deficiency related to this missing splice form. However, hypocotyl length was the only trait in which all examined STR alleles resembled the \emph{pft1-2} mutant. The significant functional differentiation among the STR alleles for root length and germination suggests that the large splice form does retain at least some function in early seedling traits. 

\subsection{Implications for STR and PFT1 biology}
Implications for STR and PFT1 biology:
Coding and regulatory STRs have been previously studied and discussed as a means of facilitating evolutionary innovation \cite{Verstrepen2005}. However, this means of innovation is based upon the same sequence characteristics that promote protein-protein and protein-DNA binding \cite{Escher2000,Schaefer2012}, such that STR variability must be balanced against functional constraints. This balance has recently been described for a set of 18 coding dinucleotide STRs in humans, which are maintained by natural selection even though any mutation is likely to cause frame-shift mutations \cite{Haasl2014}. These results, coupled with our observations, lend credence to these authors previous argument that not all STRs act as agents of adaptive change \cite{Haasl2013}. Considering again the possibility that more conserved coding tandem repeats have distinct functions from non-conserved tandem repeats \cite{Schaper2014}, we suggest that PFT1 and ELF3 can serve as models for these two selective regimes, and that the structural roles of their respective polyQs underlie the differences in natural variation between the two. In some cases, such as ELF3, high variability is not always inconsistent with function, even while holding genetic background constant \cite{Undurraga2012}. In PFT1, we have identified a STR whose low variability reflects strong functional constraints. We speculate that these constraints are associated with a structural role for the PFT1 polyQ in the Mediator complex, either in protein-protein interactions with other subunits or in protein-DNA interactions with target promoters. Given that a glutamine-rich C-terminus appears to be a conserved feature of MED25 even in metazoans, we expect that our results are generalizable to Mediator function wherever this protein is present. Future work will be necessary in understanding possible mechanisms by which the MED25 polyQ might facilitate Mediator complex function and contribute to ontogeny throughout life. Moreover, attempts must be made to understand the biological and structural characteristics unique to polyQ-containing proteins that tolerate (or encourage) polyQ variation, as opposed to those polyQ-containing proteins (like PFT1) that are under strong functional constraints.

% ========== MAX put in a chapter here
\chapter{Short tandem repeats and quantitative genetics}



Portions of this chapter were published under the following references:
\begin{itemize}
\item Maximilian O. Press, Keisha D. Carlson, and Christine Queitsch. 

The overdue promise of short tandem repeat variation for heritability. \emph{Trends in Genetics}, 30(11):504-512, August 2014.\footnote{Reprinted with permission from Cell Press.}
\item Keisha D. Carlson, Peter H. Sudmant, Maximilian O. Press, Evan E. Eichler, Jay Shendure, and Christine Queitsch. MIPSTR: a method for multiplex genotyping of germline and somatic STR variation across many individuals. \emph{Genome Research}, 25(5):750-761, May 2015.
\end{itemize}

Keisha Carlson and Peter Sudmant contributed STR genotyping data and analysis. 
% have any? don't think so...
Supporting figures and tables can be found in Appendix C.

\section{Abstract}
Short tandem repeat (STR) variation has been proposed as a major explanatory factor in the heritability of complex traits in humans and model organisms. However, we still struggle to incorporate STR variation into genotype-phenotype maps. Here, we review the promise of STRs in contributing to complex trait heritability, and highlight the challenges that STRs pose due to their repetitive nature. We argue that STR variants are more likely than single nucleotide variants to have epistatic interactions, reiterate the need for targeted assays to accurately genotype STRs, and call for more appropriate statistical methods in detecting STR-phenotype associations. Lastly, somatic STR variation within individuals may serve as a read-out of disease susceptibility, and is thus potentially a valuable covariate for future association studies. 

\section{The missing heritability of complex diseases and STR variation.}
Complex diseases such as diabetes, various cancers, cardiovascular disease, and neurological disorders cluster in families, and are thus considered to have a genetic component \cite{Manolio2009,Eichler2010,Gibson2011}. The identification of these genetic factors has proven challenging; although genome-wide association (GWA) studies have identified many genetic variants that are associated with complex diseases, these generally confer less disease risk than expected from empirical estimates of heritability. This discrepancy, termed the missing heritability, has been attributed to many factors \cite{Manolio2009,Eichler2010,Gibson2011,Zuk2012,Bloom2013,Heng2010}. A trivial explanation is that shared environments among relatives may artificially inflate estimates of heritability. However, missing heritability may also be due to variants in the human genome that are currently inaccessible at a population scale \cite{Manolio2009,Eichler2010}]. One such class of variation is short tandem repeat (STR) unit number variation. Some have previously suggested that adding STR variation to existing genetic models would considerably increase the proportion of heritability explained by genetic factors in human disease \cite{Fondon2008,Hannan2010}. Three percent of the human genome consists of STRs \cite{Subramanian2003} and 6\% of human coding regions are estimated to contain STR variation \cite{ODushlaine2005,Mularoni2006}. Recently, the first catalog of genome-wide population-scale human STR variation has appeared \cite{Willems2014}, opening up new possibilities for understanding the contribution of STRs to human genetic diseases. This catalog, and similar data sources \cite{Mackay2012}, have appeared decades after initial calls for the assessment of the role of STRs in phenotypic variation \cite{kashi1997}, lagging behind surveys of other genomic elements. Much of the initial interest in STRs was generated by the discovery of phenomena such as genetic anticipation, which are mediated by the unique features of STRs \cite{Sutherland1991}. As we will discuss, new and forthcoming data sources will help to realize the long-deferred promise of STRs for explaining heritability. 
STRs consist of short (2-10 bp) DNA sequences (units) that are repeated head-to-tail multiple times. This structure causes frequent errors in recombination and replication that add or subtract units, leading to STR mutation rates that are 10-fold to 104-fold higher than those of non-repetitive loci \cite{Eckert2009,Legendre2007}. Due to technical barriers, STR variation has until very recently remained inaccessible to genome-wide assessment. 
STRs are often conserved (even if their unit number or even sequence changes), especially in coding sequences \cite{Zhao2014,Schaper2014,Li2012,Gemayel2010}. In both humans and the yeast Saccharomyces cerevisiae, promoter regions are known to be dramatically enriched for STRs \cite{Vinces2009,Sawaya2013}. In coding regions, STRs tend to occur in genes with roles in transcriptional regulation, DNA binding, protein-protein binding, and developmental processes \cite{Eckert2009,Gemayel2010,Vinces2009}. These consistent functional enrichments across vastly diverged lineages suggest important functional roles for STRs.
Indeed, analysis of STR variation in the Drosophila Genetic Reference Panel identified dozens of associations between STR variants and quantitative phenotypes in recombinant inbred fly lines \cite{Mackay2012}. Moreover, accumulating evidence from exhaustive genetic studies shows that STR variation has dramatic, often background-dependent phenotypic effects in model organisms \cite{Undurraga2012,Michael2007,Sawyer1997,Rosas2014,Scarpino2013,Boland1998}. Together, these findings suggest that STR variation has the potential to dramatically revise the heritability estimates attributable to genetic factors. 
The high STR mutation rate also leads to substantial somatic variation of STR loci within individuals. In fact, this somatic variation, also called microsatellite instability (MSI), has been used for decades as a biomarker for different classes of cancer \cite{Kovalchuk2003}. Recent studies demonstrate that organisms exposed to various environmental stresses and perturbations show increased genome instability, including MSI \cite{Specchia2010,Mittelman2010,Queitsch2012}. MSI may be useful as a biomarker for cellular stress states that may predispose to disease. 
The broad interest in STR variation has led to the development of techniques for high-throughput genotyping of STRs \cite{Guilmatre2013,Duitama2014} and an explosion of analysis tools for extracting STR variation from existing sequence data \cite{Highnam2013,Gymrek2012,Cao2014}. However, the precision of these methods remains limited, due to a combination of low effective coverage of STRs and the lack of robust models for distinguishing technical error from somatic variation. Attempts to use STR variation for GWA in a fashion equivalent to SNV variation may be underpowered and confounded by the unique characteristics of this class of variants. In this review, we discuss the latest advances in these fields, and lay out a set of priorities for the future study of STRs. 

\section{STR variation is associated with human genetic diseases}
Within coding regions, STR mutations are generally in-frame additions and subtractions of repeat units, resulting in proteins with variable, low-complexity amino acid runs \cite{Gemayel2010}. These mutations can result in phenotypic effects and lead to genetic disorders; several neurological diseases (spinocerebellar ataxias, Huntingtons disease, spinobulbar muscular atrophy, dentatorubral-pallidoluysian atrophy, intellectual disability, etc.) are a consequence of dramatically expanded STR alleles \cite{Fondon2008,Gatchel2005,Poeta2013}. Many of these disease-associated STR expansions behave as dominant gain-of-function mutations \cite{Fondon2008}. However, even comparatively modest coding STR variation may confer disease risk or behavioral phenotypes, according to a variety of single-marker association studies \cite{Caspi2003,Zhang2012,Eisenegger2013,Inanir2013}; for instance, variants in separate coding STRs in RUNX2 are associated with defects in bone mineralization, higher incidence of fractures \cite{Morrison2012,Morrison2013}; STR variation in this gene in dogs is also associated with craniofacial phenotypes \cite{Fondon2004}. Noncoding STR variation in regulatory sequences can affect transcription, RNA stability, and chromatin organization. For instance, certain STR variants alter CFTR expression and thus cystic fibrosis status \cite{Eckert2009}. We take these studies as evidence that STR variation, even in the absence of large expansions, may contribute significantly to the heritability of human traits and genetic diseases. 
The severity of the STR expansion-associated diseases may suggest that natural selection should eliminate STRs in functional regions, but several recent studies across many organisms indicate that variable STRs are globally maintained \cite{Zhao2014,Schaper2014,Sawaya2013,Persi2013,Mularoni2007}. For example, the pre-expansion polyQ-encoding STR in the human gene SCA2 is under positive selection, suggesting that this variable STR is actively maintained in spite of the pathogenic expansions that do occasionally occur and cause spinocerebellar ataxia \cite{Mularoni2010}. Considering both the evidence of positive selection on STRs and the functional enrichments of STR-containing genes, several authors have proposed that functional STRs are maintained because they confer `evolvability', or the capacity for fast adaptation \cite{Li2012,Gemayel2010,Yu2005,Gemayel2012,Laidlaw2007}. This suggestion is intriguing, in part because many STR mutations are dominant, and, when beneficial, can quickly sweep to fixation. Although we do not further discuss these evolutionary considerations here, they underscore the phenotypic potential of STR variation.

\section{STR variation has dramatic background-dependent effects on phenotype}
To date, the functional consequences of unit number variation in selected STRs have been studied in plants, fungi, flies, voles, dogs, and fish \cite{Undurraga2012,Sawyer1997,Scarpino2013,Sureshkumar2009,Hammock2005}, among other organisms. In \emph{Saccharomyces cerevisiae}, STR unit number in the \emph{FLO1} gene accurately predicts the phenotype of cell-cell and cell-substrate adhesion (flocculation); flocculation provides protection against various stresses \cite{Verstrepen2005}. STR variation in yeast promoters has been shown to alter gene expression \cite{Vinces2009}. In \emph{Drosophila melanogaster}, \emph{Neurospora crassa}, and \emph{Arabidopsis thaliana}, natural coding STR variation in circadian clock genes alters diurnal rhythmicity and developmental timing \cite{Michael2007,Sawyer1997,Rosas2014,Butler2007}. Some have proposed that the large phenotypic responses to selection observed in the Canidae are a consequence of elevated STR mutation rates relative to other mammalian clades \cite{Fondon2004,Laidlaw2007}. We can state unambiguously that naturally variable STRs underlie dramatic phenotypic variation in model organisms. 
Beyond the observable fact that variable STRs affect phenotype, we can make specific predictions about the components of phenotypic variation that they affect. Both theoretical expectations and empirical data indicate that STR variants are likely to participate in epistatic interactions, and probably more so than most SNVs. One plausible hypothesis is that STRs act as mutational modifiers of other loci, as may be expected intuitively from their elevated mutation rate (Figure XXXXX). % figure here 
This expectation is borne out in the handful of studies reporting exhaustive genetic analysis of STRs. For instance, in the \emph{Xiphophorus} genus of fish, a genetic incompatibility has recently been attributed to the interaction between the \emph{xmrk} oncogene and an STR in the promoter of the tumor suppressor \emph{cdkn2a/b} . If the xmrk gene product is not properly regulated by cdkn2a/b, fish develop fatal melanomas, a two-locus Bateson-Dobzhansky-Muller incompatibility described in classic genetic experiments (Figure 1A) \cite{Gordon1927,Kosswig1928,Meierjohann2006}. Expansions in the \emph{cdkn2a/b} promoter STR are associated with the presence of a functional copy of the xmrk oncogene across species, and are thought to functionally repress the activity of the xmrk gene product through increased dosage of the tumor suppressor \cite{Scarpino2013}. 
Similarly, we have shown that natural variation in the polyQ-encoding ELF3 STR significantly affects all ELF3-dependent phenotypes in the plant \emph{A. thaliana}, with ELF3 STR length and phenotype showing a strikingly nonlinear relationship (Figure 1B) \cite{Undurraga2012}. Some naturally occurring ELF3 STR variants phenocopy elf3-loss-function mutants in a common reference background (Figure 1B), suggesting background-specific modifiers. Indeed, when we compare the phenotypic effects of each \emph{ELF3} STR variant between two divergent backgrounds, Columbia (Col-0) and Wassilewskija (Ws), we find dramatic differences. The endogenous STR alleles from these two strains (Col-0 7 units, Ws 16 units) show mutual incompatibility when exchanged between backgrounds. The ELF3 protein is thought to function as an adaptor protein or physical bridge in diverse protein complexes \cite{Nusinow2011,Yu2008}. We speculated that background-specific polymorphisms in these interacting proteins underlie the \emph{ELF3} STR-dependent background effect.
	Also in A. thaliana, a variable STR in the promoter of the \emph{CONSTANS} gene has been linked to phenotypic variation in the onset of flowering \cite{Rosas2014}. \emph{CONSTANS} encodes a major regulatory protein that promotes flowering. Transgenic experiments demonstrate that this regulatory STR variation affects \emph{CONSTANS} expression and hence onset of flowering. However, the effects of this STR variation depend on the presence of a functional allele of \emph{FRIGIDA}, a negative regulator of flowering that is highly polymorphic across A. thaliana populations.
	A dramatic example of incompatibility can be found in an intronic repeat in the \emph{IIL1} gene in A. thaliana, which was found to be dramatically expanded in one strain \cite{Sureshkumar2009}. The expansion delayed flowering under high temperatures, but when crossed into the reference genetic background, a strongly interacting locus modifies this phenotype.
	In the Drosophila genus, coding STR variation in the \emph{per} gene co-evolves with other variants \cite{Peixoto1998,Peixoto1993}. Transgenic flies expressing chimeric per genes with a \emph{D. melanogaster} STR domain fused to a \emph{D. pseudoobscura} flanking region (and vice versa) have arrhythmic circadian clocks, indicating the modifying effect of flanking variation in generating an STR-based genetic incompatibility. Among STRs subjected to exhaustive genetic study, to our knowledge, only the yeast \emph{FLO1} coding STR has no known modifiers due to variation in genetic background \cite{Verstrepen2005}.
In addition to these exhaustive genetic studies, there are several other observations that support the role of the genetic background in controlling the phenotypic effects of STRs. For instance, experiments in Caenorhabditis elegans and human cells indicate that the phenotypic effects of proteins with expanded polyQ tracts are modulated by genetic background \cite{Gidalevitz2013}, or by variants in interacting proteins \cite{Metzger2008}. In humans, genetic association studies indicate the existence of genetic modifiers of polyQ expansion disorders for both Huntingtons disease \cite{Rubinstein1997} and spinocerebellar ataxias \cite{Zuhlke2005}. Taken together, these experimental and observational data support our argument that functional STRs are likely to be enriched for variants in epistasis with other loci.
STRs with background-dependent phenotypic effects tend to either encode polyQ tracts or reside in promoter regions. There are good reasons to expect that these STR classes might be enriched in DNA/protein-protein interactions that could underlie epistasis. PolyQ tracts, specifically, often bind DNA surfaces \cite{Escher2000}, and an analysis of human protein interactome data found that polyQ-containing proteins engage in more physical interactions with other proteins than those without polyQs \cite{Schaefer2012}. Similarly, noncoding STRs in regulatory regions may compensate for mutations in trans-acting factors, as observed for the STRs in the cdkn2a/b promoter in Xiphophorus \cite{Scarpino2013} and in the CONSTANS promoter in A. thaliana \cite{Rosas2014}. We suggest that polymorphisms in protein interaction partners or in transcriptional regulators are plausible explanations for the observed background effects. In summary, we expect that STR variation is likely to contribute a substantial epistatic component to heritability, which has important implications for their use in explaining phenotypic variation. 

\begin{figure}
\includegraphics[width=6in]{060414_tig_fig1_marked-01.png}
\caption[Examples of STRs mediating genetic incompatibilities.]{Genetic and transgenic analysis reveals STR-mediated incompatibilities. A, the Gordon-Kosswig-Anders cross shows a genetic incompatibility between two fish species in the Xiphophorus genus. Modified from Meierjohann and Schartl \cite{Meierjohann2006}. F\textsubscript{1} hybrids back-crossed to their \emph{X. helleri} parent yield a 3:1 ratio of viability, where the inviables result from co-segregation of the functional \emph{xmrk} gene and a short STR allele in the \emph{cdkn2a/b promoter}. Shading indicates melanism conferred by \emph{xmrk}. B, genetic background is epistatic to effects of ELF3 STR variation in \emph{A. thaliana}. Expression-matched transgenic plants with various alleles of the \emph{ELF3} STR in the Columbia (Col-0) and Wassilewskija (Ws) backgrounds, showing endogenous, exogenous, and synthetic (0) alleles in each background \cite{Undurraga2012}. White boxes indicate transgenic plants carrying the \emph{ELF3} STR endogenous to their respective background; white arrowheads indicate early-flowering \emph{ELF3} STR genotypes (elf3 mutants and poorly-functioning \emph{ELF3} STR alleles confer early flowering).}
\label{fig:strfig1}
\end{figure}
%\addtocounter{figure}{-1}
%\captionsetup[figure]{list=no}
%\captionof{figure}{ }


\section{Lack of statistical models for detecting STR-phenotype associations in GWA.}
Assuming that we obtain accurate, population-scale genotype data for STRs, we may not yet have statistical tools appropriate for detecting STR associations with phenotype \cite{Hannan2010}. In diploid organisms, a biallelic SNV is typically analyzed by modeling phenotype as a function of the number of non-reference alleles at that locus (0, 1, or 2) in each individual. A null hypothesis of no monotonic relationship between phenotype and the allele count is then formulated and tested \cite{Hayes2013}. This framework cannot accommodate more than two alleles, which we would expect for many STRs. Simply using tagged SNVs linked to STRs to perform GWA is unfeasible, because linkage disequilibrium decays very quickly between SNVs and STRs across human populations \cite{Willems2014}.
To address these complications, a previous study attempted GWA between STR genotypes and human disease phenotypes by comparing relative frequencies of various alleles in pooled DNA from cases and controls \cite{Oka2003}. By pooling samples, this approach eases the analysis of multiallelic loci, but it loses information by ignoring specific individuals. 
In a more recent study, the authors used logistic regression and the analysis of variance to detect associations between STR alleles and quantitative phenotypes in an inbred Drosophila mapping population \cite{Mackay2012}. Given that significant associations were detected, such approaches may be sufficiently powerful in recombinant inbred lines. However, their strategy relied on homozygosity, and considered multiallelic STRs in a pairwise fashion, so these straightforward methods will lose power with outbred populations and multiallelic STRs. 
The central confounder of these studies is that most STRs of appreciable variability (and thus, interest) are multiallelic, as a simple consequence of the STR mutational mechanism \cite{Legendre2007}. This multiallelic feature could be accommodated by treating STR alleles categorically, but this choice entails a corresponding reduction in power, because many alleles are rare. 
Some studies have reported linear associations between STR unit number and quantitative phenotypes \cite{Sawyer1997,Verstrepen2005}, suggesting that using simple tests of linear correlation between these variables may be a powerful option. However, this linearity (or even monotonicity) of the relationship between STR unit number genotype and phenotype is a poorly-supported assumption \cite{Undurraga2012}. Nonetheless, STR unit number is a numerical variable, and it would be preferable to gain power from treating it as such. For instance, more similar STR unit number genotypes might be associated with more similar phenotypes, but this intuition may be difficult to generalize. 
	Lastly, both intuition (Figure %%% the figure
	) and the studies discussed above lead us to expect that relatively many phenotypically relevant variable STRs will show epistasis with other loci. This epistasis will reduce power in tests of association between STRs and phenotype \cite{Mackay2014}, given the inadequacy of the current paradigm of quantitative genetics in detecting and modeling the effects of epistasis \cite{Mackay2014,Nelson2013}. At present, targeted and exhaustive genetic studies (as described above) are the only effective method for understanding the effects of epistasis. 
In total, these obstacles present a daunting challenge for the integration of STR genotypes into the current genotype-phenotype maps. Overall, we call for a reappraisal of statistical methodologies for use in GWA with STR variation to account for these various STR-specific confounders.     

\begin{figure}
\includegraphics[width=6in]{tig_figs/repeatmumodel_051214.pdf}
\caption[A locus with higher mutation rates allows genetic modification of unfavorable genotypes at interacting loci.]{A locus with higher mutation rates allows genetic modification of unfavorable genotypes at interacting loci. Top, a model of evolution under epistasis with only one slow mutation rate. Middle, a model of evolution under epistasis with a slow and a fast mutation rate. Boxes represent loci, stars represent SNV-type mutations, black and white checkering indicates an STR locus (a/b, a*/b, and a*/b* signify different genotypes). Arrows with numbers represent possible mutations and their respective rates. Bottom, fitness of each genotype under both models. We expect that the model with two mutation rates will occupy the fully derived state (a*/b*) more quickly.
}
\label{fig:strfig2}
\end{figure}

\section{Modifier mutations leading to epistasis are expected in STRs.}
We have previously proposed that STRs might be more susceptible to genetic interactions \cite{Undurraga2012}, as we will briefly explicate here. Consider a simple two-locus haploid model under panmixis, in which loci A and B each start with a single allele ($ab$) and have the same probability $p$ per generation of mutating to a second allele ($a^*$ or $b^*$), with $p$ also as the probability per generation of reverting mutations (Figure ~\ref{fig:strfig2}). Let us further assume that A and B are in sign epistasis [99] (that is, $a^*b$ and/or $ab^*$ have fitness less than $ab$ and $a^*b^*$). To escape the unfavorable $a^*b$ genotype, the organism may either revert to $ab$ or mutate forward to $a^*b^*$. When the A and B loci have equal mutation rates, we expect that the reversion of a single mutant is just as likely as a second mutation, and consequently that $a^*b^*$ individuals will appear only relatively rarely and slowly. However, consider a similar model, in which locus B has an elevated mutation rate $p_{b} > p_{a}$. In this case, the a*b genotype has a higher probability of a second, modifying mutation to $a^*b^*$ than of a reversion to $ab$. Moreover, flux along the other mutational path ($ab \Rightarrow ab^* \Rightarrow a^*b^*$) will be increased. In sum, $a^*b^*$ genotypes will arise at higher rates, and will attain their equilibrium frequency much more rapidly, if either A or B has an elevated mutation rate \cite{Felsenstein2013} (p.131). This scenario can lead quickly to an equilibrium population in which incompatible epistatic alleles are frequent, even though recombinants have lower fitness. Relaxing the assumption of no population structure will further speed this process. Consequently, we would expect STRs and other loci with high mutation rates to be more likely to modify other alleles than loci with lower mutation rates, as long as we assume that all loci are equally capable of genetic interactions. This process may be referred to as coadaptation. For a rigorous model of the evolution of hybrid incompatibility, see Orr \cite{Orr1995}.


\section{Analysis of selection on STR variation in \emph{A. thaliana}.}
If the predicted phenotypic effects of STR variation exist and are relevant, then we should be able to observe signatures of selection on STR variation. For SNV variation, measures such as $\omega$ (ratio of nonsynonymous to synonymous coding variants) and $\Pi$ (nucleotide diversity) can be used to assess this question, but for STRs no analogous measures exist. Consequently, we developed a heuristic method to estimate the ratio of observed to expected STR variation.

The unit number, unit length, and purity of a given STR locus in a high-quality reference genome predict its variation across individuals \cite{Legendre2007}. STRs with high unit number, short unit length, and high purity are typically highly variable. From population-scale STR genotype data \cite{Carlson2015}, we assessed the correlation of predicted variation of STRs (VARscore) and observed STR variation across A. thaliana strains.

\begin{figure}
\includegraphics[width=4in]{tig_figs/logsd_varscore_011814.pdf}
\caption[Observed and predicted STR variation showed greater correlation for non-coding STRs than coding STRs. ]{Observed and predicted STR variation showed greater correlation for non-coding STRs than coding STRs. The correlation between the observed log10 of the standard deviation of STR unit number across strains (y-axis) and the VARscore (x-axis), which predicts STR variation from sequence characteristics. Black points are non-coding STRs, red points are coding STRs. Outliers may indicate functional importance (ELF3 STR is indicated).
}
\label{fig:strfig3}
\end{figure}

In general, VARscore was well correlated with observed variation across STRs (r=0.68, Figure ~\ref{fig:strfig3}), a substantially better agreement than previously observed \cite{Duitama2014}. However, this correlation was substantially weaker among coding STRs (r=0.46) than among non-coding STRs (r=0.75). This discrepancy suggests that sequence characteristics alone do not suffice to predict whether coding STRs vary on a population scale. Coding STRs are more likely to be functionally important, and thus are less subject to the `neutral model' of the VARscore prediction. Deviation of predicted STR variation (i.e. VARscore) from observed variation may thus hold information about STR function and selective pressures acting upon it. Specifically, STRs that are observed to be more variable than predicted may be under diversifying selection, whereas those STRs that are observed to be less variable than predicted may be functionally constrained and under purifying selection. For example, the STR in the gene ELF3 is highly variable across strains, ranging from 7 units to as many as 29 units in a set of strains previously analyzed by Sanger sequencing \cite{Undurraga2012}. The phenotypes associated with variation in the ELF3 STR change dramatically in different genetic backgrounds, suggesting co-evolution of the ELF3-STR with epistatically interacting loci \cite{Undurraga2012}. Given this STRs strong background-dependent phenotypes, it is likely under diversifying selection and, correspondingly, it is much more variable than predicted. 

\section{Association of STR variation with phenotypic characters in \emph{A. thaliana}.}
A complementary approach for identifying STRs with important function in modulating phenotype is genome-wide association of STR genotypes with phenotypes. The standard statistical methods for associating genotype with phenotype were developed for common, biallelic SNVs \cite{Hayes2013}. STRs are typically multiallelic and often involved in epistatic interactions, both of which make it difficult to associate STR genotype with phenotype using standard methods \cite{Press2014}. Nevertheless, we performed a nave association analysis to determine whether STR variation across strains was associated with well-characterized phenotypes \cite{Atwell2010}. These phenotypes included morphology, developmental timing (flowering), ionomics, and gene expression, among others. We used the one-way analysis of variance (ANOVA) to detect associations between STR loci and phenotypes following previous studies \cite{Mackay2012}, modeling STR alleles as factors to avoid assumptions of linearity. To minimize spurious associations, we excluded STRs that were typed in fewer than ten strains from this analysis, and for each STR we excluded all strains carrying alleles present in fewer than three strains (rare alleles). We identified 124 significant associations involving 27 STRs and 41 phenotypes at a 1\% false discovery rate (for a list of associations see Supplemental Table 4 in \cite{Carlson2015}). However, an important caveat is that this analysis did not consider population structure, which is another challenge given the different evolutionary trajectories of SNVs and STRs \cite{Willems2014}). Consequently, we also performed a mixed-model analysis treating population structure \cite{Nordborg2005} as a random effect and STR unit numbers as fixed factorial effects. Although more conservative, of the 70 associations found by this method, 56 were shared with nave ANOVA, indicating that most variants thus identified are robust to association method. Furthermore, as previously observed with SNV variation, the complex trait of flowering time has many associations with variable STRs across its various potential measurements \cite{Atwell2010}. 
We further investigated whether these STR-phenotype associations could be identified with common, linked SNVs \cite{Atwell2010}. For each STR-phenotype association, we identified the SNV associated with the same phenotype that is closest to the STR in question. Most phenotype-associated STRs were unlinked to any SNVs associated with the same phenotype; often, associated SNVs were only found on different chromosomes. In fact, the closest SNV resided over 21 kb away from the STR associated with the same phenotype. In A. thaliana, linkage disequilibrium decays at 10 kb and likely decays even faster between SNVs and STRs \cite{Nordborg2005,Kim2007,Willems2014}. Thus, at least for this small set of loci, STR-phenotype associations are not captured with common SNV variation. 

\section{Concluding remarks}
The study of STRs and other under-ascertained genomic elements has the potential to reshape our model of the heritability of complex diseases and traits, both in terms of the overall proportion of heritability explained, and in terms of the components of heritability themselves . Experimental studies in model organisms have taught us that the phenotypic effects of genome-wide STR variation are both dramatic and impossible to understand without taking epistasis into account. In the future, our understanding will be improved by 1) accurate STR population-scale genotyping, 2) more appropriate statistical methods for analyzing STR-phenotype associations, and 3) a broader description of epistasis between STR variation and other loci in determining phenotype.

% ========== MAX put in a chapter here
\chapter{The variable ELF3 polyglutamine hubs an epistatic network}

%A version of this chapter was published under the following reference:

Supporting figures and tables can be found in Appendix D.


% ========== MAX put in a chapter here
\chapter{ELF3 polyglutamine variation reveals a PIF4-independent role in thermoresponsive flowering}

%A version of this chapter was published under the following reference:

Supporting figures and tables can be found in Appendix E.


% ========== MAX put in a chapter here
\chapter{Genome-scale Co-evolutionary Inference Identifies Functions and Clients of Bacterial Hsp90}

A version of this chapter was published under the following reference: 

Maximilian O. Press, Hui Li, Nicole Creanza, Guenter Kramer, Christine Queitsch, Victor Sourjik, and Elhanan Borenstein. Genome-scale co-evolutionary inference identifies functions and clients of bacterial Hsp90. \emph{PLoS Genetics}, 9(7):e1003631, 2013.

Hui Li, Guenter Kramer, and Victor Sourjik contributed E. coli strains, experiments, and figures. Nicole Creanza and Aviv Regev contributed ideas and preliminary analyses at the initiation of this work.

% ========== MAX put in a chapter here
Supporting figures and tables can be found in Appendix F.


\section{Abstract}
The molecular chaperone Hsp90 is essential in eukaryotes, in which it facilitates the folding of developmental regulators and signal transduction proteins known as Hsp90 clients. In contrast, Hsp90 is not essential in bacteria, and a broad characterization of its molecular and organismal function is lacking. To enable such characterization, we used a genome-scale phylogenetic analysis to identify genes that co-evolve with bacterial Hsp90. We find that genes whose gain and loss were coordinated with Hsp90 throughout bacterial evolution tended to function in flagellar assembly, chemotaxis, and bacterial secretion, suggesting that Hsp90 may aid assembly of protein complexes. To add to the limited set of known bacterial Hsp90 clients, we further developed a statistical method to predict putative clients. We validated our predictions by demonstrating that the flagellar protein FliN and the chemotaxis kinase CheA behaved as Hsp90 clients in E. coli, confirming the predicted role of Hsp90 in chemotaxis and flagellar assembly. Furthermore, normal Hsp90 function is important for wild-type motility and/or chemotaxis in E. coli. This novel function of bacterial Hsp90 agreed with our subsequent finding that Hsp90 is associated with a preference for multiple habitats and may therefore face a complex selection regime. Taken together, our results reveal previously unknown functions of bacterial Hsp90 and open avenues for future experimental exploration by implicating Hsp90 in the assembly of membrane protein complexes and adaptation to novel environments.

\section{Introduction}
In eukaryotes, the universally conserved and essential chaperone Hsp90 aids the folding of key proteins in development and responses to environmental stimuli [13]. In yeast, up to 10\% of all proteins are estimated to be Hsp90 clients under standard culture conditions [4]. Hsp90 function is even more important under stressful conditions that challenge protein folding, such as increased temperature [5]. The activity of eukaryotic Hsp90 is further modulated by various co-chaperones, which confer substrate specificity and alter protein folding kinetics [2,5]. Depletion of eukaryotic Hsp90 in vivo increases phenotypic variation, reveals cryptic heritable variation, and increases penetrance of mutations [69]. Accordingly, eukaryotic Hsp90 enables organisms to maintain a stable phenotype in the face of environmental and genetic perturbation and to correctly interpret environmental stimuli. 
In stark contrast, in prokarya, Hsp90 is not essential [10] and many bacterial genomes lack Hsp90 altogether [11]. Among Archaea, only very few species contain Hsp90, and those are thought to have gained Hsp90 horizontally from bacteria [11,12]. This fragmented phylogenetic pattern likely results from multiple independent gains and losses, though phylogenetic reconstructions are confused by ancient Hsp90 paralogy [11,12]. At the amino acid level, the E. coli Hsp90 (High-temperature protein G or HtpG) is 42\% identical to its human homolog, suggesting strong stabilizing selection consistent with functional conservation [13]. Indeed, E. coli Hsp90 appears to retain generic protein chaperone activity [14] and homologous mutations cause chaperone defects in both the prokaryotic E. coli and eukaryotic yeast Hsp90 [15]. However, there are no identified obligate Hsp90 co-chaperones in bacteria, adding to the uncertainty regarding the extent of its client spectrum and specificity. 
To date, only three proteins have been implicated as Hsp90 clients in bacteria, which have non-overlapping functions in ribosome assembly, the assembly of light-harvesting complexes, and the CRISPR/Cas immunity system [1618]. Several other proteins have been shown to physically interact with the chaperone [19,20]. These data, together with our knowledge on eukaryotic Hsp90 function, have given rise to the speculation that Hsp90 may facilitate the assembly of oligomeric protein complexes in bacteria, much like it does in eukaryotes [21]. Unlike in eukaryotes, however, further exploration of Hsp90s functional role in bacteria has proven challenging because there are no pleiotropic Hsp90-dependent phenotypes.
To address this challenge, we used a genome-scale co-evolutionary guilt-by-association approach [22,23] to explore the spectrum of conserved Hsp90-associated genes, functions, and organismal traits. Hsp90-associated genes tended to function in flagellar assembly, chemotaxis, and secretion. Consistent with these functions, Hsp90-associated organismal traits included the ability to inhabit multiple environments. To add to the sparse list of known bacterial Hsp90 clients, we further developed a statistical method to predict putative Hsp90 clients, which included flagellar, ribosomal, and chaperone proteins. We validated our predictions experimentally, focusing on two candidates functioning in motility and chemotaxis. Indeed, both the flagellar protein FliN and the kinase CheA were found to be Hsp90 clients in vivo. Our findings demonstrate the power of co-evolutionary inference to correctly identify substrates and functions of conserved genes like bacterial Hsp90. 

\section{Methods}
\subsection{Prokaryotic Hsp90 paralogs}
We downloaded all Hsp90 amino acid sequences (including all paralogs) for bacteria with full KEGG genome annotations from the KEGG database \cite{Ogata1999,Tanabe2012}.  We aligned these sequences using ClustalO [58], and used the PHYLIP package [59] to construct neighbor-joining trees and assess their phylogenetic support through bootstrapping. We assigned Hsp90 families to branches according to bootstrap support for the branch and previous classifications [11,12].  
\subsection{Genome data}
We acquired presence/absence patterns of genes across organisms from the KEGG database release 60.0 (in the form of KEGG Orthology/KO profiles) \cite{Tanabe2012}, and functional annotations from KEGG Class. Genes that were either present in fewer than five species or absent in fewer than five species in the tree of interest were dropped from our analysis, as these genes are unlikely to show meaningful signatures of co-evolution by this method. 
\subsection{Phylogenetic trees}
We obtained the tree constructed by Ciccarelli et al. (Ciccarelli tree) \cite{Ciccarelli2006} and pruned it to 148 bacterial species for which KEGG genome data was available. We also obtained the LTP104 version of the 16S/23S rRNA tree from the All-Species Living Tree Project (Yarza tree) \cite{Yarza2008,Munoz2011}. We used ARB \cite{Ludwig2004} to prune this tree to bacterial species for which KEGG genome data was available. We further pruned this tree to omit clades placed paraphyletically at the taxonomic levels of phylum, class, order, and family. This filtered tree included 797 bacterial species. As BayesTraits cannot process trees with zero-length branches, all branch lengths equal to zero were replaced with a negligible branch length (0.00001, approximately an order of magnitude smaller than the next smallest branch length in each tree). 
\subsection{Organismal trait data}
We acquired organismal trait data from the NCBI Entrez genome project, November 2011 [62]. We recoded all traits into presence/absence patterns for the trait in question. For instance, an organism found to be pathogenic towards any other organism was coded as `1' for the trait of pathogenicity, whereas an annotated organism that was never found to be pathogenic was coded as `0'. Similarly, we coded both thermophilic and hyperthermophilic organisms as `1' for the trait of thermophilicity, whereas all other annotated organisms were coded as `0'; anaerobic organisms were coded as 0 for the trait of aerobicity, whereas all other annotated organisms were recoded as `1'. We define as inhabiting multiple habitats any organism that inhabits more than one of NCBIs habitat categories. For BayesTraits analysis, the tree was pruned to include only species annotated for the trait in question (each trait analysis was accordingly performed on a slightly different set of species; see Table S5 for details on species number for each analysis). 
\subsection{Detecting evolutionary associations with BayesTraits}
A complete description of the BayesTraits (v1.0) framework can be found elsewhere \cite{Pagel1994}. Briefly, consider a character with 2 states, 0 and 1. If a species has 2 such distinct characters, it can occupy 4 possible states: 1:(0,0), 2:(0,1), 3:(1,0), and 4:(1,1). Specifically, if these 2 characters represent the presence or absence of two genes, hsp90A and gene X, these four states correspond to (hsp90A-, X-), (hsp90A+, X-), (hsp90A-, X+), and (hsp90A+, X+). Evolution is then the process by which these genes are gained and lost over time. Consider accordingly an evolutionary process where only one character can change state at a time. Such a process can then be described by 8 parameters for the rates of transition per unit time between these 4 states: $Q = [q_{12}, q_{13}, q_{21}, q_{31}, q_{24}, q_{34}, q_{42}, q_{43}]$, where $q_{xy}$ is the rate of transition from state x to state y. BayesTraits implements this model of evolution as a continuous-time Markov process and estimates each of these rate parameters by maximum-likelihood (ML). We further validated that these ML-based rates are consistent with reversible-jump Markov chain Monte Carlo-derived estimates (Methods; Supporting Text). This estimation is based on a phylogeny and on the states of the two characters at the tips of the phylogeny. Having estimated these rates, BayesTraits additionally calculates the likelihood of the model based on the character states at the tips of the phylogeny. 
We can further compare different models of evolution by forcing certain parameters to be equal. We specifically considered the following 4 models:
\begin{enumerate}
\item	hsp90A and X are independent ($Q: q12 = q34, q21 = q43, q13 = q24, q31 = q42$; 4 parameters total)
\item hsp90A and X are mutually dependent (No parameter restrictions; 8 parameters total)
\item X depends on hsp90A but not vice versa ($Q: q12 = q34, q21 = q43$; 6 parameters total)
\item hsp90A depends on X but not vice versa ($Q: q13 = q24, q31 = q42$; 6 parameters total)
\end{enumerate}

\subsection{Identifying hsp90A-associated genes }
We used discrete from the BayesTraits package \cite{Pagel1994,Barker2005,Barker2007} to infer associations between hsp90A and other bacterial genes and between hsp90A and various organismal traits. We first tested for an evolutionary association with hsp90A by comparing model 1 to model 2 above with a likelihood ratio test (LRT), as previously described [28]. In our likelihood-ratio tests, the 2Log(LR) approximates a $\mathrm{X^2}$ test statistic for rejecting the independent model as a null hypothesis, and is calculated as twice the difference of the log-likelihoods of a co-evolutionary model and a model of evolutionary independence. The set of genes for which model 2 is preferred (i.e., model 1 is rejected as a null hypothesis) have an evolutionary association with hsp90A. Since different runs of the BayesTraits maximum likelihood method can potentially produce different parameter values, we repeated this procedure 100 times, each potentially resulting in a different gene set. We validated that these sets are similar and the choice of gene set does not substantially affect downstream analysis (Supporting Text). Any gene that was found to be associated with hsp90A in at least 90 runs was defined as hsp90A-associated gene. See Supporting Text for more details.  

\subsection{Reversible-jump Markov chain Monte Carlo analysis}
We selected 10 genes at random from the hsp90A-associated set and used the BayesTraits implementation of reversible-jump Markov chain Monte Carlo to estimate the rate parameters for their gain and loss in concert with hsp90A [63]. For each of these 10 genes, we used an exponential rate prior with mean and variance equal to 30, and ran the chain for 150 million iterations while sampling every 100 iterations. We discarded the first 75 million iterations as burn-in and used the remaining iterations as a posterior distribution of rate parameter estimates. We used Tracer v1.5 \cite{Rambaut2008} and previously described criteria to evaluate chain convergence in this remaining sample [65]. For each rate, we used the median of its posterior distribution in this sample as a point estimate.
\subsection{Co-evolutionary model selection}
To provide an accurate description of the co-evolutionary dynamics of hsp90A-associated genes, we further applied BayesTraits to these genes, estimating the likelihood of each of the four models described above. We identified the best fit model for each gene using the Akaike Information Criterion (AIC) \cite{Akaike1974}, taking into account both the likelihood score and the number of parameters in each model. We again repeated this procedure 100 times and classified a gene into a specific co-evolutionary model only if it fit this same model in at least 90 runs (see Supporting Text for more details). This two stage scheme, first identifying associated genes and then selecting a model that best describes their evolutionary relationship with hsp90A, provides a more stringent test of co-evolution and supports a simple approach for multiple testing correction. 
\subsection{Prediction of Hsp90A clients in bacteria}
We used BayesTraits-derived evolutionary transition rates under the fully unrestricted model to estimate residence times in specific states (for instance, the proportion of time spent by bacteria in a state where both hsp90A and some other gene are present, vs. the time when only the other gene is present) under steady state dynamics. For a given gene, the probability of being in one of the four states, A: (hsp90A absent, Gene absent), B: (hsp90A present, Gene absent), C: (hsp90A absent, Gene present), D: (hsp90A present, Gene present) at a very small increment of time$\Delta t$ after time t is given by:
\begin{eqnarray*}
 A_{t+\Delta t} = A_{t} - (q12+q13)*A_{t}\Delta t  + (q21*B_{t})\Delta t + (q31*C_{t})\Delta t + (0*D_{t})\Delta t \\
 A_{t+\Delta t} = B_{t} + (q12*A_{t})\Delta t - (q21+ q24)*B_{t}\Delta t + (0*C_{t})\Delta t + (q42*D_{t})\Delta t \\
 A_{t+\Delta t} = C_{t}+ (q13*A_{t})\Delta t + (0*B_{t})\Delta t - (q31+q34)*C_{t}\Delta t + (q43*D_{t})\Delta t \\
 A_{t+\Delta t} = D_{t} + (0*A_{t})\Delta t + (q24*B_{t})\Delta t + (q34*C_{t})\Delta t - (q43+q42)*D_{t}\Delta t \\
\end{eqnarray*}

We can differentiate this to obtain the instantaneous change in each probability:
\begin{eqnarray*}
dA/dt = (q12+q13)*A_{0}  + (q21*B_{0}) + (q31*C_{0}) + (0*D_{0}) \\
dB/dt = (q12*A_{0}) - (q21+ q24)*B_{0} + (0*C_{0}) + (q42*D_{0}) \\
dC/dt = (q13*A_{0}) + (0*B_{0}) - (q31+q34)*C_{0} + (q43*D_{0}) \\
dD/dt = (0*A_{0}) + (q24*B_{0}) + (q34*C_{0}) - (q43+q42)*D_{0} \\
\end{eqnarray*}
At steady state $dA/dt = 0, dB/dt = 0$, etc., and therefore:
\begin{eqnarray*}
0 = -(q12+q13)A + q21B + q31C + 0 \\
0 = q12A - (q21+ q24)B + 0 + q42D \\ 
0 = q13A + 0 - (q31+q34)C + q43D \\
0 = 0 + q24B + q34C - (q42+q43)D \\
\end{eqnarray*}

This set of linear equations can be solved for A, B, C, and D, with the requirement that A+B+C+D=1. We replaced 0 rates with the smallest nonzero rate in the model multiplied by 0.001 to allow transitions between all states. The positive nonzero solution for A, B, C, and D can then be conceived as the expected residence times along some arbitrary bacterial lineage. We used these residence times to estimate a Putative Client Index, PCI, denoting the normalized residence time in state C:
\begin{equation*}
PCI(gene) = \frac{C}{(C+D)(A+C)} = \frac{Pr(gene=present \cap hsp90A=absent)}{Pr(gene=present)*Pr(hsp90A=absent)}
\end{equation*}

Notably, if Hsp90A and the genes product have no client relationship, the proportion of time spent in state C is expected to be equal to $(C+D) * (A+C)$. Small values of PCI therefore indicate that a gene is observed less frequently than expected without hsp90A. Since no obvious threshold value can be defined, we considered the 20 genes with the lowest PCI values as putative clients (Figure 3 and Table 2; Methods). To account for variation in rates between BayesTraits runs we repeated this procedure 100 times and defined as putative clients those that were identified as clients in at least 90 of these runs (see Supporting Text). PCI scores shown in Table 2 and Figure 3 are averages across all runs. 
\subsection{Functional enrichment analysis}
We used a hypergeometric test to assess whether each KEGG functional annotation is overrepresented in the various Hsp90-associated gene classes. As a background set in each case we used the entire set of genes analyzed. Any annotation present in less than 4 copies in the background set was not considered. We accepted enrichments at a 5\% FDR.
\subsection{E. coli strains and growth assays }
Escherichia coli K-12 strains and plasmids used in this study are listed in Table S2. Cells were grown in tryptone broth (TB; 1\% tryptone and 0.5\% NaCl) and when necessary supplemented with ampicillin, chloramphenicol and/or kanamycin at final concentrations of 100, 35 and 50 $\mathrm{\mu}$g/ml, respectively. Overnight cultures, grown at 30C, were diluted 1:100 and grown at 34C for about 4 h, to an OD600 of 0.45-0.5. All expression constructs for YFP and CFP fusions were constructed as described previously [19,66,67] . Induction levels for protein expression were 1 $\mathrm{\mu}$M IPTG (pHL24, pHL35, pVS129 and pVS132), 20 $\mathrm{\mu}$M IPTG (pVS64 and pVS99), 25 $\mathrm{\mu}$M IPTG (pDK36, pDK90 and pDK91), 50 $\mathrm{\mu}$M IPTG (pDK19 and pVS18), 0.005\% arabinose (pHL13, pVS108 and pVS109) and 0.01\% arabinose (pHL52, pHL70, pDK14, pDK29, pDK30 and pDK49). Cells were harvested by centrifugation (4,000 rpm, 5 min), washed once with tethering buffer (10mM potassium phosphate, 0.1 mM EDTA, 1 mM L-methionine, 67 mM sodium chloride, 10 mM sodium lactate, pH 7) and resuspended in 10 mL tethering buffer prior to FRET measurements. 
TB soft agar plates were prepared by supplementing TB with 0.3\% agar (Applichem) and when necessary with 100 g/mL ampicillin and 1 $\mathrm{\mu}$M IPTG. Equal amounts of cells from different overnight cultures, adjusted depending on their optical density to the equivalent of 2.5 $\mathrm{\mu}$L of culture with OD600 of 2.0, were inoculated and allowed to spread at indicated temperatures for indicated times. Following incubation, photographs of plates were taken with a Canon EOS 300D (DS6041) camera. Images were analyzed with ImageJ (Wayne Rasband, NIH, http://rsb.info.nih.gov/ij/) to determine the diameter of the rings of spreading colonies.
For analysis of motility at different growth stages (indicated by OD600 value), percentages of motile cells were estimated from the microscopy movies of swimming cells. The experiment was performed with the RP437 strain, which is non-motile above 37C. Cells were grown overnight in TB medium at 37C to completely inhibit their motility. After dilution in fresh TB medium to OD600 0.01, cells were grown at 34C for measurements.
\subsection{Fluorescence imaging}
For microscopy, cells were taken from the soft-agar plates and applied to a thin agarose pad (1\% agarose in tethering buffer). Fluorescence imaging was performed on a Zeiss AxioImager microscope equipped with an ORCA AG CCD camera (Hamamatsu), a 100? NA 1.45 objective, and HE YFP (Excitation BP 500/25; Emission BP 535/30) and HE CFP  (Excitation BP 436/25; Emission BP 480/40) filter sets. Each imaging experiment was performed in duplicate on independent cultures. All images were acquired under identical conditions. Images were subsequently analysed using ImageJ software. 
\subsection{Acceptor photobleaching FRET measurement}
FRET measurements by acceptor photobleaching were performed on a custom-modified Zeiss Axiovert 200 microscope as described before [66]. Briefly, cells expressing YFP and CFP fusions of interest were concentrated about tenfold by centrifugation, resuspended in tethering buffer and applied to a thin agarose pad (1\% agarose in tethering buffer). Excitation light from a 75 XBO lamp, attenuated by a ND60 (0.2) neutral-density filter, passed through a band-pass (BP) 436/20 filter and a 495DCSP dichroic mirror and was reflected on the specimen by a Z440/532 dual-band beamsplitter (transmission 465-500 and 550-640 nm; reflection 425-445 and 532 nm). Bleaching of YFP was accomplished by a 20 sec illumination with a 532 nm diode laser (Rapp OptoElectronic), reflected by the 495DCSP dichroic mirror into the light path. Emission from the field of view, which was narrowed with a diaphragm to the area bleached by the laser, passed through a BP 485/40 filter onto a H7421-40 photon counter (Hamamatsu). For each measurement point, photons were counted over 0.5 s using a counter function of the PCI-6034E board, controlled by a custom-written LabView 7.1 program (both from National Instruments). CFP emission was recorded before and after bleaching of YFP, and FRET was calculated as the CFP signal increase divided by the total signal after bleaching. $\Delta flhC$ strains were used to define direct interactions between HtpG and flagellar and chemotaxis components. In this background expression of endogenous flagellar and chemotaxis genes is inhibited, thus eliminating indirect interactions that may result from concomitant binding of HtpG and tested protein to a third flagellar or chemotaxis protein. 

\section{Results}

\subsection{Hsp90 paralogs in bacteria }
Our method for inferring the function of bacterial Hsp90 is based on analysis of its distribution across the bacterial phylogeny. However, this analysis is complicated by the existence of multiple ancient Hsp90 paralogs in bacteria. These paralogs may be older than existing phyla in bacteria [11,12], and may have evolved distinct functions on this enormous time scale. To address this issue and to identify each paralog, we first clustered bacterial Hsp90s by sequence identity. We identified 897 bacterial Hsp90 protein sequences in the KEGG database [24] and built a neighbor-joining gene tree of bacterial Hsp90s (Figure E1A-B). We observed two well-supported long-branching clades as well as several less confident divisions in the tree (Figure E1B). These two long-branching clades contain sequences corresponding to the hsp90B and hsp90C paralogs that were described previously [11,12]. All other branches correspond to hsp90A [11], which is the largest of the Hsp90 families in bacteria (Figure E1C, Supporting Text). Notably, hsp90A is the lineage out of which all eukaryotic Hsp90s (excluding mitochondrial and chloroplast Hsp90s) are derived. Moreover, the E. coli gene htpG belongs to the hsp90A family, and its gene product is the best-studied bacterial Hsp90 protein. For these reasons, we restricted our analysis to hsp90A.  

\subsection{Genome-wide detection of genes co-evolving with hsp90A}
We set out to identify orthologous groups whose presence and absence profiles across bacterial species are associated with the presence and absence profile of hsp90A. To avoid spurious associations, any such comparative analysis must go beyond a nave comparison of presence/absence patterns across genomes and incorporate phylogenetic information [25]. To this end, we used BayesTraits [2628], a computational framework for phylogenetic analysis of character evolution. Given the states (e.g., presence/absence) of two characters across some set of species and a phylogenetic tree relating these species, BayesTraits evaluates the likelihood of various evolutionary models throughout the tree. This approach can be utilized, for example, to determine whether these two characters evolve in a mutually dependent vs. an independent fashion. 
	We used BayesTraits to detect associations between hsp90A and 4646 other orthologous groups in bacteria (which hereafter we shall refer to as genes for simplicity). We used the tree constructed by Ciccarelli et al. [29] as a model phylogeny (Figure 1). In this initial analysis, we tested for any kind of dependency between hsp90A and other genes, and did not make specific assumptions about the nature of the relationship between hsp90A and the genes in question [28]. Specifically, we compared a model in which the rate of gain and loss of a given gene is independent of the rate of gain and loss of hsp90A (independent evolution) vs. a model in which the rate of gain and loss of this gene is affected by the presence or absence of hsp90A or vice-versa (co-evolution). 
	In total, we found 327 genes that co-evolve with hsp90A. We will refer to this set as hsp90A-associated genes. These hsp90A-associated genes were significantly enriched for annotations related to the flagellum and to bacterial secretion systems (Table 1). Moreover, out of the 16 hsp90A-associated bacterial secretion genes, 10 were part of the non-flagellar Type III secretion system, suggesting that hsp90A is associated specifically with this system rather than with secretion systems in general. Using a different and markedly more extensive phylogeny [30] provided similar results (see Supporting Text, Table S1), as did a pruned Ciccarelli tree without the species containing the hsp90B or hsp90C (see Supporting Text). 
\subsection{Characterization of co-evolutionary dynamics }
The associations of hsp90A with other genes identified above are agnostic to the specific nature of the dependency between hsp90A and the gene in question. For example, our initial analysis could not distinguish between a positive association (i.e. genes tend to be gained and lost together) and a negative association (i.e. genes tend not to co-occur in genomes). Similarly, this analysis did not distinguish between genes whose gains and losses are affected by the presence of hsp90A (but that do not themselves affect hsp90A evolution) and genes that exhibit mutually dependent dynamics with hsp90A. Without a quantitative estimate of the effects that hsp90A and its co-evolving partners have upon one another, inference of Hsp90A function and its relationship with other genes is challenging. 
To characterize the specific nature of the dependency between hsp90A and hsp90A-associated genes, we therefore examined rates of gain and loss inferred by BayesTraits. We focused on the two major non-overlapping hsp90A-associated functional categories, flagellar assembly and bacterial secretion. Considering, for example, fliI, a representative flagellar gene, we found that its gain and loss was strongly affected by the presence of hsp90A. Specifically, in the presence of hsp90A, fliI was often gained and rarely lost, whereas it was rarely gained and often lost when hsp90A is absent (Figure 2A). This pattern was common to all hsp90A-associated flagellar genes (Figures 2C, S2), suggesting a positive association between hsp90A and flagellar genes throughout evolution. In contrast, the co-evolutionary relationship between hsp90A and yscN, a representative nonflagellar type III secretion system gene, was markedly different, with yscN presence strongly affecting the gain and loss of hsp90A (Figure 2B). Specifically, the presence of yscN was associated with a large increase in the rates of gain and (even more dramatically) loss of hsp90A relative to these rates in its absence. Again, this pattern was common to all hsp90A-associated bacterial secretion genes (Figures 2D, S3, S4), suggesting a negative association between hsp90A and nonflagellar secretion genes throughout evolution. 
To further validate the fundamentally distinct co-evolutionary dynamics of these two groups of genes, we considered four different co-evolutionary models: (1) hsp90A and the gene in question are independent (null); (2) hsp90A and the gene in question are mutually dependent; (3) hsp90A is dependent on the gene in question but not vice versa, and (4) the gene in question is dependent upon hsp90A but not vice versa (Methods). We used the Akaike Information Criterion (AIC [31]) to determine which of these 4 models best fit the co-evolutionary dynamics of each hsp90A-associated gene. As expected, none of the hsp90A-associated genes fit the independent model. Of the 27 hsp90A-associated flagellar genes, 25 were classified as being dependent on hsp90A but not vice-versa (model 4). Of the 16 hsp90A-associated secretion system genes, 10 genes were classified as mutually dependent with hsp90A (model 2; 6 of which were Type III secretion system genes), whereas 6 were classified as affecting the evolution of hsp90A (model 3). Furthermore, considering all hsp90A-associated genes, we found that genes that best fit each of the evolutionary dependency models above (models 2, 3, and 4) were enriched for different functions (Table 1). Specifically, among genes dependent on hsp90A, flagellar motility was strongly enriched, whereas among genes mutually dependent on hsp90A, secretion system components were enriched. Taken together, these patterns suggest that flagellar genes and secretion system genes had markedly different regimes of co-evolution with hsp90A. 
\subsection{Prediction of Hsp90A clients}
Although many genes exhibited distinct patterns of co-evolution with hsp90A, these patterns could be the result of indirect evolutionary relationships rather than the outcome of a direct interaction with Hsp90A. We therefore aimed to predict specific genes that encode putative hsp90A clients. Our method is based on the assumption that strong, conserved clients should be heavily dependent on Hsp90A, and thus should be found only rarely in the absence of hsp90A throughout evolution. To estimate the expected frequency of each hsp90A-associated gene with and without hsp90A, we used the inferred BayesTraits rates to calculate the steady-state probabilities of each of the 4 possible presence/absence states (Methods). These probabilities represent the proportion of the time that some arbitrary bacterial lineage will spend in each of the presence/absence states throughout evolution. From these probabilities we calculated a Putative Client Index (PCI) for each hsp90A-associated gene to evaluate how often it was present without hsp90A throughout evolution, compared to a null expectation (see Methods). This index is close to zero for genes that were infrequently present without hsp90A and were hence likely to be Hsp90A clients. We defined the genes with the lowest PCI values as putative clients (Table 2; see also Supporting Text). 
\subsection{Novel and known functions of putative Hsp90 clients}
Consistent with our prior analysis, several flagellar genes behaved as potential clients (Table 2). In particular, our set of putative clients included several genes (fliH, fliI, fliN) whose products had been previously shown to physically interact with Hsp90A in E. coli [19]. The products of these genes are cytoplasmic components of the flagellar rotor and export apparatuses. In contrast, nonflagellar type III secretion genes were all absent from the list of potential clients. In fact, nonflagellar type III secretion system components were rated as some of the least likely clients by our index (Figure 3). This disparity in predicted client status mirrors the different evolutionary relationships of these complexes with hsp90A (Figure 2). 
Chaperone/proteases (e.g. ClpA and PpiD) also ranked high in our list of potential clients. Hsp90A is known to collaborate with other chaperone systems such as DnaK [14,32] but to date no obligate co-chaperones have been described. The identified chaperone/proteases may represent such co-chaperones or collaborating chaperone systems, since our index cannot discriminate between Hsp90 clients and Hsp90 co-chaperones (or other collaborating proteins). Alternatively, these observed associations could simply indicate that components of the cytoplasmic stress response are dependent upon Hsp90A.
We also found several unexpected putative clients, such as the 3-hydroxybutyryl-CoA dehydrogenase PaaH and the transcription termination factor Rho, which we predict to be the two strongest clients. Further study will be necessary to understand these associations and the underlying cause of the co-evolutionary association between these genes and hsp90A.
\subsection{Swimming motility and chemotaxis assays of Hsp90A-defective E. coli}
Our putative clients and the predicted chaperone role of Hsp90A in flagellar assembly are consistent with previous observations. Specifically, the deletion of E. coli hsp90A, also known as htpG, resulted in reduced surface swarming movement [33]. We also previously observed physical interactions between the HtpG protein and certain flagellar proteins [19]. Yet, these observations lacked a clear demonstration of client status or mechanism, and E. coli swarming is a complex behavior that depends on numerous factors in addition to flagellar function [34]. We therefore set out to test our hypothesis that Hsp90A is physiologically important for flagellar assembly and function and that flagellar components are indeed Hsp90A clients.
We examined the swimming motility phenotype of $\Delta htpG$ E. coli strains on soft-agar plates (Methods). In contrast to surface swarming, swimming is a less complex behavior, in which bacteria use functional flagella and chemotaxis components to swim from an inoculation point through agar pores, following nutrient gradients that are created by nutrient depletion within the colony. The soft-agar assay is routinely used to assay bacterial swimming motility and chemotaxis. To enhance our ability to detect differences between wild-type and $\Delta htpG$ cells, the assays were performed competitively. Competitive assays emphasize small differences between strains and reduce experimental error, thereby increasing the sensitivity of the assay. After mixing equal amounts of YFP-labeled WT and CFP-labeled $\Delta htpG$ strains, this mixture was inoculated in the center of a soft-agar plate and incubated at 34C for 8 hrs. We then counted cells of each strain in the plate center vs. the outer edge using fluorescence microscopy (Figure 4A). $\Delta htpG$ mutants migrated less efficiently to the plates outer edge relative to WT, confirming that they are partially deficient in their motility and/or chemotaxis (Figure 4B). This defect is apparently subtle, since little difference between WT and $\Delta htpG$ cells was observed in a non-competitive assay (Figure E5), but it could be revealed due to strong selection for cells with optimal motility and chemotaxis at the outer edge of the spreading bacterial population.  
We also tested the phenotype of the HtpG(E34A) mutant, which has reduced rates of ATP hydrolysis and is deficient in substrate refolding [14,35]. Since HtpG ATPase activity is necessary for release of clients, HtpG(E34A) is less efficient at releasing clients [3638]. Indeed, this mutant showed stronger motility/chemotaxis defects than the $\Delta htpG$ strain (Figure E5), presumably due to sequestration of its client proteins. We therefore employed the HtpG(E34A) mutant in all subsequent assays as a more sensitive test of HtpG involvement. Taken together, our observations suggest that the motility defect may be due to the improper function or sequestration of HtpG clients. 
\subsection{FRET observation of HtpG interactions with flagellar motor components}
To further investigate the in vivo interaction of HtpG with flagellar components, we used htpG-yfp and htpG(E34A)-yfp constructs expressed in WT cells to perform acceptor photobleaching FRET between HtpG and FliN-CFP over an E. coli growth curve. Motility of E. coli is known to increase at the transition from the early exponential to post-exponential phase of growth [39], and this experimental design enabled us to examine the HtpG-FliN interaction in the context of the flagellar assembly process. If HtpG is indeed involved in the assembly process of these structures, the interaction of HtpG with FliN should correspond temporally to the timing of flagellar assembly. Indeed, we found that the interaction with FliN peaked at OD600 = 0.2 (Figure 5A) and correlated well with the onset of cell motility in wild-type cells (Figure 5B). Moreover, the interaction of HtpG(E34A) with FliN was stronger and delayed compared to the binding of wild-type HtpG. This is consistent with the delayed release of clients by HtpG(E34A). Correspondingly, the onset of motility was delayed in cells expressing HtpG(E34A) (Figure 5B). These findings suggest that HtpGs role in motility derives from a direct involvement in flagellar complex assembly. 
Given that both bacterial and eukaryotic Hsp90s are known to collaborate with Hsp70 in refolding proteins [14,4042], we considered the possibility that this was also the case for bacterial flagellar assembly. We previously showed that some flagellar motor components interact with DnaK, the E. coli Hsp70 homolog [19]. Therefore, we repeated the FRET experiments testing for interactions between HtpG or HtpG(E34A) and FliN in a $\Delta cbpA\Delta dnaJ$ background. CbpA and DnaJ are DnaK co-chaperones and are essential for DnaKdependent refolding activity [14]. DnaK should not be able to pass substrates to HtpG in this mutant background. Indeed, we found that FRET interactions with FliN disappear for both HtpG proteins in this background (Figure E6A), suggesting that DnaK-dependent remodeling precedes HtpG action in flagellar complex assembly. 

\subsection{FRET observation of HtpG interactions with chemoreceptor components}
Since a recent high-throughput assay showed kinases to be overrepresented among eukaryotic Hsp90 clients [43,44], we next examined whether the HtpG-dependent defects in chemotaxis may also be due to defective chemoreceptor kinase activity. Although no chemotaxis proteins were found in our list of the strongest putative clients, we did observe a significant enrichment of these components in the hsp90A-associated set (Table 1). We thus tested interactions between six chemoreceptor cluster components and HtpG(E34A) using, as before, acceptor photobleaching FRET (Table S4). We observed a strong interaction of HtpG(E34A) with the chemoreceptor kinase CheA. Our results suggest that the interactions between FliN and HtpG and CheA and HtpG are direct and do not depend on other flagellar or chemotaxis proteins, since these interactions are robust to deletion of flhC, which ablates expression of all endogenous flagellar and chemotaxis genes (Table S4) [19]. Moreover, the CheA dimerization domain was required for association with HtpG, supporting the hypothesis that HtpG aids oligomerization of its clients [17,45]. Testing HtpG interactions with other chemotaxis proteins of E. coli revealed an additional strong interaction with the dimeric phosphatase CheZ but not with other proteins (Table S4).  
We again examined the temporal dynamics of these interactions. Due to the hierarchical order of flagellar and chemotaxis gene expression [39,46], the assembly of chemoreceptor clusters is delayed compared to the assembly of flagellar motors as non-motile cells transition into motile cells. Indeed, the interaction of HtpG with CheA peaked at OD600 = 0.3, after the FliN peak (Figure 5A). Just as for FliN, the interaction of HtpG(E34A) with CheA was stronger and delayed compared to wild-type HtpG, and the HtpG-CheA interaction disappeared in a $\Delta cbpA\Delta dnaJ$ background (Figure E6B). Collectively, these findings suggest that HtpG plays an important role in the assembly of both the flagellar motor and chemoreceptor clusters through separate client interactions.
\subsection{Association of hsp90A with life history traits in Bacteria}
Given the role of HtpG in chaperoning proteins that mediate interactions with the environment, and the known role of eukaryotic Hsp90 in phenotypic robustness, we finally examined whether hsp90A directly co-evolved with certain bacterial organismal traits. We considered several organismal traits, including aerobism, thermophilicity, halophilicity, the ability to form endospores, pathogenicity, motility, and habitat preferences (see Methods). We used BayesTraits and the Ciccarelli tree to identify traits that co-evolve with hsp90A. Out of the 11 analyzed traits, 4 exhibited significant associations with hsp90A (p < 0.05; Table S5), with the strongest association observed between hsp90A and the capacity to inhabit multiple habitats. Moreover, examining the gain and loss rates obtained, we found that hsp90A is gained and lost at significantly higher rates in organisms that inhabit multiple habitats (with no gains inferred in single habitat organisms), suggesting that a preference for multiple habitats imposes a different selection regime on hsp90A (Figure 6). We also tested whether the co-evolutionary dependency between hsp90A and multiple-habitat preferences was unidirectional, as we observed for some hsp90A-associated genes. Comparing the four co-evolutionary models described above and applying AIC to identify the best-fitting model, we found that hsp90A gain and loss depended on habitat preference, but not vice versa. This observation suggests that in organisms inhabiting multiple environments hsp90A is subjected to dynamically shifting selective pressures, potentially alternating between selection for and against hsp90A. 
\section{Discussion}
We set out to discover Hsp90 functions conserved throughout the bacterial tree of life. We found that hsp90A, the most common paralog of bacterial Hsp90, bore strong signatures of co-evolution with several hundred genes and with specific life history traits, shedding light on its function and impact on evolutionary history. Most notably, we found that hsp90A co-evolved with membrane protein complexes such as flagella and other Type III secretion (T3S) systems. Our results suggest that Hsp90s role in sensing and responding to environmental stimuli is conserved between bacteria and eukaryotes.
Similar to verified eukaryotic Hsp90 clients [5], our predicted putative Hsp90A clients were a diverse group of proteins (e.g. the flagella protein FliN, the chaperone ClpA, and the ribosomal protein RluB; see Table 2) that tended to belong to specific functional categories (e.g. flagellar proteins, chaperones, and ribosomal components). As our methods can only infer associations between genes that are frequently gained and lost, we may substantially underestimate the number of hsp90A-associated genes and clients. However, the non-essentiality and frequent loss of hsp90A throughout bacterial diversity argues that genes not captured in our analysis (since they are not frequently gained and lost) are unlikely to be strongly dependent on the chaperone. 
The subtlety of the bacterial Hsp90 mutant phenotypes that we (and others) report implies that Hsp90s role in cellular physiology has diverged between eukaryotes and prokaryotes [17,45,47]. In other words, either essential pieces of cellular physiology changed, or Hsp90 function changed. We favor the first hypothesis, because Hsp90 is well-conserved among bacteria, archaea, and humans at the sequence level [13], and retains a similar quaternary structure [48] and biochemical activity [15,37,44]. In contrast, bacterial and archaeal cells differ significantly from eukaryotic cells. Eukaryotic cells have higher cell compartmentalization, longer and multifunctional proteins with multiple domains [49], and increased protein interactome complexity [50]. Together with the existence of many eukaryotic Hsp90 co-chaperones, all these features may contribute to the greater essentiality of Hsp90 in eukaryotes.
The dependence of HtpG-client interactions upon the DnaK chaperone system, as observed by us and by others [14,15], argues that Hsp90A is well-integrated with other chaperone systems. Our putative clients included ClpA, the substrate adaptor for the ClpAP/ClpAXP chaperone/protease complexes, and PpiD, a periplasmic chaperone [51]. Like HtpG, PpiD is necessary for optimal swarming motility [33], suggesting that it may participate in flagellar assembly. We speculate that these proteins act as Hsp90A co-chaperones in some bacteria; alternatively, their dependence on Hsp90A may represent an example of collaborating chaperone systems. 
The best-characterized Hsp90 client in bacteria is the structural ribosomal protein L2 [15,18], which is near-universally conserved throughout life (and hence not detectable by our method). In addition to L2, other ribosomal proteins were found to interact with HtpG in large-scale proteomics analyses. In agreement with these observations, we found the ribosomal proteins RlmE and RluB among the predicted hsp90A clients. 
Although these chaperone and ribosomal proteins were predicted to be stronger clients than flagellar proteins, our experimental validation focused on the latter as their client status was suggested by previous observations [19,33]. We present four lines of evidence for HtpG client status for the flagellar protein FliN and the chemoreceptor kinase CheA, including direct interactions with HtpG, physiologically relevant timing of HtpG-FliN/CheA interactions, phenotypic consequences of reduced HtpG function in CheA/FliN-dependent traits, and dependence of CheA/FliN interactions with HtpG upon the Hsp40-Hsp70 pathway.  The identification of FliN and CheA as HtpG clients is consistent with the hypothesis that bacterial Hsp90 facilitates the assembly of large membrane-associated protein complexes [17,45]. 
Curiously, whereas the flagellar T3S system contained Hsp90A clients, the nonflagellar T3S system is predicted to have an antagonistic relationship with Hsp90A. Nonflagellar T3S systems and the flagellar T3S systems are closely related (NF-T3SS and F-T3SS) [52,53]. 9 NF-T3SS components are directly homologous to flagellar components, of which 8 were found to co-evolve with hsp90A in our analysis. Yet, these 8 genes are predicted to co-evolve antagonistically with hsp90A (Figure 3), whereas their flagellar homologs are mostly predicted to be clients (for instance, the fliI and yscN genes shown in Figure 2 are homologous). This result suggests that some relationship with Hsp90A is conserved between the two T3S systems, but with apparently opposite effects in each system. This result may reflect the fact that each of these systems is an adaptation to different ecological challenges. Specifically, we have shown that Hsp90A is important for flagella-enabled motility and chemotaxis in E. coli. This mode of motility is strongly adaptive in certain physical environments [34,54,55], and thus Hsp90A is likely to be associated with fitness in these environments through flagellar assembly. The presence of NF-T3SS is likewise an adaptation to certain biotic environments [55,56]. Our observation that organisms inhabiting multiple habitats experience fluctuating selection for hsp90A is also consistent with competing selection pressures. Representative genes of these homologous T3S families were not significantly associated with habitat preferences, arguing that hsp90As association with habitat preferences is not a byproduct of associations with T3S systems. We suggest that these two T3S systems constitute a link between Hsp90A and phenotypic robustness across different environments.
Inferring function from evolutionary associations has some caveats. For instance, F-T3S systems can be found in genomes that lack hsp90A. If F-T3S systems include Hsp90A clients, then what may render Hsp90A-dependent stabilization dispensable in some bacteria? Experimental validation will be necessary to answer such questions, and to distinguish true client relationships from indirect co-evolutionary associations. As discussed before, our method is subject to gene set bias, in that only genes that are gained and/or lost frequently will have enough statistical power to reject the null hypothesis. Similarly, as our method assumes that relationships are maintained throughout the analyzed phylogeny, we cannot reliably detect genes that are associated with hsp90A in some organisms but not in others.
Although much work remains to articulate the precise mechanistic relationships between hsp90A and its co-evolving genes, our results highlight the tremendous potential of evolutionary inference for guiding experimental research. More generally, our study provides a successful example of how evolutionary perspectives and phylogenetic analyses can inform and advance the study of complex biological systems and the inference of elusive biological functions. 


\section{Acknowledgments}
We thank Joe Felsenstein for extensive methodological guidance. We thank Aviv Regev for valuable discussions and advice at the initiation of this work. We thank Matthias Mayer for providing strains and for valuable discussions. We thank Evgeni Sokurenko, Willie Swanson, Olivier Genest, and members of the Queitsch, Sourjik, and Borenstein laboratories for helpful discussions. We thank Andrew Meade and Mark Pagel for help with the BayesTraits software.

\chapter{Evolutionary assembly patterns of prokaryotic genomes}

A version of this chapter is under review for publication, and is available at:

http://biorxiv.org/content/early/2015/09/27/027649.

Supporting figures and tables can be found in Appendix G.

\section{Abstract}Evolutionary innovation must occur in the context of some genomic background, which limits available evolutionary paths. For example, protein evolution by sequence substitution is constrained by epistasis between residues. In prokaryotes, evolutionary innovation frequently happens by macrogenomic events such as horizontal gene transfer (HGT). Previous work has suggested that HGT can be influenced by ancestral genomic content, yet the extent of such gene-level constraints has not yet been systematically characterized. Here, we evaluated the evolutionary impact of such constraints in prokaryotes, using probabilistic ancestral reconstructions from 634 extant prokaryotic genomes and a novel framework for detecting evolutionary constraints on HGT events. We identified 8,228 directional dependencies between genes, and demonstrated that many such dependencies reflect known functional relationships, including, for example, evolutionary dependencies of the photosynthetic enzyme RuBisCO. Modeling all dependencies as a network, we adapted an approach from graph theory to establish chronological precedence in the acquisition of different genomic functions. Specifically, we demonstrated that specific functions tend to be gained sequentially, suggesting that evolution in prokaryotes is governed by functional assembly patterns. Finally, we showed that these dependencies are universal rather than clade-specific and are often sufficient for predicting whether or not a given ancestral genome will acquire specific genes. Combined, our results indicate that evolutionary innovation via HGT is profoundly constrained by epistasis and historical contingency, similar to the evolution of proteins and phenotypic characters, and suggest that the emergence of specific metabolic and pathological phenotypes in prokaryotes can be predictable from current genomes.\section{Introduction}
A fundamental question in evolutionary biology is how present circumstances affect future adaptation and phenotypic change (Gould and Lewontin 1979). Studies of specific proteins, for example, indicate that epistasis between sequence residues limits accessible evolutionary trajectories and thereby renders certain adaptive paths more likely than others  (Weinreich et al. 2006; Gong et al. 2013; de Visser and Krug 2014; Harms and Thornton 2014). Similarly, both phenotypic characters (Ord and Summers 2015) and specific genetic adaptations (Christin et al. 2015; Conte et al. 2012) show strong evidence of parallel evolution rather than convergent evolution. That is, a given adaptation is more likely to repeat in closely related organisms than in distantly related ones. This inverse relationship between the repeatability of evolution and taxonomic distance implies a strong effect of lineage-specific contingency on evolution, also potentially mediated by epistasis (Orr 2005).Such observations suggest that genetic adaptation is often highly constrained and that the present state of an evolving system can impact future evolution. Yet, the studies above are limited to small datasets and specific genetic pathways, and a more principled understanding of the rules by which future evolutionary trajectories are governed by the present state of the system is still lacking. For example, it is not known whether such adaptive constraints are a feature of genome-scale evolution or whether they are limited to finer scales. Moreover, the mechanisms that underlie observed constraints are often completely unknown. Addressing these questions is clearly valuable for obtaining a more complete theory of evolutionary biology, but more pressingly, is essential for tackling a variety of practical concerns including our ability to combat evolving infectious diseases or engineer complex biological systems.Here, we address this challenge by analyzing horizontal gene transfer (HGT) in prokaryotes. HGT is an ideal system to systematically study genome-wide evolutionary constraints because it involves gene-level innovation, occurs at very high rates relative to sequence substitution \cite{Nowell2014, Puigbo2014}, and is a principal source of evolutionary novelty in prokaryotes (Gogarten et al. 2002; Jain et al. 2003; Lerat et al. 2005; Puigb et al. 2014). Clearly, many or most acquired genes are rapidly lost due to fitness costs (van Passel et al. 2008; Baltrus 2013; Soucy et al. 2015), indicating that genes retained in the long term are likely to provide a selective advantage. Moreover, not all genes are equally transferrable (Jain et al. 1999; Sorek et al. 2007; Cohen et al. 2011), and not all species are equally receptive to the same genes (Smillie et al. 2011; Soucy et al. 2015). However, differences in HGT among species have been attributed not only to ecology (Smillie et al. 2011) or to phylogenetic constraints (Nowell et al. 2014; Popa et al. 2011), but also to interactions with the host genome (Jain et al. 1999; Cohen et al. 2011; Popa et al. 2011). Indeed, studies involving single genes or single species support the influence of genome content on the acquisition and retention of transferred genes (Pal et al. 2005; Iwasaki and Takagi 2009; Chen et al. 2011; Press et al. 2013; Sorek et al. 2007; Johnson and Grossman 2014). For example, it has been demonstrated that the presence of specific genes facilitates integration of others into genetic networks (Chen et al. 2011), and that genes are more commonly gained in genomes already containing metabolic genes in the same pathway (Pal et al. 2005; Iwasaki and Takagi 2009). However, to date, a systematic, large-scale analysis of such dependencies has not been presented. In this paper, we therefore set out to characterize a comprehensive collection of genome-wide HGT-based dependencies among prokaryotic genes, analyze the obtained set of epistatic interactions, and identify patterns in the evolution of prokaryote genomes. 

\section{Methods}All mathematical operations and statistical analyses were performed in R 2.15.3 (R Core Team 2015). Probabilistic ancestral reconstructions were obtained using the gainLoss program (Cohen and Pupko 2010). Phylogenetic simulations and plots were performed with the APE library (Paradis et al. 2004). Network analyses and algorithms were implemented using either the igraph (Csardi and Nepusz 2006) or NetworkX (Hagberg et al. 2013) libraries, and visualized using Cytoscape v3.1.1 (Shannon et al. 2003). \subsection{Phylogenies}We used a pre-computed phylogenetic tree (Dehal et al. 2010) as a model of bacterial evolution. We mapped all extant organisms in this tree to organisms in the KEGG database by their NCBI genome identifiers, and pruned all tips that did not directly and uniquely map to KEGG. This yielded a phylogenetic tree connecting 634 prokaryotic species. For analyses involving subtrees of this phylogenetic tree, we used iTOL (Letunic and Bork 2011) to extract subtrees.\subsection{Inferring phylogenetic histories for genes}We used the gainLoss v1.266 software (Cohen and Pupko 2010), a set of presence/absence patterns of orthologous genes from KEGG (Kanehisa et al. 2012), and the phylogenetic tree described above to infer 1) the probabilities of presence and absence of genes at internal nodes of the tree, 2) gain and loss rates of each gene, and 3) tree branch lengths within a single model. Specifically, in running gainLoss, we assumed a stationary evolutionary process, with gene gain and loss rates for each gene modeled as a mixture of three rates drawn from gamma distributions defined based on overall initial presence/absence patterns. A complete list of parameters used for gainLoss runs is given in the Supplemental Text and as Supplemental File S2. The gainLoss log file for the principal run on the full tree is also included as Supplemental File S3. Based on these models, we obtained a probabilistic ancestral reconstruction based on stochastic mapping for each of 5801 genes that were present in at least one species and absent in at least one species, and filtered out genes that were found to be gained less than twice throughout the tree, yielding 5031 genes which we further analyzed. \subsection{Inferring gains and presence of genes on branches.}To focus on gain events with strong support and where the gained gene is retained (rather than gain events where the gene is subsequently lost along the same branch), we used a simple model for computing the probability of different evolutionary gain/loss scenarios based on gainLoss ancestral reconstructions rather than directly using gainLoss gain inferences (Supplemental Text). Specifically, we assumed that unobserved gains and losses are not relevant, and that evolutionary scenarios are defined by the states at the ancestor and descendant nodes of each branch (regardless of branch length). With these assumptions, we used the probabilities of presence and absence of each of 5031 genes at each node and tip on the tree to compute the probability of each branch undergoing each scenario: 1) gain (absent in ancestor and present in descendant), 2) presence (present in both ancestor and descendant), and 3) loss (present in ancestor and absent in descendant; Supplemental Text). For a gene X on a branch with ancestor A and descendant B, we assume: \begin{eqnarray*}
Pr(X present on branch) = Pr(X present in A \cap X present in B) = \\
Pr(X present in A) * Pr(X present in B) \\Pr(X gained on branch) = Pr(X absent in A \cap X present in B) = \\
Pr(X absent in A) * Pr(X present in B) \\Pr(X lost on branch) = Pr(X present in A \cap X absent in B) = \\
Pr(X present in A) * Pr(X absent in B) \\
\end{eqnarray*}
Note again that these probability estimates are distinct from those obtained by using the gainLoss continuous-time Markov chain on the same ancestral reconstruction, which consider also hypothetical gains that are not retained and are thus not relevant to our analysis (Supplemental Text). 	\subsection{Robustness analysis of reconstruction method}We used a maximum-parsimony reconstruction as inferred by gainLoss to benchmark the accuracy of the gainLoss reconstruction by stochastic mapping. In this analysis, only internal node reconstructions were considered, as tip reconstructions (for which the states are known) are not informative about algorithm performance. Since the maximum-parsimony reconstruction is binary (presence/absence) and the stochastic mapping reconstruction is probabilistic, for purposes of comparison we rounded the probabilities of the stochastic mapping reconstruction to obtain a presence/absence reconstruction (i.e., a probability >0.5 denotes presence and $\leq$ 0.5 denotes absence). We computed the agreement between the two reconstructions as the percentage of internal node reconstructions that agree on the state of the gene.\subsection{Comparison of analyzed gains to reconciliation-based HGT inference.}We compared gains inferred by our method for several genes central to the PGCE network to gain events reported in a searchable database of horizontally acquired genes inferred by a sequence-based reconciliation method (Jeong et al. 2015). To this end, we classified all branches supporting a gain event for each of these genes with >50\% probability by our method as true gains. We next searched the reconciliation database (all queries performed between January 15th and February 20th, 2016) for each gene, identifying orthologous genes across 2,472 genomes that exhibit HGT according to reconciliation (excluding events that occurred on branches without descendants). We manually compared descendants of the remaining events from our method with the genomes experiencing gene acquisition in the reconciliation dataset to assess overlap between these two methods (see Supplemental Text). \subsection{Quantifying PGCEs}We defined a `pair of genes with conjugated evolution' (PGCE) as a gene pair $(i, j)$ for which the presence of one gene $i$ encourages the gain of the other, $j$. Considering these genes as phylogenetic characters, we therefore aim to detect pairs for which gain state transitions for character j are enriched on branches where character i remains in the present state. This problem is related to previous methods for detecting coevolution or correlation between phylogenetic characters (Maddison 1990; Huelsenbeck et al. 2003; Cohen et al. 2012). Given N branches and k genes, there are 2 $N$ X $k$ matrices, $P$ and $G$, describing the probabilities, respectively, of presence and gain of each gene along each branch (using our model for estimating gains described above). The test statistic for a dependency between each gene pair $(i, j)$ is the expected number of branches where the gain of gene $j$ occurs, while conditioning on the presence of gene $i$ (cell $C_ij$ in a $k$ x $k$ matrix $C$). Counting transitions of one character (gene j gain) given some state of another character (gene i presence) yields a standard test statistic for testing correlated evolution of binary characters on phylogenies (Maddison 1990). To compute C across N branches, we sum the conditional probabilities of the gain of gene j in the presence of gene i across the tree, i.e. the products of the two $N$ x $k$ matrices, $P$ (presence) and $G$ (gain), for each gene pair:\begin{equation*}
C_{ij} = \sum\limits{n=1}^N G_{nj} P_{ni} 
\end{equation*}
Entries in C which are significantly larger than a null expectation of gains represent PGCEs between the row and column genes of C. \subsection{Null distribution for PGCEs}For two independently evolving genes $i$ and $j$, the counted gains of $j$ in the presence of $i$, $C_{ij}$, will be distributed under the null hypothesis (independent evolution) as some function of the prevalence of i (the sum of $P_i$, the vector of probabilities of presence of $i$ across branches of the tree), the expected number of branches where $j$ is gained (the sum of $G_j$, the vector of probabilities of gains of $j$ across nodes of the tree), and the topology and branch lengths of the tree ($\tau$):\begin{equation*}
C_{ij}  ~ f(P_i,G_j, \tau )
\end{equation*}We followed previous studies \cite{Cohen2012,Huelsenbeck2003,Maddison1990} by approximating this null distribution via parametric bootstrapping. Specifically, we simulated the evolution of $10^5$ genes along the tree using the APE library function rTraitDisc() \cite{Paradis2004}. For the gain and loss rates used in these simulations, we used gainLoss gain and loss rates estimated for the 5801 empirical genes. We fit gamma distributions to these values by maximum likelihood using the function fitdistr() from the MASS library \cite{Venables2002}. For both gains and losses, we increased the shape parameter of the gamma distribution (by a factor of 3 for gains, 1.5 for losses), to ensure that simulated genes showed sufficiently large numbers of gains. This was necessary because parametric bootstrapping with the rates inferred by gainLoss resulted in left skewed distributions of gene gains (compare  Figures ~\ref{fig:app8fig2}A, C, E), which were likely to confound null models. For our null models to be applicable, the distribution of simulated gene gains should be roughly similar to the distribution of gains among empirical genes (see Figure ~\ref{fig:app8fig2}, Appendix G). These simulated genes should evolve independently and thus represent a null model for PGCEs. As above, we constructed matrices representing the probabilities of presence and gain of these $10^5$ genes across all of the branches of the phylogeny ($P_{null}$ and $G_{null}$). We then multiplied these matrices of simulated genes to compute a $10^5$ x $10^5$ matrix $C_{null}$ of expected branch counts under a model of independence. We excluded gene pairs with $C_{ij} \leq 1$ from further analysis, as it may be difficult to distinguish between no association and a lack of statistical power for such pairs (Figure ~\ref{fig:app8fig3}A), reducing overall power in computing false discovery rates \cite{Bourgon2010}. As a null distribution for each pair of genes i and j with $C_{ij} > 1$, we used the 1000 simulated genes with prevalence closest to gene $i$ (rows of $C_{null}$), and the 1000 simulated genes with a number of gains closest to gene $j$ (columns of $C_{null}$). We used the $10^6$ simulated observations in the resulting submatrix of $C_{null}$ as a null distribution for $C_{ij}$. Notably, $C_{ij}$ includes non-integer count expectations, whereas $C_{null}$ represents integer counts (because the true reconstruction is known). Consequently, we floored values in $C_{ij}$, such that all counts were truncated at the decimal point. The comparison of $C_{ij}$ to this null distribution yields an empirical p-value; we rejected the null hypothesis of independence between genes $i$ and $j$ for the $C_{ij}$ observation at a 1\% false discovery rate \cite{Benjamini1995} ($P < 7 x 10^-6$). \subsection{Constructing a PGCE network.}For each entry in $C_{ij}$ for which we observed a significant association, we recorded an edge from gene $i$ to gene $j$ in a network of PGCEs. To focus purely on direct interactions, we subjected this network to a transitive reduction \cite{Hsu1975}. This reduction requires a directed acyclic graph (DAG). To identify the largest possible DAG in our PGCE network, we identified and removed the minimal set of edges inducing cycles (Appendix G). We performed a transitive reduction of the resulting DAG using Hsus algorithm \cite{Hsu1975} (Appendix G). \subsection{Mapping biological information to the network.}We used network rewiring (as implemented in the rewire() function of the igraph library \cite{Csardi2006}) to generate null distributions of the PGCE network by randomly exchanging edges between pairs of connected nodes, while excluding self-edges. In each permutation, we performed 5N rewiring operations, where there are N edges in the network, to ensure sufficient randomization. To estimate the relationship between the PGCE network and biological information we calculated the number of edges shared between the PGCE network and a metabolic network of all bacterial metabolism obtained from KEGG \cite{Kanehisa2012,Levy2013}, and the number of edges shared between members of the same functional pathway as defined by KEGG, in both the original and randomized networks. To determine whether genes with certain functional annotations were more likely to associate with one another in the PGCE network, we examined the KEGG Pathway annotations of each pair of genes in the network. We counted the number of edges leading from each pathway to each other pathway, and obtained an empirical p-value for this count by comparing it to a null distribution of the expected counts obtained by random rewiring as above. \subsection{Topological sorting of PGCE networks}To identify global patterns in our PGCE network, we performed topological sorting \cite{Kahn1962} with grouping. Topological sorting finds an absolute ordering of nodes in a directed acyclic graph (DAG), such that no node later in the ordering has an edge directed towards a node earlier in the ordering. Grouping the sort allows nodes to have the same rank in the ordering if precedence cannot be established between them, giving a unique solution. For a description of the algorithm used, see Appendix G.\subsection{Prediction of HGT events on branches.}We used the PGCE network to predict the occurrence of specific HGT events (gene acquisitions) on the tree in the following fashion. We used two test/training set partitions, with the clades of Firmicutes and the Alpha/Betaproteobacteria as independent test sets, and the training sets as the rest of the tree without these clades. To train PGCE networks, we performed ancestral reconstruction of gene presence, PGCE inference, and network processing just as for the entire tree. We only attempted to predict genes with at least one PGCE dependency (predictable genes). We then considered each branch in the test set independently, attempting to predict whether each predictable gene was gained on that branch based on the reconstructed genome at the ancestor node. For each predictable gene-branch combination, our prediction score was the proportion of the predictable genes PGCE dependencies that are present in the ancestor. This is the dot product of the gene presence/absence pattern of the ancestor node (Ai across i potentially present genes) and a binary vector denoting which genes in the PGCE network the predictable gene depends on (Pi across i genes in potential PGCEs), scaled by Pi:$score=  \frac{\sum A_i P_i}{\sum P_i} $Note that this value ranges between 0 and 1 for each predicted gene. As true gains, we used our reconstructed gene acquisition events for each branch in the test set. We arbitrarily called any predictable gene-branch pair with a Pr(gain) > 0.5 as a gain, and any predictable gene-branch pair with $Pr(gain) \leq 0.5$ as no gain. We filtered out any gene-branch pair where the gene was known to be present with Pr > 0.4, as in these cases the gene is probably already present. We analyzed the accuracy of our prediction scores using receiver operating characteristic (ROC) analysis and by comparing scores of the gain branches to those of the no-gain branches.\subsection{Data Access}Parameter and log files for principal analyses are provided as Supplemental Files S2 and S3. Data and code are provided as Supplemental File S4. These files are available at the paper site either at the journal or on bioRXiv.

\section{Results}\subsection{PGCE Inference}We first set out to detect pairs of genes for which the presence of one gene in the genome promotes the gain of the other gene (though not necessarily vice versa) (Figure ~\ref{fig:hgtfig1}). Such pairs of genes with conjugated evolution (PGCEs) represent putative epistatic interactions at the gene level and may guide genome evolution. To this end, we obtained a collection of 634 prokaryotic genomes annotated by KEGG \cite{Kanehisa2012} and linked through a curated phylogeny \cite{Dehal2010}. For each of the 5801 genes that varied in presence across these genomes, we reconstructed the probability of this genes presence or absence on each branch of the phylogenetic tree using a previously introduced method \cite{Cohen2010}, as well as the probability that it was gained or lost along these branches using a simple heuristic (Methods). We confirmed that genes presence/absence was robust to the reconstruction method employed (99.5\% agreement between reconstruction methods used; Methods). As expected \cite{Mira2001}, gene loss was more common than gene gain for most genes  (Figure ~\ref{fig:app8fig1}, Appendix G). We additionally confirmed that inferred gains of several genes of interest were consistent with gains inferred by an alternative HGT inference method (Methods; Appendix G, Table ~\ref{tab:app8tab1}). From the reconstructions, we estimated the frequency with which each gene was gained in the presence of each other gene, and followed previous studies \cite{Maddison1990,Cohen2012} in using parametric bootstrapping (Figure~\ref{fig:app8fig2}) to detect PGCEs  gene pairs for which one gene is gained significantly more often in the presence of the other (Figure ~\ref{fig:app8fig3}, Appendix G). In total, we identified 8,415 PGCEs. We finally applied a transitive reduction procedure to discard potentially spurious PGCEs, resulting in a final network containing 8,228 PGCEs connecting a total of 2,260 genes (Figure ~\ref{fig:app8fig4}, ~\ref{fig:app8fig4}, Appendix G). A detailed description of the procedures used can be found in Methods. \subsection{PGCEs represent biologically relevant dependencies}Comparing this final set of PGCEs to known biological interactions, we confirmed that the obtained PGCEs represent plausible biological dependencies. For example, genes sharing the same KEGG Pathway annotations were more likely to form a PGCE (Figure ~\ref{fig:hgtfig2}A), as were genes linked in an independently-derived network of bacterial metabolism \cite{Levy2013} (Figure ~\ref{fig:hgtfig2}B). Moreover, PGCEs often linked genes in functionally related pathways (Supplemental Figure S6, Supplemental Text). We similarly identified specific examples in which PGCEs connected pairs of genes with well-described functional relationships. One such example is the PGCE connecting rbsL and rbsS (sometimes written rbcL/rbcS), two genes that encode the large and small subunits of the well-described photosynthetic enzyme ribulose-1-5-bisphosphate carboxylase-oxygenase (RuBisCO), respectively. The rbsL subunit alone has carboxylation activity in some bacteria, but the addition of rbsS increases enzymatic efficiency, consistent with its PGCE dependency on rbsL (Figure ~\ref{fig:hgtfig3}A) \cite{Andersson2008}. Moreover, these genes are known to undergo substantial horizontal transfer \cite{Delwiche1996}. Multiple additional genes were found to promote rbsS gain (88 PGCEs in total, Table ~\ref{tab:app8tab2}), many of which, as expected, are associated with carbon metabolism. Other genes in this set, however, unexpectedly implicated nitrogen acquisition, as well as other pathways (Table ~\ref{tab:app8tab3}), in promoting rbsS gain. For example, all components of the urt urea transport complex had a PGCE link with rbsS, as shown by the reconstructed phylogenetic history of urtA and rbsS (Figure ~\ref{fig:hgtfig3}B). This strict dependency could reflect nitrogens role as a rate-limiting resource for primary production in phytoplankton and other photosynthetic organisms \cite{Eppley1979,Sohm2011}. In comparing the reconstructions from which urtA-rbsS and rbsL-rbsS dependencies were inferred, we further observed that rbsS is gained only in lineages where both urtA and rbsL were previously present. This indicates that while both rbsL and urtA may be necessary for the acquisition of rbsS, neither rbsL nor urtA are independently sufficient for the acquisition of rbsS. Other PGCEs may interact in similarly complex fashions in controlling the acquisition of genes, and thus such relationships may be gene-specific and involve a variety of biological mechanisms that may be difficult to generalize. For further analyses, we therefore focused on analyzing large-scale patterns of PGCE connectivity and on exploring how the dependencies between various genes structure the relationships between functional pathways.\subsection{PGCE network analyses reveal evolutionary assembly patterns}The rbsS-associated PGCEs described above show how PGCEs captured an assembly pattern involving multiple pathways. Therefore, we next set out to infer global evolutionary assembly patterns based on the complete set of PGCEs identified. Specifically, we used a network-based topological sorting approach (Appendix G) to rank all genes in the PGCE network. According to this procedure, genes without dependencies occupy the first rank, genes in the second rank have PGCE dependencies only on first rank genes, genes in the third rank have dependencies only on first and second rank genes, and so on until all genes are associated with some rank. In other words, the obtained ranking represents general patterns in the order by which genes are gained throughout evolution, with the gain of higher-ranked genes succeeding the presence of the lower-ranked genes on which they depend. Using this approach, we found that genes could be fully classified into five ranks (Figure ~\ref{fig:hgtfig4}A). The first rank was by far the largest at 1,593 genes (most genes do not have detectable dependencies), the second rank had 498 genes, and successive ranks showed declining membership until the last (fifth) rank, with only 5 genes (Table ~\ref{tab:app8tab4}). 

%% table 1
To identify evolutionary assembly patterns from these ranks, we examined the set of genes in each rank and identified overrepresented functional categories (Table ~\ref{tab:hgttab1}). These enriched functional categories indicate that certain functional groups of genes consistently occupy specific positions in these evolutionary assembly patterns, whether in controlling other genes gain or in being controlled by other genes. For example, we found that the first rank was enriched for flagellar and pillar genes involved in motility, in addition to Type II secretion genes (many of which are homologous to or overlap with genes encoding pillar proteins) and certain two-component genes. The second rank was enriched for various metabolic processes, whereas later ranks were enriched for Type III and Type IV secretion systems and conjugation genes. This finding suggests that habitat commitments are made early in evolution, mediated by motility genes that could underlie the choice and establishment of physical environments. This environmental choice is followed by a metabolic commitment to exploiting the new habitat. Last, genes for interaction with the biotic complement of these habitats are gained, and replaced frequently in response to evolving challenges. Considering two distinct but highly homologous pilus assembly pathways, one (fimbrial) was enriched in a low rank and one (conjugal) was enriched in a high rank, suggesting that the specific function of the gene rather than other sequence-level gene properties drove the ranking (Figure ~\ref{fig:app8fig7}A). We additionally confirmed that the observed rank distribution for these functions is not explained by variation in the frequency of gene gain (Figure ~\ref{fig:app8fig7}B). Furthermore, as expected, we observed that the gains of genes appearing late in the sort were overrepresented in later branches of the tree compared to the gains of lower-ranked genes (Figure ~\ref{fig:hgtfig4}, Figure ~\ref{fig:app8fig5}8), suggesting that the chronology of gene acquisition reflects the overall assembly patterns in gain order. \subsection{Evolution by HGT is predictable}The chronological ordering of ranks was relatively consistent across the tree (Figure 4B), indicating that PGCE dependencies are universal across prokaryotes. Notably, this universality also implies that gene acquisition is predictable from genome content. Put differently, if PGCEs are universal, then PGCEs inferred in one clade of the tree are informative in making predictions about gene acquisition in a different clade. Indeed, studies of epistasis-mediated protein evolution indicate that the constriction of possible mutational paths should lead to predictability in evolution, if epistasis is sufficiently strong (Weinreich et al. 2006). To explore this hypothesis explicitly, we partitioned the tree into training and test sets (Figure 5A). As test sets, we selected the Firmicutes phylum, and the Alphaproteobacteria/Betaproteobacteria subphyla. Choosing whole clades as test sets (rather than randomly sampling species from throughout the tree) guarantees that true predictions are based on universal PGCEs, rather than clade-specific PGCEs. For each test set, we used a model phylogeny that excluded the test subtree as a training set, and inferred PGCEs based on this pruned tree (Supplemental Table S5, Supplemental Figure S9A). We then used these inferred PGCEs to score the relative likelihood of the gain of dependent genes on each branch in the test set, based on the genome content of the branchs ancestor (Figure 5A, Supplemental Table S5, Supplemental Text). We used a nave and simplistic score: the proportion of genes upon which the gained gene depends that are present in the reconstructed ancestor of each branch. In both test sets, we found that prediction quality was surprisingly high (Figure 5B, Supplemental Figure S9B-C), suggesting that PGCEs are taxonomically universal and statistically robust in describing relationships between genes. This predictability is consistent with the hypothesis that gene-gene dependencies constrain the evolution of genomes by HGT. More broadly, this analysis and our finding that PGCEs predictably can determine future evolutionary gains provide substantial evidence that the preponderance of parallel evolution over convergent evolution (Ord and Summers 2015; Conte et al. 2012) may be the result of specific, identifiable genetic dependencies entraining the evolutionary trajectory taken by similar genomes. \section{Discussion}Combined, our findings provide substantial evidence to suggest that gene acquisitions in bacteria are governed by genome content through numerous gene-level dependencies. Our ability to detect these underlying dependencies is clearly imperfect, owing to various data and methodological limitations (Supplemental Text, Supplemental Figure S3). Therefore, in reality the complete dependency network is likely much denser than that described above and includes numerous dependencies and constraints that our approach may not be able to detect. Consequently, our estimates should be considered as a lower bound on the extent of gene-gene interactions, and accordingly, the predictability of HGT.Notably, even considering such caveats, our observations dramatically expand our knowledge of the constraints on HGT. Previous studies of such constraints demonstrated that genes frequently acquired by HGT tend to occupy peripheral positions in biological networks, are often associated with specific cellular functions, and are phylogenetically clustered (Jain et al. 1999; Cohen et al. 2011). These observations suggested that properties of transferred genes are also important determinants of HGT regardless of recipient genome content (Jain et al. 1999; Cohen et al. 2011; Gophna and Ofran 2011) and that the acquisition of certain genes is clade-specific (Popa et al. 2011; Andam and Gogarten 2011). In contrast, our analysis demonstrates the importance of recipient genome content in influencing the propensity of a new gene to be acquired. In fact, to some extent, properties previously reported as determining the general acquirability of genes across all species may reflect an average constraint across genomes. By considering also variation in genomes acquiring genes, our analysis focused on specific biological effects, whose strengths may vary from genome to genome.Importantly, our model that gene acquisition is affected by recipient genome content is consistent with the observed enrichment of HGT among close relatives, which presumably have similar genome content (Gogarten et al. 2002; Andam and Gogarten 2011; Popa et al. 2011; Popa and Dagan 2011). This taxonomic clustering of innovation by HGT is also in agreement with previous studies that demonstrated that phenotypic and genetic parallel evolution is more common than convergent evolution, potentially due to the effects of historical contingency (Gould and Lewontin 1979; Conte et al. 2012; Christin et al. 2015; Ord and Summers 2015). However, in contrast to other studies, we present direct evidence that the mechanism by which contingency controls evolution is epistasis. Furthermore the universality of PGCEs shows that the constraints underlying the effect of contingency operate outside the context of parallel evolution. Put differently, since each phylum-level clade is subject to an independent evolutionary trajectory, it is unlikely that the same dependency patterns would repeat solely due to parallel evolution. Moreover, our ability to predict where exactly along the tree gains of a specific gene are likely to occur (Figure 5B) suggests that PGCEs successfully capture how variation in the genomic content (even among closely related species) affects future gain events. Such PGCE specificity therefore indicates that observed dependencies are not a trivial byproduct of prevalent gene transfer events among taxonomically closely related genomes (e.g., due to homologous recombination constraints; Popa et al. 2011). Nonetheless, the relative contribution of each of these various processes governing the assembly of prokaryotic genomes (and the evolution of complex systems in general) clearly deserves future study.It should also be noted that while our analysis revealed several intriguing patterns, the precise interpretation of some of these patterns remains unclear. For instance, the observed correspondence of topological ranks of genes to chronology suggests that evolutionary age is a potential contributor to such ranking, especially considering that our reconstructions likely lack many genes that have not been retained in any extant genomes. However, the biological plausibility and statistical robustness of PGCEs demonstrated above strongly argue that the observed evolutionary patterns are the result of constraint-inducing dependencies. Future work may therefore aim to quantify the trade-off between functional and chronological determinants in apparent evolutionary constraints.  Finally, we demonstrate the predictability of genomic evolution by horizontal transfer from current genomic content. As stated above, this finding also suggests that such dependencies are fairly universal across the prokaryotic tree. It should be noted that our approach was designed specifically to understand the PGCE networks significance and universality, rather than predict gene acquisition. It is likely that an approach specifically engineered for gene acquisition prediction would substantially outperform our approach. The estimates of predictability of genomic evolution presented here are accordingly quite conservative.The determinism and predictability of evolutionary patterns therefore appear to be an outcome not only of intramolecular epistasis in proteins or phylogenetic constraints, but also of genome-wide interactions between genes. This suggests that the evolution of medically, economically, and ecologically important traits in prokaryotes depends on ancestral genome content and is hence at least partly predictable, potentially informing research in the epidemiology of infectious diseases, bioengineering, and biotechnology.\section{Acknowledgements}We are obliged to members of the Borenstein and Queitsch laboratories, and to Evgeny Sokurenko, Joe Felsenstein, and Willie Swanson for helpful discussions. We thank Ofir Cohen for help with the gainLoss program. We thank Hyeon Soo Jeong for help with the HGTree database. MOP was supported in part by National Human Genome Research Institute Interdisciplinary Training in Genome Sciences Grant 2T32HG35-16. CQ is supported by National Institute of Health New Innovator Award DP2OD008371. EB is supported by National Institute of Health New Innovator Award DP2AT00780201. We thank UW Genome Sciences Information Technology Services for high-performance computing resources. 


%%%%% CONCLUSIONS IS ITS OWN CHAPTER RIGHT?????
\chapter{Conclusions and future work}

\section{Epistasis, STRs, and the shifting-balance theory}

\subsection{Mutation rate heterogeneity}
\subsubsection{HGTs as mutations}

\section{Predictability of whole-genome evolutionary trajectories}

\subsection{Intramolecular epistasis, interlocus epistasis, and parallel evolution}
\subsection{Assembly patterns vs. trajectories}

\section{Some final observations}

\section{Next steps}

\subsection{STR genotype data collection}

\subsection{Towards better evolutionary predictions.}

% ========== Chapter 3
 
\chapter{The Thesis Unformatted}
 
This chapter describes the uwthesis class (\texttt{uwthesis.cls},
version dated 2011/06/27)
in detail 
and shows how it was used to format the thesis.
A working knowledge of Lamport's \LaTeX\ manual\cite{Lbook} is assumed.
 
\section{The Control File}
 
The source to this sample thesis is contained in a single file
only because ease of distribution was a concern.
You should not do this.  Your task will be much easier if you
break your thesis into several files:  a file for the preliminary pages,
a file for each chapter,  one for the glossary, and one for each
appendix.  Then use a control file to tie them all together.
This way you can edit and format parts of your thesis much more
efficiently.
 
Figure~\ref{control-file} shows a control file that
might have produced this thesis.
It sets the document style, with options and parameters,
and formats the various parts of the thesis---%
but contains no text of its own.
 
 
%  control file caption and figure
%
\begin{figure}[p]
 \begin{leftfullpage}
  \caption[A thesis control file]%
   {\narrower A thesis control file ({\tt thesis.tex}).
   This file is the input to \LaTeX\ that will produce a
   thesis.  It contains no text, only commands which
   direct the formatting of the thesis.
   This is also an example of a `facing page' caption.  It is guaranteed
   to appear on a lefthand page, facing the figure contents on the right.
   See the text.}
  \label{control-file}
 \end{leftfullpage}
\end{figure}
%
\begin{figure}[p]
%
 \begin{fullpage}
  \footnotesize
  \begin{verbatim}
    % LaTeX thesis control file
 
    \documentclass[11pt,twoside]{uwthesis}
 
    \begin{document}
 
    % preliminary pages
    %
    \prelimpages
    \include{prelim}
 
    % text pages
    %
    \textpages
    \include{chap1}
    \include{chap2}
    \include{chap3}
    \include{chap4}
 
    % bibliography
    %
    \bibliographystyle{plain}
    \bibliography{all}
 
    % appendices
    %
    \appendix
    \include{appxa}
    \include{appxb}
 
    \include{vita} 
    \end{document}
  \end{verbatim}
 \end{fullpage}
\end{figure}
 
The first section, from the \verb"\documentclass" to
the \verb"\begin\{document\}", defines the document class and options.
This thesis has specified two-sided formatting, which is now
allowed by the Graduate School.  Two sided printing is now
actually \LaTeX's default.  If you want one sided printing
you must specify \verb"oneside".
This sample also specified a font size
of 11 points. 
Possible font size options are: \verb"10pt", \verb"11pt", and \verb"12pt".
Default is 12 points, which is the preference
of the Graduate School. If you choose a smaller size be sure to
check with the Graduate School for acceptability.  The smaller fonts
can produce very small sub and superscripts.

Include most additional formatting packages with \verb"\usepackage",
as describe by Lamport\cite{Lbook}.  The one exception to this
rule is the \verb"natbib" package.  Include it with the \verb"natbib"
document option.
 
Use the \verb"\includeonly" command to format only a part of your
thesis.  See Lamport\cite[sec. 4.4]{Lbook} for usage and limitations.

 
\section{The Text Pages}
 
A chapter is a major division of the thesis.  Each chapter begins
on a new page and has a Table of Contents entry.
 
\subsection{Chapters, Sections, Subsections, and Appendices}
 
 
Within the chapter title use a \verb"\\" control sequence to separate lines
in the printed title (recall Figure \ref{start-2}.).
The \verb"\\" does not affect the Table of Contents entry.
 
Format appendices just like chapters.
The control sequence \verb"\appendix" instructs \LaTeX\ to
begin using the term `Appendix' rather than `Chapter'.
 
 
Sections and subsections of a chapter are specified
by  \verb"\section" and \verb"\subsection", respectively.
In this thesis chapter and section
titles are written to the table of contents.
Consult Lamport\cite[pg. 176]{Lbook} to see which
subdivisions of the thesis can be written to the table of contents.
The \verb"\\" control sequence is not permitted in section and
subsection titles.
 
 
\subsection{Footnotes}
 
\label{footnotes}
 Footnotes format as described in the \LaTeX\ book.  You can also
 ask for end-of-chapter or end-of-thesis notes.
 The thesis class will automatically set these up if
 you ask for the document class option \texttt{chapternotes}
 or \texttt{endnotes}.  
 
If selected, chapternotes will print automatically.  If you choose
endnotes however you must explicitly indicate when to print the notes 
with the command \verb"\printendnotes".  See the style guide for
suitable endnote placement.  

\subsection{Figures and Tables}
Standard \LaTeX\ figures and tables, see Lamport\cite[sec.~C.9]{Lbook},
normally provide the most convenient means to position the figure.
Full page floats and facing captions are exceptions to this rule.

If you want a figure or table to occupy a full page enclose the
contents in a \texttt{fullpage} environment.  
See figures~\ref{facing-caption}.

Facing page captions are described
in the Style Manual\cite{SP}.  They have different meanings
depending on whether you are using
the one-side or two-side thesis style.


If you are using the two-side style,
facing captions
are full page captions for full page figures or tables
and must face the illustration to which they refer.
You must explicitly format both pages. 
The caption part must appear on an even page
(left side) and the figure or table must
come on the following odd page (right side).
Enclose the float contents for the caption 
in a \texttt{leftfullpage} environment,
and enclose the float contents for the figure or table 
in a \texttt{fullpage} environment.
Figure~\ref{control-file}, for example,
required a full page so its caption (on a facing caption page)
would have been formatted as shown in figure~\ref{facing-caption}a.
The first page (left side) contains the caption. The second page
(right side) could be left blank.  A picture or graph might be pasted onto
this space.


\begin{figure}[t]
\footnotesize
\begin{verbatim}
     \begin{figure}[p]% the left side caption
       \begin{leftfullpage}
         \caption{ . . . }
       \end{leftfullpage}
     \end{figure}
     \begin{figure}[p]% the right side space
       \begin{fullpage}
          . . .
          ( note.. no caption here )
       \end{fullpage}
     \end{figure}
\end{verbatim}
\caption(a){This text would create a
  double page figure in the two-side style.}
\label{facing-caption}
\end{figure}
 
\begin{figure}[t]
\footnotesize
\begin{verbatim}
     \begin{figure}[p]
        \begin{leftfullpage}
           \caption{ . . . }
        \end{leftfullpage}
     \end{figure}
     \begin{figure}[p]% the right side space
       \begin{xtrafullpage}
          . . .
          ( note.. no caption here )
       \end{xtrafullpage}
     \end{figure}
\end{verbatim}
\caption(b)[Generating a facing caption page]{This text would create a
  facing caption page with the accompaning figure in the one-side style.}
\end{figure}
 
If instead you are using the one-side style,
facing caption pages are still
captions for full page figures or tables
that appear on the left-hand page (facing the illustration on the
right-hand page).  
However, the page number and binding offset are reversed
from their normal positions.
Format these captions by enclosing the float contents
in a \texttt{leftfullpage} environment.
Because you are printing on only one side of each sheet, you must manually
turn over this caption sheet. 
You then have the choice of inserting a preprinted illustration or
formatting one to print with the thesis. 
In either case no page number should appear
on the illustration page, nor should the page number increment. 
Enclose your figure's text
in an \texttt{xtrafullpage} environment, which will cause the
page numbers to come out right.  
You can, of course, leave out the illustration and insert a preprinted
copy later. 
Figure~\ref{facing-caption}b shows how to format a facing caption page
in the one-side style. Note that, in this case, the illustration
was also printed.

In the two-side style the \texttt{xtrafullpage} environment acts just like the
\texttt{fullpage} environment.  It does not produce a numberless page.


 
\subsection{Horizontal Figures and Tables}
Figures and tables may be formatted horizontally
(a.k.a.\ landscape) as long as their captions appear
horizontal also.  \LaTeX\ will format landscape material for you
if a couple of conditions are met.  You have to have a printer
and printer driver that allow rotations and
you have to have a couple of add-on \LaTeX\ packages.  

% Users of PostScript printers and Uniform Access computers 
% at the University of Washington will conform to both requirements,
% as will users of PC\TeX\ if they use postscript.

Include the \texttt{rotating} package 
\begin{demo}
\\usepackage[figuresright]\{rotating\}
\end{demo}
and read the documentation that comes with the package. 

Figure~\ref{sideways} is an example of how a landscape
table might be formatted. 

\begin{figure}[t]
\footnotesize
\begin{verbatim}
     \begin{sidewaystable}
         ...
         \caption{ . . . }
     \end{sidewaystable}
\end{verbatim}
\caption[Generating a landscape table]{This text would create a
  landscape table with caption.}
\label{sideways}
\end{figure}
 


\subsection{Figure and Table Captions}
Most captions are formatted with the \verb"\caption" macro as described 
by Lamport\cite[sec. C.9]{Lbook}. 
The uwthesis class extends this macro to allow
continued figures and tables, and to provide multiple figures and
tables with the same number, e.g., 3.1a, 3.1b, etc.
 
To format the caption for the first part of
a figure or table that cannot fit
onto a single page use the standard form:
\begin{demo}
\\caption[\textit{toc}]\{\textit{text}\}
\end{demo}
To format the caption for the subsequent parts of 
the figure or table 
use this caption:
\begin{demo}
\\caption(-)\{(continued)\}
\end{demo}
It will keep the same number and the text of the caption will be 
{\em(continued)}.

To format the caption for the first part of
a multi-part figure or table
use the format:
\begin{demo}
\\caption(a)[\textit{toc}]\{\textit{text}\}
\end{demo}
The figure or table will be lettered (with `a') as well as numbered.
To format the caption for the subsequent parts of 
the multi-part figure or table
use the format:
\begin{demo}
\\caption(\textit{x})\{\textit{text}\}
\end{demo}
where {\em x} is {\tt b}, {\tt c}, \ldots.
The parts will be lettered (with `b', `c', \ldots).

\section{The Preliminary Pages}
 
These are easy to format only because they are relatively invariant
among theses.  Therefore the difficulties have already been encountered
and overcome by \LaTeX\ and the thesis document classes.

Start with the definitions that describe your thesis.
This sample thesis was printed with the parameters:

\begin{demo}
\\Title\{The Suitability of the \\LaTeX\\ Text Formatter\\\\
   for Thesis Preparation by Technical and\\\\
   Non-technical Degree Candidates\}
\\Author\{Jim Fox\}
\\Program\{UW Information Technology\}
\\Year\{2012\}

\\Chair\{Name of Chairperson\}\{title\}\{Chair's department\}
\\Signature\{First committee member\}
\\Signature\{Next committee member\}
\\Signature\{etc\}

\end{demo}
 
Use two or more \verb"\Chair" lines if you have co-chairs.
 
\subsection{Copyright page}
Print the copyright page with \verb"\copyrightpage".

\subsection{Title page}
Print the title page with \verb"\titlepage".
The title page of this thesis was printed with%
\footnote{Actually, it wasn't.  I added a footnote---something you would not do.}
 
\begin{demo}
\\titlepage
\end{demo}
 
You may change default text on the title page with these
macros.  You will have to redefine \verb"$\degree$text", for instance,
if you're writing a Master's thesis instead of a dissertation.\footnote{If you use these they can
be included with the other information before \\copyrightpage".}

\begin{list}{}{\itemindent\parindent\itemsep0pt
   \def\makelabel#1{\texttt{\char`\\#1}\hfill}}
\item[Degree\char`\{{\it degree name}\char`\}]
   defaults to ``Doctor of Philosophy''
\item[School\char`\{{\it school name}\char`\}] defaults to
``University of Washington''
\item[Degreetext\char`\{{\it degree text}\char`\}] defaults to
``A dissertation submitted \ldots''
\item[textofCommittee\char`\{{\it committee label}\char`\}] defaults to
``Reading Committee:''
\item[textofChair\char`\{{\it chair label}\char`\}] defaults to
``Chair of the Supervisory Committee:''
\end{list}

These definitions must appear \underline{before} the \verb"\titlepage" command.

 
\subsection{Abstract}
Print the
abstract with \verb"\abstract".
It has one argument, which is the text of the abstract.
All the names have already been defined.
The abstract of this thesis was printed with
 
\begin{demo}
\\abstract\{This sample . . . `real' dissertation.\}
\end{demo}
 
 
\subsection{Tables of contents}
Use the standard \LaTeX\ commands to format these items.
 
 
\subsection{Acknowledgments}
Use the \verb"\acknowledgments" macro to format the acknowledgments page.
It has one argument, which is the text of the acknowledgment.
The acknowledgments of this thesis was printed with
 
\begin{demo}
\\acknowledgments\{The author wishes . . . \{\\it il miglior fabbro\}.\\par\}\}
\end{demo}
 
 
%%  
%% \section{Customization of the Macros}
%%  
%% Simple customization, including 
%% alteration of default parameters,  changes to dimensions,
%% paragraph indentation, and margins, are not too difficult.
%% You have the choice of modifying the class file ({\tt uwthesis.cls})
%% or loading
%% one or more personal style files to customize your thesis.
%% The latter is usually most convenient, since you do not need
%% to edit the large and complicated class file.
%% 
 
\printendnotes

%
% ==========   Bibliography
%
%\nocite{*}   % include everything in the uwthesis.bib file
\bibliographystyle{plain}
\bibliography{all}	% points to file all.bib
%
% ==========   Appendices
%
\appendix
\raggedbottom\sloppy
 
% ========== Appendix A
 
\chapter{Supporting Chapter 2}

\section{Supporting Figures}
\begin{figure}
\graphicspath{{}}
\includegraphics[width=6in]{undurraga2012_figures/SupplText/SuppFig1.png}
\caption[The ELF3-TR variation is not correlated with ELF3 expression.]{} \label{fig:app1fig1}
\end{figure}
\addtocounter{figure}{-1}
\captionsetup[figure]{list=no}
\captionof{figure}{The ELF3-TR variation is not correlated with ELF3 expression. (A) Histogram of ELF3-TR copy number across 181 accessions. TR copy number was determined by Sanger sequencing. (B) ELF3 expression levels in selected natural accessions were measured by quantitative RT-PCR. Expression values are given relative to the Col-0 wild-type reference. Three biological replicates with three technical replicates each were used to obtain expression values. Bars indicate $\pm$ SEM. (C and D) ELF3-TR transgenic lines are expression-matched in both genetic backgrounds. (C) elf3-4, Ws; (D) elf3-200, Col. ELF3 mRNA levels were measured by quantitative PCR (for primers see Table A4) in pooled 10-d-old seedlings that were grown under LD and collected at ZT 20 for each independently generated ELF3-TR transgenic line. ELF3 expression levels are shown relative to either Ws (C) or Col-0 (D) wild-types. Because ELF3 expression levels are known to substantially affect ELF3-dependent phenotypes \cite{Kim2005}, ELF3 expression is an important variable to consider in our assessment of polyQ tract-length effects. We made efforts to consider only lines within a certain range of ELF3 expression and to test multiple independent lines per ELF3-TR allele (Tables A2A4), but because of the technical constraints of transgenic plant construction, we cannot entirely exclude the possibility that ELF3 expression partially explains our observations. Although the effects of both ELF3 expression level and ELF3-TR copy number were highly significant, they appear to be largely independent. For example, the ELF3-23Q and ELF3-16Q alleles, which were among the most distinct ELF3-TR alleles in both backgrounds, had very similar ranges of ELF3 expression. In Ws, the alleles ELF3-7Q, ELF3-23Q, and ELF3-10Q phenocopied an elf3 loss-of-function mutant for some phenotypes. Their ELF3 expression levels, however, were very similar to the ELF3-16Q allele, which complemented many ELF3 functions in elf3-4. As observed with individual ELF3-TR alleles, the phenotypic effects of ELF3 expression levels appear to be largely independent of ELF3-TR copy number, which consistently explained a larger portion of phenotypic variation. }

\captionsetup[figure]{list=yes}
\begin{figure}
\graphicspath{{}}
\includegraphics[height=8in]{undurraga2012_figures/SupplText/SuppFig2.png}
\caption[ELF3-TR variation has nonlinear phenotypic effects in the Ws background]{}\label{fig:app1fig2}
\end{figure}
\addtocounter{figure}{-1}
\captionsetup[figure]{list=no}
\captionof{figure}[]{ELF3-TR variation has nonlinear phenotypic effects in the elf3-4 background (Ws accession). (A) Days to flower (DTF) under SD (n = 6 plants per line). (B) Final number of rosette leaves (FLN) under SD (n = 6 plants per line). (C) DTF under LD (n = 15 plants per line). (D) FLN under LD (n = 15 plants per line). (E) Hypocotyl length under SD (n = 20-30 seedlings per line). (F) Hypocotyl length under LD (n = 20-30 seedlings per line). (G) PL/LL ratio under SD (n = 6 plants per line). Data are from the same plants as in B. ELF3-TR alleles are indicated with the number of Qs encoded, Ws is wild-type, VC is the elf3-4 vector control. Blue and red asterisks indicate alleles that are significantly different from the wild-type and from the VC, respectively, by Tukey-HSD test ($\mathrm{\alpha = 0.05}$). We used this analysis rather than the one presented in Figure 1B to preserve clarity. Bars indicate $\pm$ SEM. These experiments were repeated at least once with similar results. (H and I) PCA of phenotypic data for all ELF3-TR alleles in the elf3-4 background (Ws accession). (H) Biplot of PC1 and PC2, graphically showing the contribution of phenotypes to PCs as red arrows. Note that for the biplot representation, PC1 and PC2 are transformed to the same scale (bottom and left axes), whereas phenotype contributions (in red) are allowed to differ in scale (top and right axes). Phenotypes are hypocotyl length in short and long days (SD hylen and LD hylen), DTF in short and long days (SD DTF and LD DTF), and FLN in long days (LD rosette). Wild-type plants are characterized by late flowering (large SD and LD DTF, many rosette leaves) and short hypocotyls (small SD and LD hylen), relative to elf3 loss-of-function mutants. (I) PC1 and PC2. (J) PCA loadings for Ws background. hylen, hypocotyl length (mm). PCA loadings describe the composition/loading of each principal component . For PC1, flowering-time phenotypes and circadian clock phenotypes have opposite loading signs.}
\captionsetup[figure]{list=yes}


\begin{figure}
\graphicspath{{}}
\includegraphics[width=6in]{undurraga2012_figures/SupplText/S3.png}
\caption[ELF3-TR modulates expression of PIF5 and PRR9.]{} \label{fig:app1fig3}
\end{figure}
\addtocounter{figure}{-1}
\captionsetup[figure]{list=no}
\captionof{figure}{Expression levels of the ELF3-regulated genes PIF5 (A) and PRR9 (B and C). Plants were grown under LD and RNA was collected at times showing the largest expression difference between wild-type and elf3-4 mutant ZT8 for PIF5 \cite{Nusinow2011} (A) and ZT5 for PRR9 \cite{Kolmos2011, Dixon2011} (B and C). RNA levels were normalized relative to Ws wild-type. (C) Temporal variation in PRR9 expression across ELF3-TR transgenic lines. PRR9 expression levels were measured in 10-d-old plants grown under LD. RNA was collected at times demonstrating the diurnal oscillation of PRR9 expression in wild-type, as determined previously. RNA levels were normalized relative to wild-type (Ws) at ZT8. Gene expression was measured in triplicate for each biological replicate, with multiple independent transgenic lines as biological replicates for each ELF3 allele. Error bars indicate SE of expression across biological replicates. Our expression patterns of PRR9 for wild-type and the elf3-4 mutant are similar to previous observations \cite{Kolmos2011, Dixon2011}. ELF3-TR alleles are indicated with the number of Qs encoded, Ws is wild-type, VC is the elf3-4 vector control. Error bars are SEs of means. Data are from multiple independently generated expression-matched (Figure A1C) T3 and T4 lines for each TR copy number allele (Table A2).}
\captionsetup[figure]{list=yes}

\begin{figure}
\graphicspath{{}}
\includegraphics[width=5.5in]{undurraga2012_figures/SupplText/S4.png}
\caption[Circadian parameters estimated for different TR alleles in elf3-4 CCR2::Luc reporter lines.]{Circadian parameters estimated for different TR alleles in elf3-4 CCR2::Luc reporter lines. (A) Measured circadian period of CCR::LUC expression oscillation for each ELF3-TR allele. Bars correspond to 99\% confidence intervals for this proportion. (B) Measured RAE of CCR::LUC expression oscillation for each ELF3-TR allele. Bars correspond to 99\% confidence intervals for this proportion. Plants with RAE <0.4 are considered to have a robust circadian clock. (C) Estimated RAE and circadian period for each ELF3-TR allele. Points are means, dotted ellipses represent SEMs, and genotype labels indicate ELF3-TR copy number. Bioluminescence of the CCR2::LUC reporter present in ELF3-TR transgenic lines was used to measure circadian parameters (period and RAE). Seedlings were entrained in 12-h light:12-h dark cycles for 5 d and released to LL on the sixth day. Note that plants with high RAE have by definition unreliable estimates of circadian period. Number of seedlings for each genotype: Ws, 274; 0Q, 249; 7Q, 122; 10Q, 222; 11Q, 339; 14Q, 214; 15Q, 284; 16Q, 534; 20Q, 161; 21Q, 243; 22Q, 271; 23Q, 196; 29Q, 257; elf3-4 vector control, 102.} \label{fig:app1fig4}
\end{figure}

\begin{figure}
\graphicspath{{}}
\includegraphics[width=5.5in]{undurraga2012_figures/SupplText/S5.png}
\caption[ELF3-TR variation has nonlinear phenotypic effects in the elf3-200 background (Col-0 accession).]{} \label{fig:app1fig5}
\end{figure}
\addtocounter{figure}{-1}
\captionsetup[figure]{list=no}
\captionof{figure}{ELF3-TR variation has nonlinear phenotypic effects in the elf3-200 background (Col-0 accession). (A) DTF under SD (n = 9 plants/line). (B) FLN under SD (n = 9 plants per line). (C) DTF under LD (n = 15 plants per line). (D) FLN under LD (n = 15 plants per line). (E) Hypocotyl length under SD (n = 2030 seedlings per line). (F) Hypocotyl length under LD (n = 2030 seedlings per line). (G) PL/LL ratio under SD (n = 9 plants per line). Data are from the same plants as in B. ELF3-TR alleles are indicated with the number of Qs encoded, Col is wild-type, VC is the elf3-200 vector control (VC). Blue and red asterisks indicate alleles that are significantly different from the wild-type and from the vector control, respectively, by Tukey-HSD test ($\alpha$ = 0.05). Bars indicate $\pm$ SEM. These experiments were repeated at least once with similar results. (H and I) PCA of phenotypic data for all ELF3-TR alleles in the elf3-200 (Col accession) background. (H) Biplot of PC1 and PC2, graphically showing the contribution of phenotypes to PCs as red arrows. Note that for the biplot representation, PC1 and PC2 are transformed to the same scale (bottom and left axes), whereas phenotype contributions (in red) are allowed to differ in scale (top and right axes). Phenotypes are hypocotyl length in short and long days (SD hylen and SD hylen), DTF in short and long days (SD DTF and SD DTF), and FLN in long days (SD FLN). Wild-type type plants are characterized by late flowering (large SD and SD DTF, many rosette leaves) and short hypocotyls (small SD and SD hylen), relative to elf3 loss-of-function mutants. (I) PC1 and PC2. Note that PC1's orientation is inverted relative to PCAs including Ws-background plants (A and B: i.e., elf3-200 is to the negative end of the axis, and Col is at the positive end); this does not affect interpretation. In contrast to PCAs including Ws data, PC2 of Col data alone represents the differential response of LD and SD phenotypes to ELF3-polyQ copy number variation. (J) PCA loadings for Col background. hylen = hypocotyl length (mm). }
\captionsetup[figure]{list=yes}

\begin{figure}
\graphicspath{{}}
\includegraphics[width=5.5in]{undurraga2012_figures/SupplText/S6.png}
\caption[The phenotypic effects of ELF3-TR copy number variation are strongly background-dependent.]{The phenotypic effects of ELF3-TR copy number variation are strongly background-dependent. PCA of phenotypic data from all ELF3-TR alleles in both elf3-4 (Ws accession) and elf3-200 (Col accession) backgrounds. (A) Biplot of PC1 and PC2, graphically showing the contribution of phenotypes to PCs as black arrows. Note that for the biplot representation, PC1 and PC2 are transformed to the same scale (bottom and left axes), whereas phenotype contributions (in red) are allowed to differ in scale (top and right axes). Phenotypes are hypocotyl length in short and long days (SD hylen and LD hylen), DTF in short and long days (SD DTF and LD DTF), and FLN in long days (LD FLN). Wild-type plants are characterized by late flowering (large SD and LD DTF, many rosette leaves) and short hypocotyls (small SD and LD hylen), relative to elf3 loss-of-function mutants. Text in red represents a given allele in the Ws background (transgenics in elf3-4), and text in blue represents alleles in the Col background (transgenics in elf3-200). (B) PC1 and PC2. (C) PCA loadings for both back- grounds. hylen = hypocotyl length (mm).} \label{fig:app1fig6}
\end{figure}

\begin{figure}
\graphicspath{{}}
\includegraphics[width=6in]{f2_batches_badvsgood_111915.pdf} \label{tab:app1fig4}\\
\caption[elf3 mutants are not haploinsufficient between the Ws and Col backgrounds.]{We measured 8d hypocotyl length under short days in a series of $\mathrm{F_2}$ seedlings generated by various crosses. ``2012'' batches all involve a single Col individual as a parent (either male or female), which may be a spontaneous mutant. ``2013'' batches were generated subsequently and represent a variety of Col individuals as parents. "elf3-ws" is the elf3-4 mutant in the Ws background, ``elf3-col'' is the elf3-200 mutant in the Col background. Hypocotyls $\geq$10 mm are similar to elf3 null mutants, whereas WT hypocotyls are approximately 5-8 mm.} \label{fig:app1fig7}
\end{figure}


%%%% TABLES!!!!
%\begin{table}[htdp]
%\caption{default}
%\begin{center}
%\begin{tabular}{|c|c|}
%fail
%\end{tabular}
%\end{center}
%\label{default}
%\end{table}%

\section{Supporting Tables}

%\begin{multicols}{2}
\begingroup
\twocolumn
\footnotesize
%\begin{longtable}{|c|c|}
%\captionof{table}{}
\captionof{table}{ELF3-TR variation in diverse A. thaliana strains (ecotypes)}\label{tab:app1tab1}
\begin{supertabular}{|c c|}
%\caption{ELF3-TR variation in diverse A. thaliana strains (ecotypes)} \\
\hline
Ecotype  & ELF3 TR copy number \\
\hline 
%\topcaption{}

Ag-0 & 16 \\
Alc-0 & 17 \\
Algutsrum & 22 \\
An-1 & 19 \\
Ang-0 & 12 \\
Ba-1-2 & 19 \\
Ba-3-3 & 13 \\
Ba-4-1 & 17 \\
Bay-0 & 22 \\
Bg-2 & 19 \\
Bil-5 & 16 \\
Bil-7 & 16 \\
Blh-1 & 16 \\
Boo-2-1 & 14 \\
Bor-1 & 13 \\
Bor-4 & 13 \\
Br-0 & 23 \\
Bro-1-6 & 9 \\
Bs-1 & 12 \\
Bu-0 & 19 \\
Bur-0 & 23 \\
C24 & 9 \\
Can-0 & 20 \\
Cen-0 & 18 \\
CIBC-17 & 17 \\
CIBC-5 & 19 \\
Co & 13 \\
Co-1 & 9 \\
Col-0 & 7 \\
Cvi-0 & 9 \\
Dem-4 & 16 \\
Di-0 & 17 \\
Dra-3-1 & 16 \\
Drall-1 & 11 \\
Dralll-1 & 13 \\
Eden-1 & 11 \\
Eden-2 & 11 \\
Edi-0 & 16 \\
Eds-1 & 14 \\
Ei-2 & 15 \\
En-1 & 11 \\
Es-0 & 9 \\
Est-0 & 19 \\
Est-1 & 19 \\
Fab-2 & 14 \\
Fab-4 & 14 \\
Fei-0 & 19 \\
Ga-0 & 9 \\
Gd-1 & 14 \\
Ge-0 & 18 \\
GOT-22 & 16 \\
GOT-7 & 16 \\
Gr-1 & 21 \\
Gu-0 (=Gue-0) & 19 \\
Gul-1-2 & 11 \\
Gy-0 & 19 \\
H55 & 7 \\
Hi-0 & 16 \\
Hod & 12 \\
Hov-4-1 & 16 \\
Hovdala-2 & 16 \\
HR-10 & 14 \\
HR-5 & 19 \\
Hs-0 & 16 \\
Hsm & 11 \\
In-0 & 16 \\
Is-0 & 17 \\
Jm-0 & 11 \\
Ka-0 & 19 \\
Kas-1 & 16 \\
Kas-2 & 29 \\
Kavlinge-1 & 9 \\
Kent & 11 \\
Kin-0 & 19 \\
Kni-1 & 19 \\
Knox-10 & 16 \\
Knox-18 & 14 \\
Koln & 12 \\
Konchezero (N13) & 16 \\
Kondara & 14 \\
KZ-1 & 21 \\
Kz-13 & 14 \\
Kz-9 & 14 \\
Lc-0 & 19 \\
Ler-1 & 17 \\
Liarum & 16 \\
Lillo-1 & 21 \\
Lip-0 & 15 \\
Lis-1 & 15 \\
Lis-2 & 13 \\
LL-0 & 13 \\
Lm-2 & 19 \\
Lom-1-1 & 11 \\
Lov-1 & 16 \\
Lov-5 & 16 \\
LP2-2 & 26 \\
LP2-6 & 21 \\
Lu-1 & 13 \\
Lz-0 & 9 \\
Mr-0 & 24 \\
Mrk-0 & 13 \\
MS-0 & 15 \\
Mt-0 & 21 \\
Mz-0 & 14 \\
N6034 & 14 \\
N6187 & 19 \\
Na-1 & 22 \\
NC-6 & 14 \\
Nd-1 & 16 \\
NFA-10 & 18 \\
NFA-8 & 14 \\
Nok-3 & 16 \\
Nw-0 & 14 \\
Nyl-2 & 14 \\
Omo-2-1 & 22 \\
Omo-2-3 & 21 \\
Or-1 & 9 \\
Ost-0 & 11 \\
Oy-0 & 16 \\
Pa-1 & 9 \\
Per-1 & 11 \\
Petergof & 14 \\
Pi-0 & 12 \\
Pla-0 & 9 \\
PNA-10 & 16 \\
Pro-0 & 13 \\
Pu2-23 & 12 \\
Pu2-7 & 16 \\
Pu2-8 & 11 \\
Ra-0 & 12 \\
Rd-0 & 19 \\
REN-1 & 19 \\
Rev-1 & 19 \\
RMX-A02 & 16 \\
RMX-A180 & 18 \\
RRS-10 & 13 \\
RRS-7 & 21 \\
Rsch-4 & 11 \\
Rubezhnoe-1 & 14 \\
San-2 & 20 \\
Sanna-2 & 16 \\
Santa Clara & 8 \\
Sap-0 & 16 \\
Se-0 & 14 \\
Sf-1 & 9 \\
Shakdara & 14 \\
Sorbo & 14 \\
Spr-1-2 & 16 \\
Spr-1-6 & 11 \\
SQ-1 & 14 \\
SQ-8 & 15 \\
St-0 & 15 \\
Stu-1-1 & 16 \\
Stw-0 & 14 \\
Ta-0 & 21 \\
TAMM-2 & 9 \\
TAMM-27 & 9 \\
Te-0 & 10 \\
Tottarp-2 & 16 \\
Ts-1 & 13 \\
Ts-5 & 13 \\
Tsu-0 & 16 \\
Tsu-1 & 16 \\
Tu-0 & 16 \\
Ull-2-3 & 15 \\
Ull-2-5 & 11 \\
Uod-1 & 12 \\
Uod-7 & 21 \\
Van-0 & 13 \\
Var-2-1 & 9 \\
Var-2-6 & 9 \\
Vimmerby & 16 \\
Wa-1 & 16 \\
Wei-0 & 12 \\
Wil-2 & 14 \\
Ws-0 & 21 \\
Ws-2 & 16 \\
Wt-5 & 14 \\
Yo-0 & 16 \\
Zdr-1 & 13 \\
Zdr-6 & 17 \\
\hline
\end{supertabular}
%
\endgroup
%\end{multicols}

\begingroup
\onecolumn
\footnotesize

\begin{longtable}{|c c c c|}
\caption{Expression of ELF3 in transgenic lines (Ws background)} \label{tab:app1tab2}\\
\hline 
TR copy number & Line & Relative expression & CV(expression) \\	
\hline
Vector & V1-1 & 0.45 & 0.04 \\
0 & 0R1-3 & 4.18 & 0.01 \\
0 & 0R4-3 & 3.08 & 0.02 \\
0 & 0R5-3 & 1.58 & 0.01 \\
7 & 7R3-1 & 2.38 & 0.02 \\
7 & 7R5-2 & 0.86 & 0.01 \\
10 & 10R1-1 & 2.41 & 0.01 \\
10 & 10R2-2 & 1.49 & 0.01 \\
10 & 10R4-2 & 2.13 & 0.01 \\
11 & 11R1-3 & 2.17 & 0.03 \\
11 & 11R2-3 & 1.65 & 0.01 \\
11 & 11R8-1 & 0.81 & 0.02 \\
11 & 11R9-1 & 2.35 & 0.03 \\
14 & 14R1-1 & 0.99 & 0.01 \\
14 & 14R4-2 & 1.81 & 0.01 \\
15 & 15R1-2 & 2.68 & 0.01 \\
15 & 15R2-1 & 2.31 & 0.02 \\
15 & 15R3-1 & 3.40 & 0.01 \\
16 & 16R1-2 & 1.86 & 0.01 \\
16 & 16R4-2 & 2.14 & 0.01 \\
16 & 16R6-2 & 3.82 & 0.02 \\
16 & 16R7-2 & 2.35 & 0.02 \\
16 & 16R8-1 & 0.97 & 0.02 \\
16 & 16R10-3 & 0.89 & 0.01 \\
20 & 20R1-2 & 1.20 & 0.02 \\
20 & 20R2-1 & 0.95 & 0.02 \\
20 & 20R3-2 & 1.28 & 0.02 \\
21 & 21R2-2 & 0.97 & 0.01 \\
21 & 21R3-2 & 3.81 & 0.01 \\
21 & 21R5-3 & 3.69 & 0.01 \\
22 & 22R3-3 & 4.53 & 0.01 \\
22 & 22R6-1 & 1.35 & 0.01 \\
22 & 22R8-1 & 3.29 & 0.01 \\
23 & 23R2-1 & 3.30 & 0.02 \\
23 & 23R5-1 & 0.98 & 0.00 \\
29 & 29R3-2 & 1.29 & 0.00 \\
29 & 29R4-1 & 2.01 & 0.02 \\
29 & 29R5-2 & 1.21 & 0.02 \\
\hline
\end{longtable}

\begin{longtable}{|c c c c|}
\caption{Expression of ELF3 in transgenic lines (Col background)} \label{tab:app1tab3}\\
\hline 
TR copy number & Line & Relative expression & CV(expression) \\	
\hline
Vector & V1-1 & 0.05 & 0.00 \\
Vector & V1-1 & 0.05 & 0.00 \\
0 & 0R1-1 & 3.48 & 0.12 \\
0 & 0R3-1 & 2.54 & 0.32 \\
0 & 0R5-1 & 3.42 & 0.42 \\
7 & 7R3-1 & 2.33 & 0.20 \\
7 & 7R4-3 & 2.21 & 0.33 \\
7 & 7R5-1 & 1.44 & 0.03 \\
9 & 9R2-1 & 0.27 & 0.02 \\
9 & 9R4-1 & 0.73 & 0.10 \\
9 & 9R5-1 & 1.63 & 0.21 \\
10 & 10R1-2  & 1.74 & 0.01 \\
10 & 10R3-3  & 2.74 & 0.09 \\
10 & 10R7-3  & 2.04 & 0.25 \\
11 & 11R3-1 & 1.38 & 0.09 \\
11 & 11R5-4 & 1.52 & 0.16 \\
11 & 11R6-1 & 1.63 & 0.21 \\
11 & 11R7-1 & 0.61 & 0.03 \\
14 & 14R3-4 & 2.10 & 0.13 \\
14 & 14R5-4 & 2.24 & 0.00 \\
14 & 14R10-3  & 1.59 & 0.49 \\
15 & 15R2-2  & 1.59 & 0.27 \\
15 & 15R4-2 & 1.81 & 0.08 \\
15 & 15R8-1  & 0.53 & 0.34 \\
16 & 16R1-1 & 0.49 & 0.00 \\
16 & 16R2-4 & 2.01 & 0.12 \\
16 & 16R3-2 & 1.37 & 0.36 \\
16 & 16R4-2  & 1.97 & 0.17 \\
20 & 20R1-2 & 4.27 & 0.39 \\
20 & 20R2-3 & 0.81 & 0.02 \\
20 & 20R3-3 & 3.07 & 0.17 \\
20 & 20R5-3 & 3.49 & 0.57 \\
21 & 21R1-2 & 3.05 & 0.13 \\
21 & 21R3-2 & 1.25 & 0.04 \\
22 & 22R3-3 & 3.34 & 0.26 \\
22 & 22R4-2 & 1.37 & 0.35 \\
22 & 22R6-1 & 2.69 & 0.10 \\
22 & 22R7-2 & 2.59 & 0.54 \\
23 & 23R2-4 & 0.39 & 0.12 \\
23 & 23R4-3 & 1.79 & 0.03 \\
29 & 29R4-2 & 0.81 & 0.21 \\
29 & 29R11-1 & 3.75 & 0.60 \\
\hline
\end{longtable} 



\begin{longtable}{|c c|}
\caption{Primer sequences} \label{tab:app1tab4}\\
\hline
a: Cloning primers. \\
\hline
Primer Name/target & Sequence \\
\hline
Nar1 Sense & GAGATCTGATAATGAACCGGCGCCACAGCAACAGCAACAG \\
Nar1 Antisense & TGTTGCTGTTGCTGTGGCGCCGGTTCATTATCAGATCTC \\
Nco1 Sense & CTCAATATCACCCCGCCATGGGATTCCCACC \\
Nco1 Antisense & GGTGGGAATCCCATGGCGGGGTGATATTGAG \\
PolyQ Sense & CCCTTCCCATGGGATTCCCACCTCCTGGTAAT \\
PolyQ Antisense & TTTTGGGGCGCCGGTTCATTATCAGATCTCTG \\
0Q Sense & ACCATAATGAACCCATATTGTTCAAGCCC \\
 & CAATGAGCAAATGAACCAGTTTGGA \\
0Q Antisense & TCCAAACTGGTTCATTTGCTCATTGGG \\
 & GCTTGAACAATATGGGTTCATTATGGT \\
ELF3 Construct S & CAATAATGGTTTCTGACGTA \\
ELF3 Construct A & ACCAATGGTACTCAAAATAGTTTGGTCATACGG \\
 \\
 \hline
b: Real-Time PCR primers. \\
\hline
Target Gene/orientation & Primer \\
ELF3F & GACATTGATAATGATCGTGAATACAG \\
ELF3R & CTAATATACCCACAACATCATCGG \\
PIF5F & AGTCGGACCGAGTCATT \\
PIF5R & TCTTTGTTGTTCCTTCCATAGC \\
PRR9F & ATAAGCTGATGGAGAATGGC \\
PRR9R & TCCAAGCTCAGGACCAACA \\
UBC21F & GACCAAGATATTCCATCCTA \\
UBC21R & GTTAAGAGGACTGTCCG \\
 \\
 \hline
c: TAIL-PCR primers. \\
\hline
Name & Sequence \\
LB-0a & GCTGGACTTCAGCCTGCCGGTGCCGCCC \\
LB-1a & ACGATGGACTCCAGTCCGGCCCCCGTCAC \\
 & CGAAATCTGATGACCCCTAGAGTC \\
LB-2a & CGCGCGCGGTGTCATCTACTATGTTACTAGATC \\
\hline
\end{longtable}
\endgroup


\chapter{Supporting Chapter 3}
PFT1



\begin{figure}
\graphicspath{{}}
\includegraphics[width=5.5in]{pft1_resubmit_2014/pft1_figs1-01.jpg}
\caption[Structure of the PFT1 protein. ]{Structure of the PFT1 protein. Important features and domains are indicated. vWFA: van Willebrand Factor A domain, FH1: Formin homology 1 domain, VP16: VP16-like interaction domain, POLYQ: glutamine-rich domain including the polyglutamine encoded by the PFT1 STR. Also indicated is the difference between the two splice forms of PFT1 (minor splice form, PFT1.2, lacks a small exon), and the site of the T-DNA insertion in the pft1-2 mutant.}
\label{fig:app2fig1}
\end{figure}

\begin{table}
\caption{Number of repeat units in two polyglutamine regions encoded by trinucleotide repeats across eight  \emph{A. thaliana} strains.} \label{tab:app2tab1}
\begin{tabular}{| c c c |}
\hline
Strain & Number of PFT1 repeat units & Number of ELF3  repeat units \\
\hline
C24 & 90 & 9 \\
Can-0 & 90 & 20 \\
Col-0 & 88 & 7 \\
Cvi-0 & 89 & 9 \\
Ler-1 & 90 & 17 \\
Mt-0 & 88 & 21 \\
Tamm-2 & 88 & 9 \\
Ws-2 & 88 & 16 \\
\hline
\end{tabular}
\end{table}


\begin{table}
\caption{Encoded amino acid sequence of the repeat regions across PFT1 constructs used in this study, named for their approximate proportion of the length of the endogenous repeat. 1R is the endogenous sequence.} \label{tab:app2tab2}
\begin{tabular}{| c c |}
\hline
Allele name & Protein sequence in repeat region \\
\hline
0R & NLQ \\
0.34R & NLQQQQQQQQQQQQQQHQLTQLQHHHQQQQQ \\
0.5R & NQQQQQQQQLHQQQQQQQQIQQQQQQQQHLQQ \\
&QQMPQLQHHHQQQQQ \\
0.75R & NQQQQQQQQLHQQQQQQQQIQQQQQQQQHLQ \\
& QQQMPQLQQQQQQHQQQQQQQHQLTQLQHHHQQQQQ \\
1R & NQQQQQQQQLHQQQQQQQQIQQQQQQQQHLQQQQ \\
& MPQLQQQQQQHQQQQQQQHQLSQLQHHQQQQQQQQQ \\
& QQQQHQLTQLQHHHQQQQQ \\
1.27R & NQQQQQQQQLHQQQQQQQQIQQQQQQQQHLQQQ \\
& QMPQLQQQQQQHQQQQQQQHQLTQLQHHHQQQQQQQQ \\
& QQQQQHQLTQLQHHHQQQQQQQQQQQQQHQLTQLQHHH \\
& QQQQQ \\
1.5R & NLQHHHQQQLQQQQQQQLHQQQQQQQQIQQQQQQ \\
& QQHLQQQQMPQLQQQQQQQLHQQQQQQQQIQQQQQQQ \\
& QHLQQQQMPQLQQQQQQHQQQQQQQHQLSQLQHHQQQ \\
& QQQQQQQQQQHQLTQLQHHHQQQQQ \\
\hline
\end{tabular}
\end{table}


\begin{table}
\caption{Transgenic T\textsubscript{3} and T\textsubscript{4} A. thaliana lines used in this study. 3 biological replicates used to estimate expression.}  \label{tab:app2tab3}
\begin{tabular}{| c c c c c |}
% & & &  \\
\hline
Repeat Unit & Line Name & Repeat Region & Expression & Standard Error \\
 & & PCR Confirmed & Relative to Col-0 &  \\
\hline
1.5 & 1.5R3-3 & Yes & 0.96 & 0.12 \\
1.5 & 1.5R4-4 & Yes & 3.21 & 0.07 \\
1.5 & 1.5R8-1 & Yes & 2.32 & 0.19 \\
1.27 & 1.27R6-2 & Yes & 1.78 & 0.10 \\
1.27 & 1.27R13-2 & Yes & 0.89 & 0.05 \\
1.27 & 1.27R14-2 & Yes & 1.49 & 0.46 \\
1  & 1R1-2 & Yes & 0.90 & 0.05 \\
1  & 1R8-3 & Yes & 1.10 & 0.22 \\
0.75 & 0.75R1-4 & Yes & 1.92 & 0.19 \\
0.75 & 0.75R2-1 & Yes & 0.69 & 0.21 \\
0.75 & 0.75R10-2 & Yes & 0.92 & 0.27 \\
0.5 & 0.5R5-1 & Yes & 2.70 & 0.09 \\
0.5 & 0.5R6-4 & Yes & 0.66 & 0.29 \\
0.5 & 0.5R7-3 & Yes & 0.98 & 0.13 \\
0.34 & 0.34R1-2 & Yes & 0.63 & 0.05 \\
0.34 & 0.34R2-2 & Yes & 0.82 & 0.02 \\
0.34 & 0.34R9-1 & Yes & 2.85 & 0.49 \\
0 & 0R4-2 & Yes & 1.33 & 0.01 \\
0 & 0R8-2 & Yes & 0.94 & 0.06 \\
V & V3-1 & NA & 0.36 & 0.04 \\
\hline
\end{tabular}
\end{table}

\begin{table}
\caption{Primers used in this study.}  \label{tab:app2tab4}
\begin{tabular}{| c c |}
% & & &  \\
\hline
Target & Primer \\
\hline
pft1-2 allele F & ATTATTTGGGTGCTTTCTCATGGCC \\
pft1-2 allele R & TGGGCTTTCCCTGCATTTAAAACAG \\ 
UBC F & GACCAAGATATTCCATCCTA \\
UBC R & GTTAAGAGGACTGTCCG \\
PFT1 (cloning) F  & ATCAACAGGAATGGCTACATC \\
PFT1(cloning) R & TTGTTTGAGGACTAAAGGCATTAT \\
PFT1 (both splice forms) & GCAAACCATCGTCTCCGACTATC \\
PFT1 (both splice forms) & ccactccgttgtaccaagcaa \\
PFT1.1 (large splice form only) F & CAGGTCTTTCTGTGGCAGTGA \\
PFT1.1 (large splice form only) R & ccactccgttgtaccaagcaa \\
PFT1.2 (small splice form only) F & CAGAGGAACCCTGTTTCTACT \\
PFT1.2 (small splice form only) R & ccactccgttgtaccaagcaa \\
\hline
\end{tabular}
\end{table}

\chapter{Supporting Chapter 4}
STR review+mipstr


\chapter{Supporting Chapter 5}
elf3 epistasis


\section{Supporting Text}

\section{Supporting Figures}

\section{Supporting Tables}


\chapter{Supporting Chapter 6}
elf3 temp sensing

\section{Supporting Figures}

\section{Supporting Tables}


\chapter{Supporting Chapter 7}
\section{Supporting Text}
\label{app7:suptext}\subsection{Hsp90 paralog distribution}All bacterial Hsp90 paralogs were spread across multiple taxa, with gaps in their distribution (Figure S1C). hsp90A was widespread but particularly abundant in Proteobacteria, Clostridia, Actinobacteria, Chlorobi, and Chloroflexi. hsp90B was less common but dominant in Cyanobacteria and Bacteroidetes. hsp90C was relatively widespread but did not seem to be particularly enriched in any clade. While multiple Hsp90 paralogs could be observed in various species, hsp90A and hsp90B never co-occurred in the same species in our classification. This co-occurrence pattern and the distribution of hsp90C throughout the phylogeny suggest mostly vertical inheritance of hsp90A and hsp90B within clades and frequent horizontal transfers (and a potentially distinct functional role) of hsp90C. There are also a few instances of multiple copies of a single Hsp90 paralog within the same species (these are not displayed in Figure S1C). \subsection{Consistency of BayesTraits runs }The stochastic nature of the BayesTraits maximum-likelihood algorithm allows for variation in rate parameter estimates from run to run. We accordingly confirmed that the results presented in the main text are robust to such variation. Specifically, we ran BayesTraits on the full set of 4645 genes 100 times, applying a 10\% FDR threshold separately to each run, obtaining 100 sets of genes found to co-evolve with hsp90A. Examining these gene sets and their functional annotations, we found the results to be robust across runs. Specifically, the size of these sets ranged from 327 to 348 genes, with the vast majority of these genes (317) included in all sets. Functional enrichment was similarly consistent between the sets, with KEGG functional classes of flagellar assembly, bacterial motility proteins, and bacterial chemotaxis significantly enriched in all 100 runs. The bacterial secretion systems class was significantly enriched in 97 runs (and was just above the significance cutoff in the other 3). Considering this strong reproducibility between runs, in the main text we focused on the 327 genes that were found to co-evolve with hsp90A in at least 90\% of runs. These genes are referred to throughout the text as hsp90A-associated genes. We additionally evaluated the level of variation in estimated rate parameters. Overall, we found that rates are largely similar qualitatively between runs, with higher rates varying more from run to run than lower rates (see for example Figures 2C and 2D). Increasing the number of maximum-likelihood optimization attempts above the default did not appreciably affect this variability. Throughout the text we present the mean rates and standard deviations to communicate our estimated rates. To evaluate the accuracy of our ML-dependent approach, we also used BayesTraits Markov chain Monte Carlo (MCMC) mode with the reversible-jump option \cite{Pagel2006}, which allows for parameter reduction, to estimate our gain and loss rate parameters for a small subset of genes (see Methods). We found that ML-based predictions of rates are highly consistent with MCMC estimates (Spearmans rho = 0.91; p < 2.2 * 10-16).We similarly examined the effect of BayesTrait variation on our ability to consistently classify genes into specific co-evolutionary models. We again ran BayesTraits 100 times, generating each of the four distinct evolutionary models for all 327 hsp90A-associated genes (see Methods). We independently applied AIC for each gene and for each of these 100 replicates to determine which model fit each gene best in each run. Genes for which at least 90 of the 100 runs agreed on one of these four models were classified with this model. This scheme was able to classify 312 of the 327 hsp90A-associated genes into either the mutual dependence model (model 2; Methods) or one of the unidirectional dependence models (models 3 and 4). Specifically, all bacterial secretion genes and all flagellar genes were successfully classified (see main text and Table 1 for a detailed discussion of these genes). We used a similar approach to estimate the impact of run-to-run variation on the study of hsp90A co-evolution with organismal traits. The traits of host-association, multiple habitat preference, and pathogenicity were all found to co-evolve with hsp90A in all 100 runs, and the trait of terrestriality was found to co-evolve with hsp90A 99 times of 100. hsp90A was always found to be dependent on multiple habitat preference and pathogenicity, whereas both terrestriality and host-association had a mutually dependent relationship with hsp90A.Finally, since our client prediction method is based on estimated rates (see Methods), we evaluated the sensitivity of our method to variation in rate estimation. To this end, we recalculated putative client index (PCI) values independently for each of the 100 runs above. For each run we recorded the 20 genes with the lowest PCI values and examined the variation observed from run to run in this set of potential clients. Overall, only 26 unique genes were identified as potential clients by any run, with 12 genes identified as clients in all 100 runs and 18 genes identified as clients in at least 90 of the runs, indicating high reproducibility. In the main text, we define these 18 genes found in at least 90 runs as putative clients and report their mean PCI values (see Table 2). Figure 3 illustrates the distribution of mean PCI values across all 327 hsp90A-associated genes.  

\subsection{Robustness of co-evolutionary associations to choice of phylogeny}One potential caveat of our analysis is its strong dependency upon a specific phylogeny. Accordingly, while the Ciccarelli tree used throughout our analysis is well-established, commonly used, and believed to be robust, we examined whether our results hold with a different phylogenetic tree. To this end, we repeated the analysis described in the main text using a significantly larger tree (including 797 species) that was constructed by a fundamentally different method [30] (termed here `Yarza tree). We find that the p-values obtained using this larger tree are correlated with those reported in the main text using the Ciccarelli tree (p < 2.2E-16; Spearman correlation test), but are generally much smaller (requiring the use of a smaller FDR threshold). While, the set of hsp90A-associated genes found in the Yarza tree is generally larger than the set of genes found in the Ciccarelli tree, there is a significant overlap between the sets (Table S1). Moreover, examining functional enrichment in the Yarza-derived hsp90A-associated gene set, we find the same set of functional categories as those found in the Ciccarelli tree across a variety of FDR thresholds (Tables S1). \subsection{Effect of alternate hsp90 paralogs on co-evolution}There are 20 species in the Ciccarelli tree which contain either hsp90B, hsp90C, or both. It is possible that these species exert a confounding effect upon our attempts to detect co-evolution with hsp90A. To address this possible confounding effect, we performed the following analysis. We first pruned the Ciccarelli tree to remove the 20 species (leaving a total of 128 species), and repeated our analysis looking for genes that co-evolve with hsp90A in this pruned tree among the 4399 genes that passed our filters in this reduced tree. Reducing the number of species reduced the statistical power of our analysis (a phenomenon that we also observed in using a larger tree, see above), so it was necessary to reduce the stringency of our FDR threshold to 25\% to obtain a co-evolving gene set of similar size (301 genes vs. 327 genes with the full set). This gene set showed almost exactly the same functional enrichments as the set obtained using the full tree (Flagellar assembly: 19/39 genes, bacterial motility proteins: 27/108 genes, and bacterial secretion: 16/64 genes; compare to Table 1), with very similar levels of enrichment. At our stringent 5\% FDR cutoff for detecting enrichment among all possible functions, chemotaxis was not significantly enriched, but nonetheless the result was very similar to that with the full set (7/26 genes, p = 0.00037). We conclude that inclusion or exclusion of species containing the alternate hsp90 paralogs did not meaningfully bias our analysis, though reduction of the phylogeny sample size reduces our power to detect associations in a genome-wide fashion.

\section{Supporting Figures}

\begin{figure}[H]
\graphicspath{{}}
\includegraphics[width=5.5in]{htpg/fig_s1_020713.png}
\caption[Phylogenetic clustering of bacterial hsp90 paralogs.]{Phylogenetic clustering of bacterial hsp90 paralogs. (A) Neighbor-joining phylogeny of 897 bacterial Hsp90 amino acid sequences. Groups Hsp90A, Hsp90B, and Hsp90C as defined by Chen et al. \cite{Chen2006} are illustrated. (B) Consensus neighbor-joining tree for 100 bootstraps with clades collapsed to highlight deep branch structure. Bootstrap support for each branch is displayed and is also reflected by the branch lengths. One species (ZIN, representing Hsp90 from the organism Candidatus Zinderia insecticola CARI), never grouped within the other divisions shown, and was excluded from our analysis. The branch separating Hsp90B and Hsp90C from the Hsp90A clades is present in 99/100 bootstrap trees. (C) Hsp90A, B, and C presence/absence patterns mapped onto a 16S/23S rRNA phylogeny of 797 bacterial species \cite{Munoz2011} (see Appendix 7 Text). Branch lengths are ignored for ease of display.}
\label{fig:app7fig1}
\end{figure}

\begin{figure}[H]
\graphicspath{{}}
\includegraphics[width=5.5in]{htpg/fig_s2.jpg}
\caption[Co-evolutionary gain and loss rates of all hsp90A- associated flagellar genes.]{Co-evolutionary gain and loss rates of all hsp90A- associated flagellar genes. The layout of each diagram is similar to that used in Figure 2.}	% need a ref
\label{fig:app7fig2}
\end{figure}

\begin{figure}[H]
\graphicspath{{}}
\includegraphics[width=5.5in]{htpg/fig_s3.png}
\caption[Co-evolutionary gain and loss rates of all hsp90A- associated secretion genes.]{Co-evolutionary gain and loss rates of all hsp90A- associated secretion genes. The layout of each diagram is similar to that used in Figure 2.}	% need a ref
\label{fig:app7fig3}
\end{figure}

\begin{figure}[H]
\graphicspath{{}}
\includegraphics[width=5.5in]{htpg/fig_s4.png}
\caption[Box plots of the rates of gain and loss of all hsp90A- associated secretion genes.]{Box plots of the rates of gain and loss of all hsp90A- associated secretion genes (n = 16). See also Figure 2D.}	% need a ref
\label{fig:app7fig4}
\end{figure}

\begin{figure}[H]
\graphicspath{{}}
\includegraphics[width=5.5in]{htpg/fig_s5.png}
\caption[The htpG(E34A) mutant strain shows decreased motility/chemotaxis. ]{The htpG(E34A) mutant strain shows decreased motility/chemotaxis. (A) Plates were inoculated with the same amount of wild-type MG1655 (top), the $\Delta$ htpG mutant (bottom left) and the htpG(E34A) mutant (bottom right) cells and incubated at indicated temperatures for 6 hr. (B) Relative motility of $\Delta$ htpG and htpG(E34A) mutants, compared to wild type, at indicated temperatures, quantified by the diameter of the outer rings of spreading colonies. Error bars indicate standard errors from two replicates.}	% need a ref
\label{fig:app7fig5}
\end{figure}

\begin{figure}[H]
\graphicspath{{}}
\includegraphics[height=5.5in]{htpg/fig_s6.png}
\caption[HtpG interactions with FliN and CheA are dependent on the DnaJ/CbpA/DnaK chaperone system.]{HtpG interactions with FliN and CheA are dependent on the DnaJ/CbpA/DnaK chaperone system. Acceptor photobleaching FRET was measured between HtpG and FliN (A) or CheA (B). In each panel, HtpG(E34A) (top row) and wild- type HtpG (bottom row) were assayed, and experiments were performed in both WT (left column) and DdnaJDcbpA (right column) backgrounds. Y-axes are normalized in each case to the mean CFP signal before bleaching (first 45 s). Photobleaching begins at ,50 s and lasts for 20 s (indicated by black bar). FRET interaction is indicated by a post-photobleaching increase in CFP signal above pre-photobleaching CFP signal (as observed in all experiments in the WT background).}	% need a ref
\label{fig:app7fig6}
\end{figure}


\section{Supporting Tables}

\begin{table}[H]
\caption{Comparable results in Ciccarelli and Yarza trees across FDR thresholds.}\label{tab:app7tab1}
\begin{tabular}{| c c c c |}
\hline
Yarza FDR threshold & FDR=0.1\% & FDR=0.05\% & FDR=0.01\% \\
\hline
Genes passing threshold in Yarza tree & 966 & 783 & 441 \\
\hline
Enriched function [KEGG Class] with & & \\
p-values across FDR thresholds & & \\
\hline
Bacterial secretion system [PATHko03070] & 3.20E-13* & 1.30E-14* & 3.42E-16* \\
Secretion System [BRko02044] & 2.44E-07* & 2.42E-08* & 1.03E-06* \\
Bacterial motility proteins [BRko02035] & 1.46E-06* & 3.38E-06* & 0.0096 \\
Flagellar assembly [PATHko02040] & 2.73E-10* & 1.17E-08* & 0.37 \\
\hline
Gene overlap with Ciccarelli tree & & \\
\hline
Genes in common & 172 & 153 & 92 \\
Hypergeometric p-value for overlap & 6.67E-48* & 1.13E-48* & 1.38E-31* \\
\hline
\end{tabular}
\end{table}

\begin{table}[H]
\caption{\emph{E. coli} strains and plasmids used in this study.} \label{tab:app7tab2}
\begin{tabular}{| c c c c |}
\hline
Strain & Relevant genotype & Background & Reference or source \\
\hline
RP437 & wild type & - & Parkinson and Houts 1982 \\
VS116 & $\Delta$ flhC & RP437 & Sourjik and Berg 2000 \\
MG1655 & wild type & - & Blattner et al. 1997 \\
HL23 & $\Delta$ htpG & MG1655 & This study  \\
HL24 & htpG::htpG(E34A) & MG1655 & This study \\
MC4100 & wild type & - & Matthias Mayer, personal gift \\
HL25 & $\Delta$ dnaJ $\Delta$ cbpA & MC4100 & Matthias Mayer, personal gift \\
\hline
\end{tabular}
\end{table}

%\addtocounter{table}{-1}
%\captionsetup[table]{list=no}

\begin{table}[H]

\begin{tabular}{| c c c c |}
\hline
Plasmid & Relevant genotype & Background & Reference or source \\
\hline
pHL13 & FliN-CFP expression & pDK79 & Li \& Sourjik, 2011 \\
pHL24 & HtpG-YFP expression & pTrc99a & Li \& Sourjik, 2011 \\
pHL35 & HtpG(E34A)-YFP expression & pTrc99a & This study \\
pHL52 & HtpG(E34A)-CFP expression & pDK79 & This study \\
pHL70 & HtpG-CFP expression & pDK79 & Li \& Sourjik, 2011 \\
pDK14 & CFP-CheW expression & pDK79 & Kentner et al, 2006 \\
pDK19 & CheR-YFP expression & pTrc99a & Kentner \& Sourjik, 2009 \\
pDK29 & CheA-CFP expression & pDK79 & This study \\
pDK30 & CFP-CheA expression & pDK79 & This study \\
pDK36 & YFP-CheA98-655 & pTrc99a & Kentner \& Sourjik, 2009 \\
 & (YFP-CheAS) expression & & \\
pDK49 & CheW-CFP expression & pDK79 & Kentner \& Sourjik, 2009 \\
pDK90 & YFP-CheA509-655 expression & pTrc99a & Kentner et al, 2006 \\
pVS18 & CheY-YFP expression & pTrc99a & Sourjik and Berg, 2002 \\
pVS64 & CheZ-YFP expression & pTrc99a & Liberman et al, 2004 \\
pVS99 & CheB-YFP expression & pTrc99a & This study \\
pVS108 & CFP-CheA156-655 expression & pBAD33 & This study \\
pVS109 & CFP-CheA259-655 expression & pBAD33 & This study \\
pVS129 & CFP expression & pTrc99a & This study \\
pVS132 & YFP expression & pTrc99a & This study \\
\hline
\end{tabular}
\end{table}

\begin{table}[H]
\caption{Swarming assay results at 34\degree C and 42\degree C.}\label{tab:app7tab3}

\begin{tabular}{| c c c c |}
\hline
Temperature & Strains & Center (\%) & Outer edge (\%) \\
\hline
34\degree C & MG1655 & 353 & 642 \\
 & MG1655$\Delta$htpG & 653 & 362 \\
42\degree C & MG1655 & 413 & 632 \\
 & MG1655$\Delta$htpG & 593 & 372 \\
\hline
\end{tabular}
\end{table}

\begin{table}[H]
\caption{Acceptor photobleaching FRET interactions of chemotaxis components with HtpG(E34A). ++: strong interaction, +:weak interaction, -:no interaction, ND:not done..}\label{tab:app7tab4}

\begin{tabular}{| c c c |}
\hline
Chemotaxis component & HtpG(E34A) FRET in WT & HtpG(E34A) FRET in $\Delta$flhC \\
\hline
CheW-FP & ++ & - \\
FP-CheW & ++ & + \\
CheZ-FP & ND & ++ \\
CheY-FP & ND & + \\
CheR-FP & ND & + \\
CheB-FP & ND & + \\
CheA-FP & ++ & ++ \\
FP-CheA & + & ++ \\
FP-CheA98-655 (CheAs ) & ++ & ++ \\
FP-CheA156-655 & ++ & ++ \\
FP-CheA259-655 & ND & ++ \\
FP-CheA509-655 & ND & - \\
\hline
\end{tabular}
\end{table}


\begin{table}[H] 
\caption{hsp90A presence and absence is associated with organismal traits in bacteria.}\label{tab:app7tab5}

\begin{tabular}{| c c c |}
\hline
Trait & p-value & Number of species with annotations for trait \\
\hline
Pathogenicity & 0.036 & 140 \\
Host-associated & 0.046 & 146 \\
Multiple habitats & 0.0057 & 146 \\
Terrestrial & 0.045 & 146 \\
\hline
\end{tabular}
\end{table}


\chapter{Supporting Chapter 8}
\section{Supporting Text}
\subsection{Inference of gains and presence of genes on branches of the tree.}To estimate the probability that specific genes were gained or present on each branch of the tree, we chose a simple heuristic, based on the joint probability of the states of the ancestor and descendant nodes (Methods). We chose this approach because we are not concerned with any gain, but rather with gains that are retained until the end of a branch. For example, any gain at all is to be expected at some rate more or less without regard to genome content of the host, due to phage infection or DNA in the environment. However, given that the vast majority of these gains are followed closely by losses (Baltrus 2013), they are not as biologically interesting as genes gained and retained adaptively, and they are also mostly unobserved. Additionally, our approach allows us to consider the probability of steady presence across a branch. We considered the average reconstruction at each node to compute the probability of gain or presence of genes on branches, rather than summing across each possible reconstructed scenario in the stochastic mapping procedure (for instance weighted by the likelihood of each possible scenario). While using all possible mappings could, in principle, reduce the numerical error of our probability estimates, it would entail an onerous and potentially intractable computation. Moreover, the biological (Figure ~\ref{fig:hgtfig2}) and statistical (Figure ~\ref{fig:hgtfig5,fig:app8fig9}) validations we have performed suggest that our results are robust. Our method of inferring gains is also different from the probabilities of gains (or, similarly, the expected number of gains) that are computed by the gainLoss software (Cohen and Pupko 2010), using a previously-developed continuous-time Markov chain (CTMC) model to count the number of gains on each branch (Minin and Suchard 2008).  These models solve the problem of counting the number of one-way transitions between two states (say, presence and absence) given transition rates, states at the start and end of the interval, and a set amount of time in the interval. Thus, the CTMC implemented in gainLoss is capable of estimating the expected number of gains of a given gene on a given branch, with knowledge of gain and loss rates. However, this approach can lead to problematic cases in which a gene can be absent in ancestor and descendant nodes, and yet, given a very long branch, is inferred to be gained on this branch. While such scenarios may have statistical support, in practice they are very hard to interpret and compare to other events that more obviously support a gain. Given the presence of Archaea in our phylogeny, which are a dramatically divergent outgroup, this was a cause for concern. Indeed, the CTMC estimated that the median gene was gained more than twice along the long branch connecting Archaea to Bacteria, with some genes gained more than 10 times on this branch alone (data not shown). This result is almost certainly artefactual, but has the potential to substantially skew the overall appraisal of gains for a given gene. For these reasons and those stated above, we chose to ignore the gainLoss CTMC estimates in favor of the less sophisticated but more interpretable gain/presence inference method described above and in Methods.\subsection{Gain/loss ratio analysis.}A consistent feature of prokaryotic genome evolution is the predominance of DNA loss over gain, or ``deletional bias'' (Mira et al. 2001; Kuo and Ochman 2009). One previous study, for example, found that the gain to loss ratio in prokaryotes varied widely across genomes, ranging approximately from 0.07 to 0.9, with most genomes exhibiting a ratio between 0.2 and 0.5. Accordingly, a reliable ancestral reconstruction and gain/loss inferences should exhibit an excess of gene losses relative to gene gains. The gainLoss program used in our study addresses this problem in part by setting prior distributions on gain and loss rates based on the average prevalence of genes in genomes at the tips of the tree, such that losses tend to dominate (Cohen and Pupko 2010). For our data, the mean of the rate prior distribution was 0.36 for gains and 1.38 for losses, corresponding to a 0.26 ratio, which is in line with previous estimates. These rates were then used in an iterative expectation-maximization model to infer ancestral genome reconstructions on the tree while optimizing these rates and other parameters. Following optimization, the corresponding rates for gains and losses were found to be 0.80 and 3.86, corresponding to an even stronger deletional bias of 0.20. After ancestral reconstruction and gain/loss inference by the heuristic outlined in Methods, we found that the mean number of gains for a gene along the tree was 13.9, whereas the corresponding mean number for losses was 24.9, suggesting a ratio of 0.56. The distribution of losses is also substantially right-shifted relative to gains (Figure ~\ref{fig:app8fig1}). Furthermore, gain and loss counts were significantly correlated ($\rho = 0.75$, p < 10-15; Pearson correlation test), indicating that frequently gained genes are also frequently lost. Combined, these finding suggest that our model indeed strongly penalizes losses, and that the actual gain to loss ratio reflects the expected excess of losses.\subsection{Simulation of gene gain/loss evolution.}Previous attempts to use the gainLoss software to make inferences about horizontal gene transfer and detect coevolution used a parametric bootstrapping approach, simulating the evolution of genes to obtain null expectations for testing hypotheses (Cohen et al. 2011, 2012). While the use of exact parametric methods to estimate this null distribution is possible in principle (Maddison 1990), these methods rely upon a single binary reconstruction of ancestral states. Clearly, our probabilistic reconstruction is unsuited for such an analysis. Again, one could in principle enumerate all possible reconstructions, and estimate the null distribution exactly as a weighted sum across each reconstructions, but developing this method for large trees lies outside the scope of this paper. In our simulations, we therefore followed the example of others with certain modifications. The simulation procedure implemented in the gainLoss program was too memory-intensive to be feasible for a sufficiently large number of genes. Consequently, we took the gain and loss rates inferred by gainLoss for the real genes and used their distribution to simulate the evolution of genes using the function rTraitDisc() in the APE library. Briefly, we fit gamma distributions to the rates of gain and the rates of loss across all genes, and used the resulting parameters to define sampling distributions for gain and loss rates of simulated genes (see Methods). We then used the approach described in Methods to infer the probability of gain on each branch. We found that using these distributions inferred relatively few gains compared to the gains of observed genes (compare Figures ~\ref{fig:app8fig2}A and ~\ref{fig:app8fig2}C). We speculated that the rate mixture model employed by gainLoss has difficulties accommodating the upper tail of the distribution of gain rates (roughly, those genes gained >50 times in this tree), given that the vast majority of genes are gained relatively few times (Figure ~\ref{fig:app8fig2}A). Consequently, we adjusted the shape parameters of the gain and loss rate distributions heuristically to find values that gave distributions of simulated gains that included genes that are gained sufficiently many times. We found that multiplying the shape parameter of the gain rate by 3 and the shape parameter of the loss rate by 1.5 gave reasonably wide distributions of gains among simulated genes (Figure ~\ref{fig:app8fig2}E). It is important to note that the shape of the distribution from which rates are drawn does not affect the simulated evolution of a given gene with single sampled gain and loss rates. Furthermore, because we are not using the entire distribution of simulated genes but only those most appropriate to each gene as a null distribution, any differences in the distributions of gain counts between simulated and real genes are unlikely to affect results.\subsection{Robustness of gain events inference to analytic method.}To assess the robustness of our gain inference approach, we set out to compare the gain events inferred by our stochastic mapping-based method to horizontally transferred genes inferred by a reconciliation-based method (Jeong et al. 2015). While these two methods are likely to yield somewhat different results, we wished to confirm that they still agree on a substantial fraction of the inferred gain events (Ravenhall et al. 2015). To this end, we used a recently published database of horizontally transferred genes inferred by a well-established sequence-based reconciliation tool \cite{Jeong2015}. Since this database provides information on horizontally transferred genes detected in extant species, we specifically examined whether the genomes of extant species that are descendants of a branch on which a specific gene was inferred to be gained by our method were indeed more likely to be identified as having acquired this gene by HGT according to reconciliation. Notably, since data in the HGT database was not readily accessible, we limited our comparison to a small number of key genes (including, for example, rbsS, the RuBisCO small subunit discussed in our paper; and see Table ~\ref{tab:app8tab1}). Indeed, we found that extant species that are descendants of the 8 rbsS gain events inferred by our method were significantly more likely to have this gene identified as horizontally transferred compared to other species (24 out of 31 vs. 30 out of 2441 for descendants vs. not descendants respectively; odds ratio = 275.5, p < 10-32, Fishers exact test). Moreover, of the 8 rbsS gain events, in 6 cases at least one descendant had this gene identified as horizontally transferred by reconciliation, suggesting that the high odds-ratio above is not simply the outcome of just one or two gain events with numerous descendants (and in fact, in these 6 cases all descendants had the gene identified by reconciliation). This extremely strong association between gains inferred by the two methods points to a high level of agreement between the two approaches. Analyzing several additional genes with many associated PGCEs revealed overall high levels of agreement between the two methods (Table ~\ref{tab:app8tab1}). One apparent exception was the kpsT gene, which showed relatively low agreement between our method and reconciliation. Interestingly, however, we found substantial evidence of acquisition of other components of the kps operon for most kpsT gains predicted by stochastic mapping (in particular kpsM, which is immediately adjacent to kpsT in the kps operon). This operon has been gained by HGT in various pathogenic \emph{E. coli} (Schneider et al. 2004), as found also by stochastic mapping.   \subsection{Power of the PGCE detection method.}One of our observations is that there are weak relationships between the prevalence of a gene, how often it is gained, and its in- and out-degrees in the PGCE network (Figure ~\ref{fig:app8fig5}). Given that these values define the null distributions that we use to infer PGCEs, it was possible that our analyses are less sensitive for certain values of these parameters. We considered to what extent a lack of power was affecting our results with a simple power analysis. For genes i and j, the maximum observable value $C_{ij}$ counting the gains of j in the presence of i is $min(p_i, g_j)$, representing respectively the prevalence of gene i and the number of gains of gene j. For a range of values of these parameters $(p_i, g_j)$, we compared this maximum potential observation to the null distribution from parametric bootstrapping appropriate to these parameter values. This represents the most extreme possible test statistic between the two genes for these parameter values, so in each case the null hypothesis should be rejected if there is sufficient power. We found that power varied substantially across various values of $(p_i, g_j)$ (Figure ~\ref{fig:app8fig3}A). Specifically, we were incapable of detecting associations for any combination involving the most-prevalent genes or the least-gained genes. This is unsurprising, given that noise is expected to be high for the former, and signal to be low for the latter. Considering our observed distribution of p-values (Figure ~\ref{fig:app8fig3}B), we find the expected spike in frequency near p = 0 (indicating true positive dependencies), but also an unexpected spike in frequency near p = 1, indicating that our parametric bootstrapping test is underpowered due to the sparsity of gains, as suggested by power analysis (Figure ~\ref{fig:app8fig3}A). Consequently, there are likely to be many more PGCEs than we detect in this study. Notably, if we relax our FDR threshold from 1\% to 5\% in inferring PGCEs, we increase the raw number of edges in our network more than ten-fold (from 8,415 to 86,719). We chose to proceed with the more stringent threshold to focus on the most confident PGCEs, but we use this example to highlight the very large potential for PGCEs structuring genome evolution in prokaryotes. \subsection{Processing and analysis of the PGCE network.}After inferring a PGCE network, we post-processed this network to both ease further analysis and to remove potentially spurious edges. First, we removed edges such that the network became a directed acyclic graph (DAG). DAGs are relatively easy to analyze and interpret topologically. We found only one cycle-inducing edge: an obviously spurious self-edge (for gene K07218). The absence of non-spurious cycles may be initially surprising, but can be explained by the relatively small number of genes with in-edges (less than one-third of genes in the network) and the anti-correlation of in-degree and out-degree across genes (Figure ~\ref{fig:app8fig5}E). To evaluate whether the lack of cycles is attributable to degree distribution, we randomly rewired the DAG five times while preserving degree distribution, and in each of these five cases the result was still a DAG. This analysis indicates that this acyclic topology is a simple consequence of degree distribution, rather than a biological property of specific PGCE relationships. Together, these results indicate that few cycles are expected for a network with such properties. However, one might still expect some number of true cycles from a biological point of view, even if the network itself is biased against them. We believe that such cycles likely exist, but we do not detect them because of our relatively low power, and the stringency of our threshold for assigning edges (Figure ~\ref{fig:app8fig3}, see above section). Next, we removed potentially spurious edges in the network that might have been introduced by indirect transitive effects. For example, if gene A encourages the gain of gene B, and gene B encourages the gain of gene C ($A\rightarrow B\rightarrow C$), we might also infer that there is a direct $A \rightarrow C$ PGCE, even if such a PGCE does not actually exist. Consequently, we performed a transitive reduction of our DAG to obtain a minimal equivalent graph (Hsu 1975), or a DAG with all potentially indirect interactions (such as the $A \rightarrow C$ example above) removed. While potentially removing true PGCEs, we thus enrich our PGCE network for the most confident interactions. This procedure removed 186 potentially indirect PGCEs. It is this DAG, with all cycles and indirect edges removed, that we used for all downstream analyses.The degree distributions for this network indicated that a slight majority of genes (nodes) are disconnected, and we omitted these genes from further analyses. Furthermore, the distribution of in-degrees was more unequal than that of out-degrees across nodes (Figure ~\ref{fig:app8fig5}A, B). The degree distributions showed weak relationships with the prevalence and gain count of genes, but these do not appear to be primary determinants of network structure (Figure ~\ref{fig:app8fig5}C, D). \subsection{Dependencies among pathways.}The urtA-rbsL PGCE (Figure ~\ref{fig:hgtfig3}) highlighted the potential importance of inter-pathway PGCE dependencies. To understand the structure of such pathway-pathway dependencies, we tested for associations between genetic pathways within the PGCE network, compared to a null distribution of rewired networks. We detected 93 pathway-pathway dependencies (each p < 0.001, compared to the rewired null distribution), which we modeled as a directed network among 65 pathways (Figure ~\ref{fig:app8fig6}). Unlike the PGCE network, the pathway-pathway dependency network has many cycles. Related pathways showed many dependencies and clustered with each other, most strikingly for the metabolism of aromatic compounds. Consequently, we expect that PGCE dependencies, rather than only representing one-to-one interactions between genes, also reflect functional relationships between whole genetic pathways.\subsection{Algorithms.}\paragraph{Feedback arc set (FAS) identification algorithm} (Hausmann and Korte 1978; Hassin and Rubinstein 1994).\begin{enumerate}
\item Start with an empty DAG and an empty FAS; \item Select a random edge E from our PGCE network, add it to the DAG; \item If adding E to the graph adds a cycle, remove E again and add it to the FAS, else accept E in the DAG; \item If there are more edges that are neither in the DAG nor in the FAS, go to 2
\end{enumerate}

\paragraph{Transitive reduction of a DAG algorithm (Hsu 1975).}
\begin{enumerate}
\item Convert the network into an adjacency matrix representation; \item Convert the adjacency matrix into a path matrix; \item Remove all edges in the path matrix that can be explained by other paths, by iterating over all groups of 3 nodes.
\end{enumerate}

\paragraph{Topological sort with grouping algorithm} (Knuth 1973)We used the following procedure to perform a topological sort of a DAG: 
\begin{enumerate}
\item Initialize the rank count with ``rank'' = 1; \item Identify the set of nodes in the DAG with in-degree = 0 (these occupy the first position in a sort); \item Label these nodes with the current ``rank'' (1 in the first step); \item Remove these nodes and their edges from the DAG (some new nodes will now have in-degree = 0; \item if there are still nodes in the DAG, increment ``rank'' by 1 and go to step 2. \end{enumerate}
The resulting labeled groups constitute the ordered ranks of the topological sort. 

\subsection{gainLoss program parameters}The following are the gainLoss parameters used to generate the principal data reported in the paper. We omitted several parameters (e.g., paths to files) to reduce confusion, but the complete parameter file can be found as Supplemental File S2.
\begin{table*}[h]
\begin{tabular}{ c c c }
\_printPij\_t & 1 & \\\_printL\_of\_Pos & 1 & \\
\_calculateAncestralReconstruct & 1 & \\
\_printAncestralReconstructFullData & 1 & \\
\_printExpPerPosPerBranchMatrix & 1 & \\
\_printTree & 1 & \\
\_optimizationLevel & mid & \\
\_rateDistributionType & GAMMA & \\
\_performOptimizationsBBL & 1 & \\
\_performOptimizations & 1 & \\
\_numberOfGainCategories & 3 & \\
\_numberOfLossCategories & 3 & \\
\_numberOfRateCategories & 3 & \\
\_maxNumOfIterationsManyStarts & 3 & \\
\_calculateRate4site & 1 & \\
\_calculeGainLoss4site & 1 & \\
\_gainLossDist & 1 & \\ 
\_calculeGainLoss4site & 1 & \\
\_printLikelihoodLandscapeGainLoss & 1 & \\
\_printPij\_t & 1 & \\
\end{tabular}
\end{table*}



\newpage
\section{Supporting Figures}

\begin{figure}[H]
\graphicspath{{}}
\includegraphics[width=5.5in]{HGT_REVISION_2016/figures/gainvsloss_022016_figs1} 
\caption[Gene losses outnumber gene gains.]{Gene losses outnumber gene gains. Each of the 5801 genes in the ancestral reconstruction is plotted according to its number of losses and gains. Dashed line indicates expected values if gains and losses were equally frequent. Gain and loss counts represent the expected number of branches experiencing gain and loss, respectively, for the gene in question. PCC: Pearson correlation coefficient.}	% need a ref
\label{fig:app8fig1}
\end{figure}
%\newpage

\begin{figure}[H]
\graphicspath{{}}
\includegraphics[height=7in]{HGT_REVISION_2016/figures/hgt_figs2_022016.pdf} 
\caption[Comparison of evolution of real genes with genes with simulated evolution under various models. ]{}	
\label{fig:app8fig2}
\end{figure}

\addtocounter{figure}{-1}
\captionsetup[figure]{list=no}
\captionof{figure}{Distributions of total gains (A) and prevalence (B) estimated for  real genes by the gainLoss program. gainLoss rate estimates lead to underestimation of gains (C) and prevalence (D) in the tree: gene gain counts across 104 genes simulated according to gain/loss rates directly estimated by gainLoss for empirical genes. Gene gain (E) and prevalence (F) counts across genes simulated for use in null distributions. Red (gain) and blue (prevalence) line plots indicate, for each value of gain count or prevalence, the absolute difference of the least similar gene in its null distribution from that value (maximum deviance). For instance, in (E), a gene with 40 gains will be compared to a null distribution of simulated genes with as few as 39 gains and as many as 41 gains (deviance of one). Relative to (A) and (B), parameters of the underlying distributions of gain and loss rates were heuristically adjusted to provide acceptable coverage of the gain/prevalence values observed for empirical genes in (E) and (F). }
\captionsetup[figure]{list=yes}
\newpage

\begin{figure}[H]
\graphicspath{{}}
\includegraphics[width=6in]{HGT_REVISION_2016/figures/figure_s3_092215-01.pdf} 
\caption[Some regions of the parameter space are underpowered to detect PGCEs.]{Some regions of the parameter space are underpowered to detect PGCEs. (A) Power analysis of the parametric bootstrapping hypothesis test for detecting PGCEs. X and Y axes represent, respectively, total prevalence and total gains for a hypothetical pair of genes with a strong PGCE (maximum observable test statistic). Colors represent the (log10-scaled) minimum possible p-value that can be attained for such a gene pair using the relevant null distribution of simulated genes. Areas that are not white/pale yellow are underpowered for detecting PGCEs. (B) The distribution of empirical p-values observed for testing hypotheses of no PGCE in the evolution of pairs of genes, according to parametric bootstrapping. The spike at p = 1.0 in (B) indicates that sparsity in the data detracts from power, as predicted in (A), even after filtering pairs of genes with Cij <= 1.}	% need a ref
\label{fig:app8fig3}
\end{figure}
\newpage

\begin{figure}[H]
\graphicspath{{}}
\includegraphics[width=6in]{HGT_REVISION_2016/figures/figs4_092215-01.pdf} 
\caption[A global network of directional dependencies between prokaryotic genes (PGCEs)]{A global network of directional dependencies between prokaryotic genes (PGCEs). Node size is scaled to total edge count for each node (see also Figure ~\ref{fig:app8fig5}).}	
\label{fig:app8fig4}
\end{figure}
\newpage

\begin{figure}[H]
\graphicspath{{}}
\includegraphics[width=6in]{HGT_REVISION_2016/figures/figs5_040116.pdf} 
\caption[Topological characteristics of the PGCE network.]{Topological characteristics of the PGCE network. (A) Out-degree distributions of the final PGCE network (nodes with out-degree equal to zero are omitted). (B) In-degree distributions of the final PGCE network (nodes with in-degree equal to zero are omitted). (C-E): Prevalence and gain counts of genes only weakly affect their PGCEs. The degrees of each gene (node) in the PGCE network are plotted against its prevalence (C) and counted gains (D) throughout the tree, and the degrees are plotted against each other (E). Pearson correlations between the plotted variables are indicated above each plot. PCC = Pearson correlation coefficient, p-value is from a correlation test.}	
\label{fig:app8fig5}
\end{figure}
\newpage

\begin{figure}[H]
\graphicspath{{}}
\includegraphics[width=5in]{HGT_REVISION_2016/figures/figs6_092315.pdf} 
\caption[A network of evolutionary dependencies between functional pathways.]{A network of evolutionary dependencies between functional pathways. Overall structure of the evolutionary pathway-pathway dependency network. Directed edges indicate that the source pathway and the sink pathway are connected by more PGCEs between individual genes in those pathways than expected from a rewired null distribution (p < 0.001). Colors indicate selected pathway clusters of similar functions (green: aromatic compound secondary metabolism; red: pathogenesis; purple: carbohydrate metabolism; yellow: DNA metabolism). }	
\label{fig:app8fig6}
\end{figure}
\newpage

\begin{figure}[H]
\graphicspath{{}}
\includegraphics[width=4in]{HGT_REVISION_2016/figures/figs7_020916.pdf} 
\caption[Differences in gain counts do not explain differential sorting of genes in different functional groups.]{Differences in gain counts do not explain differential sorting of genes in different functional groups. (A): Variation in ranks of the sort across functional categories. (B): Total branches in which gains have occurred (gains in tree) across genes in various functional categories that are differentially ranked in a topological sort of the PGCE network. Note that the categories with the highest average gain (Carbohydrate and Xenobiotics metabolism) are ranked in the middle of the sort. See Table 1. }	 % need ref!!
\label{fig:app8fig7}
\end{figure}
\newpage

\begin{figure}[H]
\graphicspath{{}}
\includegraphics[width=5in]{HGT_REVISION_2016/figures/figs8_092215.pdf} 
\caption[Phylogenetic depth of gene gains in bacteria decreases with rank in the topological sort.]{Phylogenetic depth of gene gains in bacteria decreases with rank in the topological sort. Phylogenetic depth of the gains of genes are weakly negatively correlated with their ranks in the sort (Spearmans r = -0.24, p < 10-15). For each rank, we plot the distribution of the phylogenetic depths (distance of gain branch from root) of the average depth of confident gains (Pr(gain) > 0.6)  of each gene in that rank. The mean of each distribution is plotted as a red point. Branches leading to Archaea and archaeal genomes are omitted from the analysis. Boxplot widths are scaled to the number of genes in each rank of the sort. The tree was converted to an ultrametric tree for the purpose of this analysis (the root is separated from all tips by a total branch length of 1.0). }
\label{fig:app8fig8}
\end{figure}
%\newpage

\begin{figure}[H]
\graphicspath{{}}
\includegraphics[width=4.5in]{HGT_REVISION_2016/figures/figs9_013016.pdf} 
\caption[Performance of models predicting the acquisition of genes between clades.]{Performance of models predicting the acquisition of genes between clades. (A) Overlap of edges in PGCE networks inferred from different subsets of the data. See also Table ~\ref{tab:app8tab4}. All overlaps are highly statistically significant (p < 10-15, hypergeometric test). (B) Distribution of prediction scores for gene acquisition on each branch in the test set clades. Branches with a gain (Pr(gain) > 0.5)) have a higher score than branches without a gain (Pr(gain) < 0.5) for predictable genes (p < 10-15 for each, U-test). Predictable genes are the affected genes in at least one PGCE, i.e. they have at least one in-edge in the trained PGCE model. Violin plots show density of each distribution. Each violin plot shows the distribution of prediction scores for branches in one test set for one category (gene gained/gene not gained). (C) Precision/recall plot of PGCE predictions. Notably, the precision by which any particular gain event is predicted is relatively low due to the rarity of true gain events for any particular gene, yet, as demonstrated in Figure 5B and in panel B here, ancestral genome content was overall informative about where along the tree such true gain events occur.}
\label{fig:app8fig9}
\end{figure}
\newpage

\newpage
\section{Supporting Tables}
%\begin{minipage}{\linewidth}
\begin{table}[H]
\small
\caption[Reconciliation analysis supports gene acquisitions inferred by stochastic mapping.]{Reconciliation analysis supports gene acquisitions inferred by stochastic mapping. \label{tab:app8tab1}}
\begin{tabular}{| c c c |}
\hline
Gene (KEGG Orthology) & Predicted gains\footnote{Number of branches where a gain event was inferred for this gene by our stochastic mapping-based approach.} & Supported gains\footnote{Number of gain events predicted by our stochastic mapping-based approach for which at least one descendant had this gene identified as horizontally transferred by reconciliation.}\\
\hline
rbsS (K01602) & 8 & 6\\ napE (K02571)	 & 4 & 3\\parA (K12055) & 10 & 8\\ sctD (K03200) & 8 & 4\\ kpsT (K09689) & 16 & 2\\
\hline
\end{tabular}
\begin{tabular}{| c c c c c c c |}
\hline
Gene & Descendants & Descendants & Not  & Not & Odds & P- \\
(KEGG & with HGT$^1$ & w/o HGT$^2$ & descendants & descendants & ratio & value \\
Orthology) & & & with HGT$^3$ & w/o HGT$^4$ & &  \\
\hline
rbsS & 24 & 7 & 30 & 2411 & 275.5 & < 10-32 \\
(K01602) & & & & & &  \\
napE & 4 & 2 & 102 & 2364 & 46.4 & < 10-4 \\
K02571 & & & & & &  \\
parA & 21 & 4 & 570 & 1877 & 17.3 & < 10-6 \\
K12055 & & & & & &  \\
sctD & 9 & 7 & 90 & 2366 & 33.8 & < 10-11 \\
(K03200) & & & & & &  \\
kpsT & 2 & 30 & 174 & 2266 & 0.87 & 1 \\
K09689 & & & & & &  \\
\hline
\end{tabular}
\end{table}

Table ~\ref{tab:app8tab1} Footnotes \small

 %hacking to get footnotes
1:Number of branches where a gain event was inferred for this gene by our stochastic mapping-based approach.

2:Number of gain events predicted by our stochastic mapping-based approach for which at least one descendant had this gene identified as horizontally transferred by reconciliation.

3:Number of genomes (out of 2472) that are descendants of a stochastic mapping-based gain event and have this gene identified as horizontally transferred by reconciliation.

4:Number of genomes (out of 2472) that are descendants of a stochastic mapping-based gain event but do not have this gene identified as horizontally transferred by reconciliation.

5:Number of genomes (out of 2472) that are not descendants of a stochastic mapping-based gain event but have this gene identified as horizontally transferred by reconciliation.

6:Number of genomes (out of 2472) that are not descendants of a stochastic mapping-based gain event and do not have this gene identified as horizontally transferred by reconciliation.
%\end{minipage}

\begin{longtable}{|c c|}
\caption[Genes which influence the gain of rbsS, gene encoding the RuBisCO small chain.]{Genes which influence the gain of rbsS, gene encoding the RuBisCO small chain.\label{tab:app8tab2}}  \\
\hline
KEGG Orthology (KO) & Description \\
\hline
K02584 & Nif-specific regulatory protein \\
K06139 & pyrroloquinoline quinone biosynthesis protein E \\
K06138 & pyrroloquinoline quinone biosynthesis protein D \\
K06137 & pyrroloquinoline-quinone synthase [EC:1.3.3.11] \\
K06136 & pyrroloquinoline quinone biosynthesis protein B \\
K09165 & hypothetical protein \\
K03809 & Trp repressor binding protein \\
K13483 & xanthine dehydrogenase YagT iron-sulfur-binding subunit \\
K13481 & xanthine dehydrogenase small subunit [EC:1.17.1.4] \\
K02448 & nitric oxide reductase NorD protein \\
K02597 & nitrogen fixation protein NifZ \\
K02596 & nitrogen fixation protein NifX \\
K02595 & nitrogenase-stabilizing/protective protein \\
K02593 & nitrogen fixation protein NifT \\
K02592 & nitrogenase molybdenum-iron protein NifN \\
K02022 & HlyD family secretion protein \\
K11811 & arsenical resistance protein ArsH \\
K08973 & putative membrane protein \\
K12511 & tight adherence protein C \\
K08995 & putative membrane protein \\
K07506 & AraC family transcriptional regulator \\
K10778 & AraC family transcriptional regulator,\\
 & regulatory protein of adaptative response \\
 & / methylated-DNA-[protein]-cysteine \\
 & methyltransferase [EC:2.1.1.63] \\
K07165 & transmembrane sensor \\
K07161 & NA \\
K00830 & alanine-glyoxylate transaminase / \\
 & serine-glyoxylate transaminase \\
 & / serine-pyruvate transaminase \\
 & [EC:2.6.1.44 2.6.1.45 2.6.1.51] \\
K01266 & D-aminopeptidase [EC:3.4.11.19] \\
K05559 & multicomponent K+:H+ antiporter subunit A \\
K02278 & prepilin peptidase CpaA [EC:3.4.23.43] \\
K02279 & pilus assembly protein CpaB \\
K02276 & cytochrome c oxidase subunit III [EC:1.9.3.1] \\
K02274 & cytochrome c oxidase subunit I [EC:1.9.3.1] \\
K02275 & cytochrome c oxidase subunit II [EC:1.9.3.1] \\
K02305 & nitric oxide reductase subunit C \\
K13924 & two-component system, chemotaxis family, \\
 & CheB/CheR fusion protein [EC:2.1.1.80 3.1.1.61] \\
K13926 & ribosome-dependent ATPase \\
K09924 & hypothetical protein \\
K10764 & O-succinylhomoserine sulfhydrylase [EC:2.5.1.-] \\
K07157 & NA \\
K03188 & urease accessory protein \\
K01067 & acetyl-CoA hydrolase [EC:3.1.2.1] \\
K01797 & NA \\
K00824 & D-alanine transaminase [EC:2.6.1.21] \\
K00685 & arginine-tRNA-protein transferase [EC:2.3.2.8] \\
K09796 & hypothetical protein \\
K11177 & xanthine dehydrogenase YagR \\
& molybdenum-binding subunit [EC:1.17.1.4] \\
K11178 & xanthine dehydrogenase YagS \\
& FAD-binding subunit [EC:1.17.1.4] \\
K00329 & NADH dehydrogenase [EC:1.6.5.3] \\
K09008 & hypothetical protein \\
K09005 & hypothetical protein \\
K05563 & multicomponent K+:H+ antiporter subunit F \\
K01800 & maleylacetoacetate isomerase [EC:5.2.1.2] \\
K00253 & isovaleryl-CoA dehydrogenase [EC:1.3.8.4] \\
K02258 & cytochrome c oxidase assembly protein subunit 11 \\
K11962 & urea transport system ATP-binding protein \\
K11963 & urea transport system ATP-binding protein \\
K11960 & urea transport system permease protein \\
K11961 & urea transport system permease protein \\
K05973 & poly(3-hydroxybutyrate) depolymerase [EC:3.1.1.75] \\
K07102 & NA \\
K00023 & acetoacetyl-CoA reductase [EC:1.1.1.36] \\
K15866 & 2-(1,2-epoxy-1,2-dihydrophenyl)acetyl-CoA \\
 & isomerase [EC:5.3.3.18] \\
K04561 & nitric oxide reductase subunit B [EC:1.7.2.5] \\
K05564 & multicomponent K+:H+ antiporter subunit G \\
K05562 & multicomponent K+:H+ antiporter subunit E \\
K05561 & multicomponent K+:H+ antiporter subunit D \\
K05560 & multicomponent K+:H+ antiporter subunit C \\
K02533 & tRNA/rRNA methyltransferase [EC:2.1.1.-] \\
K15011 & two-component system, sensor histidine \\
& kinase RegB [EC:2.7.13.3] \\
K03200 & type IV secretion system protein VirB5 \\
K07303 & isoquinoline 1-oxidoreductase, \\
& beta subunit [EC:1.3.99.16] \\
K07302 & isoquinoline 1-oxidoreductase, \\
& alpha subunit [EC:1.3.99.16] \\
K07234 & uncharacterized protein involved in response to NO \\
K00303 & sarcosine oxidase, subunit beta [EC:1.5.3.1] \\
K02651 & pilus assembly protein Flp/PilA \\
K01055 & 3-oxoadipate enol-lactonase [EC:3.1.1.24] \\
K02502 & ATP phosphoribosyltransferase regulatory subunit \\
K03325 & arsenite transporter, ACR3 family \\
K02225 & cobalamin biosynthetic protein CobC \\
K01991 & polysaccharide export outer membrane protein \\
K04748 & nitric oxide reductase NorQ protein \\
K00304 & sarcosine oxidase, subunit delta [EC:1.5.3.1] \\
K00305 & sarcosine oxidase, subunit gamma [EC:1.5.3.1] \\
K01429 & urease subunit beta [EC:3.5.1.5] \\
K05343 & maltose alpha-D-glucosyltransferase/ \\
 & alpha-amylase [EC:5.4.99.16 3.2.1.1] \\
K06044 & (1->4)-alpha-D-glucan 1-alpha-D-glucosylmutase \\
& [EC:5.4.99.15] \\
K13766 & methylglutaconyl-CoA hydratase [EC:4.2.1.18] \\
K01430 & urease subunit gamma [EC:3.5.1.5] \\
K11959 & urea transport system substrate-binding protein \\
K15012 & two-component system, response regulator RegA \\
K00457 & 4-hydroxyphenylpyruvate dioxygenase [EC:1.13.11.27] \\
K00104 & glycolate oxidase [EC:1.1.3.15] \\
K04756 & alkyl hydroperoxide reductase subunit D \\
K03519 & carbon-monoxide dehydrogenase \\
& medium subunit [EC:1.2.99.2] \\
K09983 & hypothetical protein \\
K06995 & NA \\
K00119 & NA \\
K00449 & protocatechuate 3,4-dioxygenase, \\
& beta subunit [EC:1.13.11.3] \\
K00114 & alcohol dehydrogenase (cytochrome c) [EC:1.1.2.8] \\
K05524 & ferredoxin \\
K02282 & pilus assembly protein CpaE \\
K02280 & pilus assembly protein CpaC \\
K03153 & glycine oxidase [EC:1.4.3.19] \\
K09959 & hypothetical protein \\
K00050 & hydroxypyruvate reductase [EC:1.1.1.81] \\
K08738 & cytochrome c \\
K07018 & NA \\
K00126 & formate dehydrogenase, delta subunit [EC:1.2.1.2] \\
K14161 & protein ImuB \\
K11902 & type VI secretion system protein ImpA \\
K07246 & tartrate dehydrogenase/decarboxylase / \\
 & D-malate dehydrogenase [EC:1.1.1.93 4.1.1.73 1.1.1.83] \\
K03198 & type IV secretion system protein VirB3 \\
K11472 & glycolate oxidase FAD binding subunit \\
K11473 & glycolate oxidase iron-sulfur subunit \\
K11475 & GntR family transcriptional regulator, \\
& vanillate catabolism transcriptional regulator \\
K07649 & two-component system, OmpR family, \\
& sensor histidine kinase TctE [EC:2.7.13.3] \\
K07395 & putative proteasome-type protease \\
K07028 & NA \\
K02391 & flagellar basal-body rod protein FlgF \\
K01601 & ribulose-bisphosphate carboxylase \\
& large chain [EC:4.1.1.39] \\
K03821 & polyhydroxyalkanoate synthase [EC:2.3.1.-] \\
K07168 & CBS domain-containing membrane protein \\
K06923 & NA \\
K00411 & ubiquinol-cytochrome c reductase \\
& iron-sulfur subunit [EC:1.10.2.2] \\
K01941 & urea carboxylase [EC:6.3.4.6] \\
K17226 & sulfur-oxidizing protein SoxY \\
K11897 & type VI secretion system protein ImpF \\
K10125 & two-component system, NtrC family, \\
 & C4-dicarboxylate transport sensor \\
 & histidine kinase DctB [EC:2.7.13.3] \\
K10126 & two-component system, NtrC family, \\
 & C4-dicarboxylate transport response regulator DctD \\
K04090 & indolepyruvate ferredoxin oxidoreductase [EC:1.2.7.8] \\
\hline
\end{longtable}

\begin{table}[H]
\small
\caption[Enrichment analysis of genes influencing the gain of rbsS.]{Enrichment analysis of genes influencing the gain of rbsS.\label{tab:app8tab3}}  
\begin{tabular}{|c c c c c|}
\hline
Annotation label & p-value$^1$ & test set$^2$ & background set$^3$ & Enrichment$^4$ \\
\hline
Nitric oxide reductase (Nor) complex & 6.73E-05 & 4 & 5 & 12.83 \\
Urea transport system (Urt) & 8.62E-07 & 5 & 5 & 16.04 \\
Purine degradation, xanthine=>urea & 0.00042 & 4 & 7 & 9.16 \\
Photorespiration & 8.49E-05 & 5 & 9 & 8.91 \\
Type IV secretion system & 0.0031 & 4 & 11 & 5.83 \\
\hline
\end{tabular}
\end{table}
Table ~\ref{tab:app8tab3} footnotes:

\small
1: from a hypergeometric test.
2: the number of genes with this annotation appearing in Supplemental Table S1 (out of 88 genes).
3: the number of genes with this annotation appearing in the set of all genes in the PGCE network (out of 2472 genes).
4: The ratio of the observed proportion of genes with this label to the expected proportion.
5: The annotation of these genes to the same pathway is not present in KEGG, so this enrichment is derived from our manual annotation.

\begin{table}[H]
\small
\caption[Summary of nodes (genes) ranked by their order in a topological sort.]{Summary of nodes (genes) ranked by their order in a topological sort.} \label{tab:app8tab4} 
\begin{tabular}{|c c c c|}
\hline
Rank & Number of genes & Total out-degree & Total in-degree \\
\hline
1 & 1593 & 7792 & 0 \\
2 & 498 & 357 & 2512 \\
3 & 118 & 73 & 2348 \\
4 & 46 & 6 & 2992 \\
5 & 5 & 0 & 376 \\
\hline
\end{tabular}
\end{table}

\begin{table}[H]
\small
\caption[Characteristics of PGCE network models inferred from data subsets.]{Characteristics of PGCE network models inferred from data subsets.\label{tab:app8tab5} } 
\begin{tabular}{|c c c c|}
\hline
Dataset$^1$ & \# PGCEs & ROC AUC$^2$ & Predictable / Total$^3$ \\
\hline
All (predicting Firmicutes)c & 8,228 & 0.80 & 667 / 3281 \\
Lacking Firmicutes & 3,703 & 0.73 & 394 / 3281 \\
Lacking A/B-proteobacteria & 1,726 & 0.68 & 204 / 3505 \\
\hline
\end{tabular}
\end{table}
Table ~\ref{tab:app8tab5} footnotes: 

\small
1: The dataset used to train the PGCE model in question. Predictions are made concerning the test set (dataset lacking Firmicutes predicts Firmicutes).
2: Area under the curve of the receiver operating characteristic curve; a random prediction is 0.5, a perfect prediction is 1.0.
3: The number of genes that are predictable using each dataset to train PGCE models, compared to the total number of genes that are actually gained at least once (defined as Pr(gain) > 0.5) in the test set clade. 
\end{document}
